#+TITLE:Oxidative Phosphorylation
#+AUTHOR: Matthew Henderson, PhD, FCACB
#+DATE: \today

* Introduction
- oxidative phosphorylation is the process in which ATP is formed as a
  result of the transfer of electrons from NADH or FADH_2 to O_2 by a
  series of electron carriers (Figure [[fig:etc]] and [[fig:oxpho]])
- responsible for ATP production
- ETC complexes I-IV
- OxPhos system complexes I-V
- ~ 90 subunits
- 13 subunits of Complexes I, III, IV and V encoded by mtDNA (Table [[tab:subunits]])
- \gt 1500 proteins in the human mitochondrial proteome
- up to 10% of human proteome may have role in maintaining mitochondrial function
- OxPhos requires mitochondrial functions for:
  - protein import, turnover
  - fission, fusion
  - mtDNA replication, transcription and translation
    - \gt 200 proteins, rRNAs and tRNAs
  - cofactors: coenzyme Q_{10}, iron-sulfur clusters, heme, copper
  - cardiolipin for cristae formation

- ETC and OxPhos are coupled by a proton gradient across the IMM
  - efflux of protons from the mitochondrial matrix creates an
    electrochemical gradient
    - used by the F_{0}F_{1} ATP synthase complex to make ATP via oxidative phosphorylation
- energy from transfer of electrons down the ETC is used to pump
  protons from the mitochondrial matrix into the intermembrane space
  - creates an electrochemical proton gradient (\Delta{}pH) across the IMM
    - largely responsible for the mitochondrial membrane potential (\Delta\Psi{}M)
  - ATP synthase uses flow of \ce{H+} through the enzyme back into the
    matrix to generate ATP from ADP and Pi
- three energy-transducing enzymes in the electron transport chain:
  - NADH:ubiquinone oxidoreductase (complex I)
  - Coenzyme Q – cytochrome c reductase (complex III)
  - Cytochrome c oxidase (complex IV)
  - ETF-QO and mitochondrial glycerol-3-P dehydrogenase (GPD) (not part of ETC)

#+BEGIN_SRC dot :file ./oxphos/figures/etc.pdf :cmdline -Kdot -Tpdf
  digraph {
           // graph from left to right
           rankdir=LR;
           splines=true;
           node [shape=oval];
           fadh [label="FADH2", fillcolor=grey, style=filled];
           etf[label="ETF", fillcolor=steelblue3, style=filled];
           etfd[label="ETF-QO", fillcolor=steelblue3, style=filled];
	       g3p[label="glycerol-3-phosphate", fillcolor=grey, style=filled];
           gpd[label="GPD", fillcolor=steelblue3, style=filled];
           nadh [label="NADH", fillcolor=grey, style=filled];
           c1 [label="Complex I", fillcolor=steelblue3, style=filled];
           q [label="CoQ", fillcolor=red3, style=filled];    
           c3 [label="Complex III", fillcolor=steelblue3, style=filled];     
           cytc [label="Cytochrome C", fillcolor=red, style=filled];     
           c4 [label="Complex IV", fillcolor=steelblue3, style=filled];     
           h2o [label="H2O", fillcolor=grey, style=filled];     
           c2 [label="Complex II", fillcolor=steelblue3, style=filled];     
           s [label="succinate", fillcolor=grey, style=filled];     
           fadh -> etf; 
           etf -> etfd;
           etfd -> q;
           g3p -> gpd;
           gpd -> q;
           nadh -> c1;
           c1 -> q;
           q -> c3;
           c3 -> cytc; 	 
           cytc -> c4;
           c4 -> h2o;
           c2 -> q;
           s -> c2;
           }    
#+END_SRC

#+CAPTION[]:Flow of electrons in the ETC
#+NAME: fig:etc
#+ATTR_LaTeX: :width 1.1\textwidth
#+RESULTS:
[[file:./figures/etc.pdf]]

- Complex I (NADH coenzyme Q reductase) accepts electrons from the
  Krebs cycle electron carrier NADH
- passes them to CoQ (ubiquinone)
- CoQ also receives electrons from CII, GDP and ETF-QO

- CoQ passes electrons to complex III (cytochrome bc1 complex) which
  passes them to cytochrome c
- cytochrome c passes electrons to Complex IV (cytochrome c oxidase)
  which uses the electrons and hydrogen ions to reduce molecular
  oxygen to water

#+CAPTION[]:Location of DNA encoding OxPhos Complex Subunits 
#+NAME: tab:subunits
|       | CI | CII | CIII | CIV | CV |
|-------+----+-----+------+-----+----|
| mtDNA |  7 |   0 |    1 |   3 |  2 |
| nDNA  | 39 |   4 |   10 |  10 | 14 |

#+CAPTION[ETC]: Oxidative Phosphorylation
#+NAME: fig:oxpho
#+ATTR_LaTeX: :width \textwidth
[[file:./figures/Slide14.png]]

#+CAPTION[redox]:Electron flow to O_2 and release free energy
#+NAME: fig:redox
#+ATTR_LaTeX: :width 0.6\textwidth
[[file:./figures/potential.png]]

** Complex I
- AKA NADH-ubiquinone oxidoreductase
- catalyzes the transfer of electrons from NADH to CoQ and
  translocates protons across the inner mitochondrial membrane

#+BEGIN_EXPORT LaTeX
{\small\ce{NADH + H+ + CoQ + 4H^{+}_{in} ->[CI] NAD+ + CoQH2 + 4H^{+}_{out}}}
#+END_EXPORT

  - mechanism: 
    1. seven iron sulfur centers carry electrons from the site of NADH
       oxidation to ubiquinone (Figure [[fig:c1]])
    2. ubiquinone (CoQ) is reduced to ubiquinol (\ce{CoQH2})
    3. the energy from the redox reaction results in conformational
       change allowing hydrogen ions to pass through four transmembrane
       helix channels

  #+BEGIN_SRC dot :file ./figures/c1.pdf :cmdline -Kdot -Tpdf
    digraph {
             // graph from left to right
             rankdir=LR;
             splines=true;
             node [shape=circle];
             nadh [label="NADH", fillcolor=grey, style=filled]
             fmn [label="FMN", fillcolor=forestgreen, style=filled]
             n3 [label="N3", fillcolor=red, style=filled]    
             n1b [label="N1b", fillcolor=red, style=filled]     
             n4 [label="N4", fillcolor=red,style=filled]     
             n5 [label="N5", fillcolor=red, style=filled]     
             n6a [label="N6a", fillcolor=red, style=filled]     
             n6b [label="N6b", fillcolor=red, style=filled]     
             n2 [label="N2", fillcolor=red, style=filled]     
             q [label="Q", fillcolor=steelblue3, style=filled]     
             nadh-> fmn
             fmn -> n3
             n3 -> n1b
		 n1b -> n4
             n4 -> n5 
             n5 -> n6a
             n6a -> n6b
             n6b-> n2
             n2 -> q
             }    
  #+END_SRC

  #+CAPTION[c1]:Complex I | NADH-ubiquinone oxidoreductase
  #+NAME: fig:c1
  #+ATTR_LaTeX: :width 0.9\textwidth
  #+RESULTS:
  [[file:./figures/c1.pdf]]

** Complex II
- AKA Succinate Dehydrogenase 
 - see section [[Succinate Dehydrogenase Deficiency]]

** Glycerol-3-phosphate shuttle
 - oxidation of cytoplasmic NADH by the cytosolic form of the enzyme
   creates glycerol-3-phosphate from dihydroxyacetone phosphate
 - glycerol-3-phosphate diffuses into IMM and is oxidised by mitochondrial glycerol-3-phosphate dehydrogenase
   - uses quinone as an oxidant and FAD as a co-factor
 - maintains the cytoplasmic NAD^{+}/NADH ratio

 #+CAPTION[g3ps]:Glycerol-3-phosphate shuttle
 #+NAME: fig:g3ps
 #+ATTR_LaTeX: :width 0.6\textwidth
 [[file:./figures/GPDH_shuttle.png]]

** Electron Transferring Flavoprotein/ Dehydrogenase
 - ETFs are heterodimeric proteins composed of an \alpha and \beta
   subunit (ETFA and ETFB)
   - contain an FAD cofactor and AMP
 - ETQ-QO links the oxidation of fatty acids and some amino acids to
   oxidative phosphorylation in the mitochondria
 - catalyzes the transfer of electrons from electron transferring
   flavoprotein (ETF) to ubiquinone reducing it to ubiquinol

 #+BEGIN_EXPORT LaTeX
 {\small\ce{Acyl-CoA + FAD ->[ACAD] FADH2 + ETF ->[ETF-QO] CoQ ->[CIII] CytC}}
 #+END_EXPORT

 - ETF-QO deficiency results in *Glutaric Acidemia Type II* (AKA MADD)
   - see section FAODs
** Complex III
- AKA Coenzyme Q Cytochrome C Reductase
- Complex III is a multi-subunit transmembrane protein encoded by both
  mitochondrial (cytochrome B) and the nuclear genomes (all other
  subunits)

  - Complex III contains 11 subunits:
    - 3 respiratory subunits (cytochrome B, cytochrome C1, Rieske protein)
    - 2 core proteins
    - 6 low-molecular weight proteins

  #+BEGIN_EXPORT LaTeX
  {\small\ce{QH2 + 2Fe^{3+}-cytC + 2H+_{in} ->[CIII]  Q + 2Fe^{2+}-cytC + 4H+_{out}}}
  #+END_EXPORT

** Complex IV
- AKA Cytochrome c oxidase
- last enzyme in the respiratory electron transport chain
- large IMM integral membrane protein composed of several metal prosthetic sites and 14 protein subunits
- eleven subunits are nuclear in origin, and three are synthesized in the mitochondria
  - contains two hemes, cytochrome A and cytochrome A3
  - two copper centers, CuA and CuB
  - cytochrome a3 and CuB form a binuclear center that is the site of
    oxygen reduction
  - receives an electron from four cytochrome C molecules and
    transfers them to one O_2 molecule
 
  #+BEGIN_EXPORT LaTeX
  {\small\ce{4Fe^{2+}-cytC + 8H^{+}_{in} + O2 ->[CIV] 4Fe^{3+}-cytC + 2H2O + 4H^{+}_{out}}}
  #+END_EXPORT

  - binds four protons from the inner aqueous phase to make two water
    molecules
  - translocates another four protons across the membrane

  #+CAPTION[cIV]:Complex IV | Cytochrome c oxidase
  #+NAME: fig:cIV
  #+ATTR_LaTeX: :width 0.5\textwidth
  [[file:./figures/CIV.png]]

** Complex V
- AKA ATP synthase 
- ATP synthase is a molecular machine that creates ATP
- overall reaction catalyzed by ATP synthase is:

#+BEGIN_EXPORT LaTeX
{\small\ce{ADP + P_i + H+_{out} <=>[CV] ATP + H2O + H+_{in}}}
#+END_EXPORT

- formation of ATP from ADP and P_{i} is energetically unfavourable
  - would normally proceed in the reverse direction

- to drive this reaction forward, ATP synthase couples ATP synthesis
  to the electrochemical gradient (\Delta\Psi{}M) created by complexes
  I, III and IV
- ATP synthase consists of two main subunits, F_0 and F_1,
  - a rotational motor mechanism \to ATP production

   #+CAPTION[]: Complex V | ATP synthase 
   #+NAME: fig:atps
   #+ATTR_LaTeX: :width 0.5\textwidth
   [[file:./figures/atp_synthase.jpg]]

* Clinical Presentation
- molecular pathogenesis of OxPhos disorders (Figure [[fig:oxphopath]])
  1) OxPhos complexes or assembly - Leigh Syndrome (COX assembly)
  2) mtDNA maintainance - MDDS, MELAS (POLG)
  3) mitochondrial translation - MIDD (MT-TL)
  4) CoQ biosynthesis - CoQ deficiency
  5) mitochondrial protein import - TOM and TIM 
  6) mitochondrial solute import - PEO (ANT1)
  7) mitochondrial membrane lipid synthesis - Barth (TAZ)
  8) motility of mitochondrial membranes - MEGDEL (SERAC1)

#+CAPTION[ETC]: Pathogenesis of OxPhos Disorders
#+NAME: fig:oxphopath
#+ATTR_LaTeX: :width \textwidth
[[file:./figures/Slide15.png]]


#+CAPTION[]:mtDNA disorders
#+NAME: fig:mtdna
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/mtDNA_disorders.png]]

 - clinical diagnosis is extremely challenging
   - can affect any organ system
   - antenatal (IUGR, birth defects) \to elderly (myopathy)

 - clinical suspicion based on:
   1) constellation of symptoms/signs consistent with a mitochondrial syndrome (Table [[tab:oxpclin]])
   2) complex multi-system presentation involving two/more organ systems,
      best described by an underlying disorder of energy generation
   3) lactic acidosis, characteristic neuro-imaging, 3-methylglutaconic
      aciduria, ragged red fiber mypopathy
   4) pathogenic mutation in a known mitochondrial disease gene

 #+CAPTION[]:Clinical Features of Mitochondrial Disease
 #+NAME: tab:oxpclin
 | System          | Manifestation                      |
 |-----------------+------------------------------------|
 | CNS             | *myoclonus* (muscle jerks)         |
 |                 | *seizures*, *ataxia*               |
 |                 | encephalopathy                     |
 |                 | strokes, GDD                       |
 |                 | peripheral neuropathy              |
 | Skeletal muscle | *myopathy, hypotonia*              |
 |                 | rhabdomyolysis                     |
 | Marrow          | sideroblastic anemia/pancytopenia  |
 | Kidney          | Fanconi                            |
 | Endocrine       | *diabetes*                         |
 |                 | hypoparathyroidism                 |
 |                 | growth/multiple hormone deficiency |
 | Heart           | cardiomyopathy                     |
 |                 | conduction defects                 |
 | GI              | pancreatic failure                 |
 |                 | villous atrophy                    |
 | Ear             | *sensorineural deafness*           |
 | Eye             | *CPEO*, ptosis, RP                 |
 | Systemic        | *lactic acidosis*, hypoglycemia    |
 |                 | IUGR, FTT                          |


*** Neonatal and Infantile Presentation
**** *Barth*
- cardiomyopathy, myopathy
  - XL :: TAZ
- Tafazzin catalyzes the remodeling of the acyl chains of immature
  cardiolipin to a mature, predominantly tetralinoleylcardiolipin
**** *Leigh Syndrome*
- AKA: subacute necrotising encephalopathy
- most frequent presentation of MD in childhood
- GDD, growth retardation
- hypo/er-ventilation, spasticity, dystonia, ataxia, tremor, optic atrophy
- cardiomyopathy, renal tubulopathy, GI disfunction
- \gt 75 genes(mt and nuclear)
  - mtDNA :: MT-ATP6 and other
  - AR :: SURF1, SDHA, COX assemmbly, CI
  - XL :: PDHA1
- Saguenay-Lac-St-Jean type incidence 1/2000, carriers 1/23
- several biochemical defects including: PDH, OxPhos
- \uparrow lactate blood and CSF
**** *MEGDEL*: 3-Methylglutaconic aciduria, Deafness, Encephalopathy and Leigh-like Disease
- phosphatidylglycerol remodeling protein found at the interface of
  mitochondria and ER
- mediates phospholipid exchange
  - AR :: SERAC1

**** *Pearson's marrow-pancreas syndrome*
 - transfusion dependent sideroblastic anemia/pancytopenia
 - exocrine pancreas failure
 - progressive liver disease
 - renal tubular disease
  - mtDNA :: large del/dup sporadic
**** *MDDS*: mitochondrial DNA depletion syndrome
- hepatocerebral disease
- progressive myopathy
  - AR :: POLG, DGUOK, Twinkle, SUCLA2, TK
**** *Alpers syndrome*
- progressive neuronal degradation of childhood
- seizures
  - AR :: POLG
**** *Reversible infantile respiratory chain deficiency*
- protein translation defect
- glutamic acid tRNA
  - mtDNA :: MT-TE
**** *CoQ deficiency*
- multisystem involvement including neurologic manifestations
  - fatal neonatal encephalopathy with hypotonia
  - late-onset slowly progressive multiple-system atrophy
    - mtDNA :: COQ genes
*** Childhood and Adolescent Presentation
**** *Kearns-Sayre Syndrome* 
- subtype of CPEO w earlier onset
- onset \lt 20
- pigmentary retinopathy
- cardiac conduction defect
- ataxia, \uparrow CSF protein
- isolated limb myopathy
  - mtDNA :: deletions \pm duplications
  - AR :: RRM2B
**** *MELAS*: Myopathy, Encephalopathy, Lactic Acidosis, Stroke-like Episodes
- most common mito encephalopathy
- RRF, PEO
  - mtDNA :: 80%
  - AR :: POLG
**** *MERRF*: myoclonic epilepsy w ragged red fibres
- ptosis (drooping eyelids), ataxia, deafness
  - mtDNA :: MT-TK, others
**** *NARP*: neuropathy, ataxia, retinitis pigmentosa
  - mtDNA :: MT-ATP6
**** *LHON*: Leber’s Hereditary Optic Neuropathy
 - most common cause of blindness in otherwise healthy young men
   - males \gt females 4:1
 - maternally inherited and manifests in late adolescence or early
   adulthood as bilateral sequential visual failure
 - 90% of patients are affected by age 40
   - mtDNA :: MT-ND1, MT-ND4, MT-ND6
**** *MNGIE*: Mitochondrial Neuro-Gastro-Intestinal Encephalopathy
- demyelinating neuropathy
- leukoencephalopathy
- \uparrow plasma & urine thymidine and deoxyuridine
  - AR :: TYMP
*** Adult Presentation
**** *MIDD*: maternally inherited diabetes and deafness
- mtDNA :: MT-TL1
**** *PEO*: Progressive External Ophthalmoplegia 
- ptosis
- mtDNA quantitation in muscle biopsy is diagnostic
  - mtDNA :: deletions \pm duplications
  - AR :: POLG, ANT1, twinkle

**** *SANDO*: Sensory Ataxic Neuropathy, dysarthria and opthalmoparesis
- AR :: POLG

* Metabolic Derangement
- insufficient ATP severely affects highly energy dependant tissues
  - a complete loss of OxPhos is not observed in human disease 
- in the absence of OxPhos cells survive using ATP from anaerobic glycolysis
  - 20x less efficient generates lactate
- lactate, pyruvate and alanine are the typical products of anaerobic glycolysis

* Diagnostic Tests
#+CAPTION[]: Diagnosis of Mitochondrial Disease
#+NAME: fig:alg
#+ATTR_LaTeX: :width 1.2\textwidth
[[file:./figures/algorithm_rot.png]]


** Biochemistry
- none of the following are specific for OxPhos disorders    
- \uparrow blood lactate, CSF lactate
- \uparrow L/P at rest \Uparrow after exercise
- renal tubular dysfunction: urine anion gap, pH, serum K
*** plasma amino acids:
 - alanine \propto pyruvate
   - \uparrow alaine \gt 450 uM
   - \uparrow alanine/lysine \gt 3:1
   - \uparrow alanine/(phenylalanine + tyrosine) \gt 4:1
 - \uparrow glyine in lipoic acid biosynthesis defects
 - \uparrow proline due to \downarrow proline oxidase activity
 - \downarrow citrulline and arginine in Leigh, NARP, MELAS and Pearson
*** urine amino acids:
 - aminoaciduria in renal Fanconi
   - mtDNA deletions
*** urine organic acids:
 - lactate, pyruvate, TCA intermediates
   - renal immaturity \to \uparrow TCA intermediates
 - 3-methylglutaconic acid in Barth, Sengers, MEGDEL, ATP synthase deficiency
 - ethylmalonic
 - MMA in succinyl-CoA ligase deficiency
 - dicarboxylic aciduria
*** plasma acylcarnitines
 - flavin cofactor metabolism
*** purine and pyrimidines (plasma or urine)
  - MNGIE \uparrow thymidine and deoxyuridine
- FGF-21, GDF15 and creatinine \propto mito disfunction in myopathy

*** L/P ratio
    - PDHC :: N-\downarrow
    - TCA :: N-\uparrow
    - ETC :: \uparrow 
*** OxPhos subunit activity
- CII is encoded by nuclear DNA (Table [[tab:subunits]])
  - \therefore not affected by mtDNA depletion
- CoQ deficiency results in:
  - \downarrow combined CI + CIII activity
  - \downarrow combined CII + CIII activity
  - normal individual CI, CII, and CII activity

** Imaging
- cranial CT shows cerebral and cerebellar atrophy in many encephalopathic patients
  - basal ganglia calcification may be seen in MELAS
- MRI in MELAS-associated stroke reveals increased T2 weighted signals in the grey and white matter
- symmetrical changes in the basal ganglia and brainstem observed in Leigh syndrome

** Histology
- muscle biopsy is diagnostic
  - mitochondrial myopathy due to mtDNA mutations and LHON may have normal biopsies
- ragged red fibres on Gomori trichrome staining, due to mitochondrial proliferation
- fibres stain strongly for succinate dehydrogenase
- fibres often negative for COX (complex IV) in CPEO, KSS, or MERRF but positive in MELAS
- Leigh syndrome patients may have no ragged red fibres and  COX-negative fibres only

#+CAPTION[rrf]:Ragged red fibers - Gomori stain
#+NAME: fig:rrf
#+ATTR_LaTeX: :width 0.5\textwidth
[[file:./figures/Ragged_red_fibers_in_MELAS.jpg]]

** Molecular
- no strict relation between phenotype and genotype
- common mutation affect tRNAs, nucleotide metabolism, fusion
- mtDNA tRNA mutations are most common of the single base change abnormalities
  - MT-TP :: tRNA^{Pro}
  - MT-TL1 :: tRNA^{Leu(UUR)}
  - MT-TK :: tRNA^{Lys}
  - Twinkle :: mtDNA helicase
  - POLG :: polymerase \gamma
  - TYMP :: thymidine phosphorylase
  - RRM2B :: ribonucleotide reductase
  - DGUOK :: deoxyguanosine kinase
  - ANT1 :: adenine nucleotide translocase type 1
  - OPA1 :: fusion


* Treatment
- Coenzyme Q disorders :: high does coenzyme Q
- CI deficiency :: some are riboflavin responsive
- vitamin and cofactor cocktails have limited evidence:
  - coenzyme Q
  - thiamine (B_1)
  - riboflavin (B_2)

