% Created 2020-06-27 Sat 11:52
% Intended LaTeX compiler: pdflatex
\documentclass[12pt]{scrartcl}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{longtable}
\usepackage{wrapfig}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{amsmath}
\usepackage{textcomp}
\usepackage{amssymb}
\usepackage{capt-of}
\usepackage{hyperref}
\hypersetup{colorlinks,linkcolor=black,urlcolor=blue}
\usepackage{textpos}
\usepackage{textgreek}
\usepackage[version=4]{mhchem}
\usepackage{chemfig}
\usepackage{siunitx}
\usepackage{gensymb}
\usepackage[usenames,dvipsnames]{xcolor}
\usepackage{lmodern}
\usepackage{verbatim}
\usepackage{tikz}
\usepackage{wasysym}
\usetikzlibrary{shapes.geometric,arrows,decorations.pathmorphing,backgrounds,positioning,fit,petri}
\usepackage[automark, autooneside=false, headsepline]{scrlayer-scrpage}
\clearpairofpagestyles
\ihead{\leftmark}% section on the inner (oneside: right) side
\ohead{\rightmark}% subsection on the outer (oneside: left) side
\addtokomafont{pagehead}{\upshape}% header upshape instead of italic
\ofoot*{\pagemark}% the pagenumber in the center of the foot, also on plain pages
\pagestyle{scrheadings}
\author{Matthew Henderson, PhD, FCACB}
\date{\today}
\title{Methods}
\hypersetup{
 pdfauthor={Matthew Henderson, PhD, FCACB},
 pdftitle={Methods},
 pdfkeywords={},
 pdfsubject={},
 pdfcreator={Emacs 26.3 (Org mode 9.3.7)}, 
 pdflang={English}}
\begin{document}

\maketitle
\setcounter{tocdepth}{2}
\tableofcontents


\tikzstyle{chemical} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=black, fill=none]
\tikzstyle{hardware} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=black, fill=gray!30]
\tikzstyle{ms} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=orange, fill=none]
\tikzstyle{msw} = [rectangle, rounded corners, text width=7em, minimum height=1em,text centered, draw=orange, fill=none]
\tikzstyle{label} = [rectangle,text width=8em, minimum height=1em, text centered, draw=none, fill=none]
\tikzstyle{hl} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=black, fill=red!30]
\tikzstyle{box} = [rectangle, rounded corners, text width=5em, minimum height=5em,text centered, draw=black, fill=none]
\tikzstyle{arrow} = [thick,->,>=stealth]
\tikzstyle{hl-arrow} = [ultra thick,->,>=stealth,draw=red]
\setchemfig{atom style={scale=0.55}}

\section{Amino Acids}
\label{sec:org4ee91fa}
\subsection{Introduction}
\label{sec:org64893dc}
\begin{itemize}
\item amino acids are mono or dicarboxylic acids with one or more amino groups
\begin{itemize}
\item zwitterion at pH 7.45
\end{itemize}

\item proteinogenic amino acids (22)
\begin{itemize}
\item 21 amino acids naturally incorportated into polypeptides in humans
\item 20 genetically encoded
\item selenocysteine
\end{itemize}

\item non-proteinogenic
\begin{itemize}
\item post-translational modification
\begin{itemize}
\item hydroxylation of proline \(\to\) hydroxyproline
\end{itemize}
\item not found in proteins
\begin{itemize}
\item gamma-aminobutryic acid
\item ornithine, citrulline
\end{itemize}
\end{itemize}

\item 76 amino acids of biological interest in humans
\end{itemize}

\subsubsection{Indications for Measurement of Amino Acids}
\label{sec:orgda48e26}
\begin{itemize}
\item diagnosis of inborn errors of amino acid metabolism and transport
\item diet monitoring in patients with known IEM
\item nutritional assessment of patients with non-metabolic conditions (e.g. short bowel syndrome)
\item signs and symptoms:
\begin{enumerate}
\item lethargy, coma, seizures or vomiting in a neonate
\item hyperammonaemia
\item ketosis
\item metabolic acidosis or lactic acidaemia
\item alkalosis
\item metabolic decompensation
\item unexplained developmental delay or developmental regression
\item polyuria, polydipsia and dehydration
\item unexplained liver dysfunction
\item unexplained neurological symptoms
\item abnormal amino acid results on newborn screening
\item previous sibling with similar clinical presentation
\item clinical presentation specific to an amino acid disorder
\item monitoring treatment and diet
\end{enumerate}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org8e77028}Recommended Plasma AA Profile for Diagnosis of Amino Acid Disorders}
\centering
\begin{tabular}{lll}
alanine & glutamine & ornithine\\
alloisoleucine & glycine & phenylalanine\\
arginine & histidine & proline\\
argininosuccinic acid & homocysteine \footnotemark & serine\\
cystine & isoleucine & sulphocysteine \footnotemark\\
citrulline & leucine & taurine\\
glutamic acid & lysine & threonine\\
valine & methionine & tyrosine\\
\end{tabular}
\end{table}\footnotetext[1]{\label{orgd3364b7}plasma total homocysteine is not detected by routine methods, plasma free homocystine analysis has poor clinical sensitivity.}\footnotetext[2]{\label{org38b16b8}sulphocysteine may not be detectable in plasma using routine methods}

\subsubsection{Sample Types}
\label{sec:orge22d845}
\begin{enumerate}
\item Plasma
\label{sec:org13203ea}
\begin{itemize}
\item patient prep
\begin{itemize}
\item fasting (overnight preferred, 4 hours minimum)
\item infants and children should be drawn just before next feeding
(2-3 hours without TPN if possible)
\end{itemize}
\item sample
\begin{itemize}
\item Li-heparin venous plasma
\end{itemize}
\item pre-analytical
\begin{itemize}
\item prompt separation and deproteinisation is essential
\begin{itemize}
\item accurate measurement of (free) sulphur containing amino acids
\item \(\downarrow\) protein binding: cystine and homocystine
\item \(\downarrow\) release of arginase from RBCs
\end{itemize}
\item store at -20\degree{}C to limit glutamine decomposition
\item serum should not be used because there may be deamination
(asparagine to aspartic acid and glutamine to glutamic acid), loss
of sulphur containing amino acids and release of oligopeptides
\item EDTA plasma is recommended in some labs as the specimen of
choice
\begin{itemize}
\item older literature reports ninhydrin positive artefacts in EDTA
plasma but modern tubes do not seem to have this problem
\end{itemize}
\item hemolysis will cause increased serine, glycine, taurine,
phosphoethanolamine, aspartic acid, glutamic acid, ornithine and
decreased arginine
\item delayed separation or leucocyte and platelet contamination will
cause increased serine, glycine, taurine, phosphoethanolamine,
ornithine, glutamic acid and decreased arginine, homocystine,
cystine, phenylalanine and tyrosine
\item amino acids are more stable in frozen deproteinised plasma than
in frozen native plasma
\item capillary blood may be used with careful cleaning of the skin prior
to specimen collection provided the blood is flowing freely
\item free tryptophan may be lost when using sulphosalicylic acid as
deproteinising agent
\begin{itemize}
\item trichloroacetic acid is the deproteinising agent of choice for
this amino acid
\end{itemize}
\item sodium metabisulphite, found in some intravenous preparations as a
preservative, can cause the conversion of cystine to sulphocysteine
\end{itemize}
\end{itemize}

\item Urine
\label{sec:org7ad49ad}
\begin{itemize}
\item aminoaciduria is due to overflow and amino acid transport defects
\item 24 hour or random urine
\item preservative free bottle
\item specimen quality is checked by testing for nitrite and pH
\begin{itemize}
\item specimen deterioration causes:
\begin{itemize}
\item \(\uparrow\) glycine due to bacteria
\item \(\downarrow\) serine
\item \(\uparrow\) or \(\downarrow\) alanine
\item decarboxylation of glutamic acid \(\to\) \(\gamma\)-aminobutyric acid
\item breakdown of phosphoethanolamine \(\to\) ethanolamine and phosphate
\item breakdown of cystathionine \(\to\) homocystine
\item hydrolysis of peptides causing \(\uparrow\) proline
\end{itemize}
\item fecal contamination causes increased proline, glutamic acid and
branched chain amino acids
\item fecal bacteria can produce \(\gamma\)-aminobutyric acid from glutamic acid and b-alanine from aspartic acid
\end{itemize}
\item reported in \textmu{}mol/g creatinine
\end{itemize}

\item Cerebral Spinal Fluid
\label{sec:org588a6a5}
\begin{itemize}
\item investigation of neurological disorders and essential for the
diagnosis of non-ketotic hyperglycinaemia
\item CSF/plasma ratio of amino acids is more informative than an isolated CSF sample
\begin{itemize}
\item a paired plasma sample should be obtained within two hours
\end{itemize}
\item Li-heparin collection tube
\item free of blood contamination
\end{itemize}
\end{enumerate}

\subsubsection{Sample prep}
\label{sec:org2e2be60}
\begin{itemize}
\item hydrophillic amino acids require deprotonization with acetonitrile or alcohol
\item deproteination to release cysteine, homocysteine and tryptophan
\end{itemize}

\subsection{Amino Acid Analyser}
\label{sec:orga0ca5cc}
\begin{itemize}
\item cation-exchange chromatography using a lithium buffer system
followed by post-column derivatization with ninhydrin
\item samples are de-proteinized with sulfosalicylic acid prior to
injection
\item utilizes a lithium-based cation-exchange column
\item eluting amino acids undergo post column reaction with ninhydrin
\item optical detection in the visible spectrum
\begin{itemize}
\item amino acids: 570nm
\item imino acids 440 nm
\end{itemize}
\end{itemize}

\begin{center}
\begin{tikzpicture}[node distance=9em]
% nodes
\node(column)[msw]{chromatography};
\node(derivatization)[msw, right of=column]{ninhyrin derivatization};
\node(detector)[ms, right of=derivatization]{UV detector};
% arrows
\draw[arrow](column) -- (derivatization);
\draw[arrow](derivatization) -- (detector);
\end{tikzpicture}

\vspace{2em}
\chemnameinit{}
\schemestart
\chemname{\chemfig{*6(=*5(-(=O)-(-[6]OH)(-[8]OH)-(=O)-)-=-=-)}}{\small ninhydrin}
\+
\chemname{\chemfig{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]OH}}{\small \textalpha{}-amino acid}
\arrow{-U>[][{\small \ce{2H2O}}]}
\chemname{\chemfig{*6(=*5(-(=O)-(=N-[::-60](-[::-60]{\color{red}R})-[::60](=[::60]O)-[::-60]OH)-(=O)-)-=-=-)}}{\small derivative}
\arrow{-U>[{\small ninhydrin}][]}
\chemname{\chemfig{*6(=*5(-(=O)-(-N=*5(-(=O)-(*6(-=-=-))--(=O)-))-(=O)-)-=-=-)}}{\small 570nm}
\schemestop

\chemnameinit{}
\schemestart
\chemname{\chemfig{*6(=*5(-(=O)-(-[6]OH)(-[8]OH)-(=O)-)-=-=-)}}{\small ninhydrin}
\+
\chemname{\chemfig{H-*5(N----(-COOH)-)}}{\small imino acids}
\arrow{->[][]}
\chemname{\chemfig{*6(=*5(-(=O)-(-*5(N-----))-(=O)-)-=-=-)}}{\small 440nm}
\schemestop
\end{center}

\begin{figure}[htbp]
\centering
\includegraphics[width=1.1\textwidth]{aa/figures/aaachrom.png}
\caption{\label{fig:org1b4e486}Amino Acids Analyser}
\end{figure}

\subsubsection{Pros and Cons of Amino Acid Analysers}
\label{sec:org59eb0dc}
\begin{enumerate}
\item Pros
\label{sec:org74f5aa5}
\begin{itemize}
\item standardized and established technology
\item interpretive experience
\end{itemize}
\item Cons
\label{sec:orgdbbb3dd}
\begin{itemize}
\item long run time (90-150 minutes/sample)
\item lack of analyte specificity (identification by retention time)
\begin{itemize}
\item interference by antibiotics (ampicillin, amoxicillin, and
gentamicin) and acetaminophen
\end{itemize}
\item co-eluting substances cannot be separated and distinguished on a standard IEC chromatogram
\begin{itemize}
\item homocitrulline co-elutes with methionine
\item ASA co-elutes with leucine
\item alloisoleucine co-elutes with cystathionine
\item tryptophan co-elutes with histidine
\end{itemize}
\end{itemize}
\end{enumerate}

\subsection{LC-MS/MS}
\label{sec:org7154bd1}
\subsubsection{LC-MS/MS schematic}
\label{sec:org7c18443}

\begin{center}
\begin{tikzpicture}[node distance=7em]
% nodes
\node(ms1)[ms]{MS1: Mass Filter};
\node(cc)[ms, right of=ms1]{Collision cell};
\node(ms2)[ms, right of=cc]{MS2: Mass Filter};
\node(ion)[ms, below of=ms1,yshift=3em]{Ionization};
\node(lc)[msw, below of=ion,yshift=3em]{Chromatography};
\node(detector)[ms, below of=ms2, yshift=3em]{Detector};
% arrows
\draw[arrow](lc) -- (ion);
\draw[arrow](ion) -- (ms1);
\draw[arrow](ms1) -- (cc);
\draw[arrow](cc) -- (ms2);
\draw[arrow](ms2) -- (detector);
\end{tikzpicture}
\end{center}

\subsubsection{LC-MS/MS sample prep}
\label{sec:org0f90205}
\begin{itemize}
\item 10 \textmu{}L of sample is mixed with 990 \textmu{}L of IS in 0.5 mM perfluoroheptanoic acid and centrifuge to deproteinize
\item 200 \textmu{}L of supernatant is removed
\item 7.5 \textmu{}L is injected onto an octadecylsilyl (C18) stationary phase
\end{itemize}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.7\textwidth]{./aa/figures/outletmethod.png}
\caption{\label{fig:orgf0b8210}LC-MS/MS outletmethod}
\end{figure}

\subsubsection{Ion-Pairing Chromatography}
\label{sec:org57e8055}

\chemnameinit{}
\chemname{\chemfig{CF_3-{(CF_2)_4}-CF_2-[::30](=[::60]O)-[::-60]OH}}{\small perfluoroheptanoic acid}

\ce{AA+ + PFHA- <=> AA+PFHA-}

\subsubsection{LC- MS/MS transitions}
\label{sec:orge84ac9c}
\begin{itemize}
\item ESI in positive mode
\begin{itemize}
\item MRM
\end{itemize}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org390aafc}AA Quantified by LC-MS/MS}
\centering
\begin{tabular}{lll}
phosphoserine & alanine & phenylalanine\\
taurine & citulline & aminoisobutyric\\
pphosphoethanolamine & 2-aminobutyric & \(\gamma\)-aminobutryic\\
aspartate & valine & ethanolamine\\
hydroxyproline & cystine & hydroxylysine\\
threonine & saccharopine & ornithine\\
serine & methionine & lysine\\
asparagine & alloisoleucine & 1-methylhistidine\\
glutamate & cystathionine & histidine\\
glutamine & isoleucine & tryptophan\\
sarcosine & leucine & 3-methylhistidine\\
aminoadipic & arginosuccinic acid & anserine\\
proline & tyrosine & carnosine\\
glycine & \(\beta\)-alanine & arginine\\
 &  & s-sulfocyteine \footnotemark\\
\end{tabular}
\end{table}\footnotetext[3]{\label{org139059e}reported in urine}

\subsubsection{Pros and Cons of LC-MS/MS vs AAA}
\label{sec:org0ad8d6c}
\begin{enumerate}
\item Pros
\label{sec:org7874822}
\begin{itemize}
\item \textasciitilde{} 30 min shorter analysis time
\item \(\uparrow\) analyte specificity
\begin{itemize}
\item based on MRM rather than RT and ninhydrin reactivity
\begin{itemize}
\item gentamycin, acetaminophen, dopamine analogs
\end{itemize}
\item co-eluting substances cannot be separated and distinguished on a
standard IEC chromatogram
\end{itemize}
\item \(\downarrow\) long term reagent expense
\end{itemize}

\item Cons
\label{sec:org14ea3ce}
\begin{itemize}
\item upfront hardware expense
\item manual peak integration
\item lab developed test - not standardized
\item increased LOQ as equipment ages
\end{itemize}
\end{enumerate}

\subsubsection{Pros and Cons of LC-MS/MS vs FIA-MS/MS}
\label{sec:orgeb35b7f}
\begin{enumerate}
\item Pros
\label{sec:orga5f25ad}
\begin{itemize}
\item 43 vs 11 amino acids quantified
\begin{itemize}
\item leu/ile/allo
\end{itemize}
\item isobaric compounds resolved
\begin{itemize}
\item leucine, isoleucine, alloisoleucine
\end{itemize}
\end{itemize}
\item Cons
\label{sec:orgeab710d}
\begin{itemize}
\item too slow for NBS
\item manual peak integration
\end{itemize}
\end{enumerate}

\subsection{Total Homocysteine}
\label{sec:org13ab31a}
\begin{itemize}
\item there are three forms of homocysteine, total homocysteine (tHcy)is
measured (Figure \ref{fig:org6f416ea})
\item tHcy is useful in evaluation of potential CBS deficiency, cobalamin
metabolism defects
\begin{itemize}
\item differentiation of mutase and CblC deficiencies
\end{itemize}
\end{itemize}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{aa/figures/hcy.png}
\caption{\label{fig:org6f416ea}The Three Forms of Homocysteine}
\end{figure}

\subsubsection{Method}
\label{sec:org2ac52f3}
\begin{itemize}
\item enzymatic total homocysteine method (Figure \ref{fig:org8329dc0})
\begin{enumerate}
\item oxidized Hcy that is bound to protein is reduced to free Hcy
\item Hcy then reacts with S-adenosylmethionine (SAM), to form methionine
(Met) and S-adenosyl homocysteine (SAH), catalyzed by a Hcy
S-methyl transferase (HMTase)
\item SAH is assessed by coupled enzyme reactions where SAH is hydrolyzed
into adenosine and homocysteine by SAH hydrolase, and homocysteine
is cycled into the homocysteine conversion reaction to form a
reaction cycle that amplifies the detection signal
\begin{itemize}
\item the formed adenosine is immediately hydrolyzed into inosine and
ammonia (NH\textsubscript{3})
\end{itemize}
\item glutamate dehydrogenase (GLDH) catalyzes the reaction of ammonia
with 2-oxoglutarate and NADH to form NAD\^{}+
\item concentration of Hcy in the sample is \(\propto\) the amount of NADH
converted to NAD\textsuperscript{+} (\(\Delta\) A340nm)
\end{enumerate}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{aa/figures/hcy_enzymatic.png}
\caption{\label{fig:org8329dc0}Enzymatic Total Homocysteine Method}
\end{figure}
\section{Acylcarnitines}
\label{sec:orge0bc26f}
\subsection{Introduction}
\label{sec:org0786daf}
\begin{itemize}
\item carnitine (\(\beta\)-hydroxy-\(\gamma\)-N-trimethylaminobutyric acid) is
an endogenous quaternary ammonium compound synthesized from lysine
and methionine
\end{itemize}

\chemnameinit{}
\chemname{\chemfig{H3C-N^{+}([2]-CH3)([6]-CH3)-CH2-C([2]-H)([6]-OH)-CH_2-C([1]=O)([7]-O^{-})}}{\small carnitine}
\hspace{3em}
\chemname{\chemfig{H3C-N^{+}([2]-CH3)([6]-CH3)-CH2-C([2]-H)([6]-O-C([0]=O)-{\color{red}R})-CH_2-C([1]=O)([7]-O^{-})}}{\small acylcarnitine}

\begin{itemize}
\item carnitine has been described as a "conditionally essential"
nutrient for humans
\item exogenous carnitine is required to maintain plasma carnitine
concentrations in humans of all ages especially
\begin{itemize}
\item infants (premature and full-term)
\item patients on long-term parenteral nutrition
\item children
\end{itemize}
\item primary function is to shuttle long chain fatty acids to the
mitochondrial matrix for \(\beta\)-oxidation
\item acylcarnitines are markers for FAODs and OAs
\end{itemize}


\begin{table}[htbp]
\caption{\label{tab:org2eda477}Acylcarnitine Panel}
\centering
\begin{tabular}{lll}
carbons & acylcarnitine & disorder\\
\hline
C0 & free & CUD\\
C2 & acetyl & CUD\\
C3 & propionyl & PA, MMA, SUCLA2\\
C4 & butyryl & SCAD, IBDH, GA2, EE\\
 & isobutyryl & \\
C5:1 & tiglyl & BKT, MCC, MHBD\\
C5 & isovaleryl & IVA, GA2, EE\\
 & 2-methylbutyryl & SBCAD\\
 & pivalyl & antibiotics\\
C4-OH & 3-hydroxybutyrl & SCHAD, ketosis\\
C6:1 & 3-methylglutaconyl & MGA\\
C6 & hexanoyl & MCAD, MKAT, GA2\\
C5-OH & 3-hydroxyisovaleryl & biotinidase, HMGCL, MCC, MCD, MGA\\
 & 2-methyl-3-hydroxybutyryl & BKT, MHBD\\
C7 & heptanoly & heptanoic acid treatment\\
C8:1 & octenoyl & \\
C8 & octanoyl & MCAD, M/SCHAD, MKAT, GA2\\
C3-DC & malonyl & malonic aciduria\\
C10:2 & decadienoyl & DCR\\
C10:1 & decenoyl & MCAD\\
C10 & decanoyl & MCAD, GA2\\
C4-DC & methylmalonyl & MMA\\
 & succinyl & SUCLA2\\
C5-DC & glutaryl & GA1\\
C10-OH & 3-hydroxy decanoyl & M/SCHAD, MKAT\\
C12:1 & dodecenoyl & \\
C12 & dodecanoyl & GA2\\
C6-DC & adipyl & \\
 & 3-methylglutaryl & HMGCL\\
C12-OH & 3-hydroxy dodecanoyl & LCHAD, TFP\\
C14:2 & tetradecadienoyl & \\
C14:1 & tetradecenoyl & CACT, CPTII, GA2, VLCAD, LCAD, TFP\\
C14 & myristoyl & CACT, CPTII, GA2, VLCAD, LCAD, TFP\\
C8-DC & suberyl & \\
C14-OH & 3-hydroxymyristoyl & LCHAD, TFP\\
C16:1 & palmitoleyl & \\
C16 & palmitoyl & CACT, CPT2, CPT1, VLCAD, LCAD, TFP\\
C10-DC & sebacyl & \\
C16:1-OH & 3-hydroxy hexadecenoyl & antibiotics\\
C16-OH & 3-hydroxy hexadecanoyl & LCHAD, TFP\\
C18:2 & linoleyl & CACT, CPT2, VLCAD, LCAD, TFP\\
C18:1 & oleyl & CACT, CPT2, CPT1, VLCAD, LCAD, TFP\\
C18 & stearoyl & CACT, CPT2, CPT1, VLCAD, LCAD, TFP\\
C18:2-OH & 3-hydroxylinoleyl & \\
C18:1-OH & 3-hydroxyoleyl & LCHAD,TFP\\
C18-OH & 3-hydroxystearoyl & LCHAD,TFP\\
C16-DC & hexadecanedioyl & \\
\end{tabular}
\end{table}

\subsubsection{Disorders Associated with Abnormal Acylcarnitine Profiles}
\label{sec:orgac9978d}
\begin{description}
\item[{BKT}] beta-ketothiolase
\item[{CACT}] carnitine-acylcarnitine translocase
\item[{CPT}] carnitine palmitoyltransferase I and II
\item[{DCR}] 2,4-dienoyl-CoA reductase
\item[{EE}] ethylmalonic encephalopathy
\item[{FIGLU}] formiminoglutamate
\item[{GA1}] glutaric acidemia type I (glutaryl-CoA dehydrogenase)
\item[{GA2}] glutaric acidemia type II (multiple acyl-CoA dehydrogenase)
\item[{HMGCL}] 3-hydroxy 3-methylglutaryl-CoA lyase
\item[{IBDH}] isobutyryl-CoA dehydrogenase
\item[{IVA}] isovaleric acidemia (isovaleryl-CoA dehydrogenase)
\item[{LCHAD}] long-chain 3-hydroxy acyl-CoA dehydrogenase
\item[{MCAD}] medium-chain acyl-CoA dehydrogenase
\item[{MCC}] 3-methylcrotonyl-CoA carboxylase
\item[{MCD}] multiple carboxylase (holocarboxylase)
\item[{MGA}] 3-methylglutaconic aciduria type I (3-methylglutaconyl-CoA hydratase)
\item[{MKAT}] medium-chain 3-ketoacyl-CoA thiolase
\item[{MMA}] methylmalonic acidemias (MUT, CblC)
\item[{MHBD}] 2-methyl 3-hydroxy butyryl-CoA dehydrogenase
\item[{PA}] propionic acidemia (propionyl-CoA carboxylase)
\item[{SBCAD}] short-branched-chain acyl-CoA dehydrongenase
\item[{SCAD}] short-chain acyl-CoA dehydrogenase
\item[{SCHAD}] short-chain 3-hydroxy acyl-CoA dehydrogenase
\item[{SUCLA2}] ATP-dependent-proteolysis-factor-formingsuccinyl-CoA synthetase
\item[{TFP}] mitochondrial trifunctional protein
\item[{VLCAD}] very long-chain acyl-CoA dehydrogenase
\end{description}


\subsection{Diagnostic FIA-MS/MS}
\label{sec:org5364ad2}
\subsubsection{Sample Prep}
\label{sec:org5711171}

\begin{itemize}
\item acylcarnitines in plasma are extracted into methanol
\item reconsituted and esterified as butyl esters with butanol-hydrogen
chloride
\item solvent delivery is via HPLC with no chromatography, called flow
injection analysis
\item 10 \textmu{}L of sample extract is injected into a flowing stream
operating at \textasciitilde{} 0.15 ml/min
\begin{itemize}
\item 3 min integration per sample
\end{itemize}
\end{itemize}

\chemnameinit{}
\definesubmol{x}{-[1,.6]-[7,.6]}
\definesubmol{y}{-[7,.6]-[1,.6]}
\definesubmol{d}{!y!y-[7,.6]{\color{red}COOH}}
\definesubmol{e}{!y!y}
\schemestart
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e)-[1]-[7]([7]=O)([1]-O^{-})}}{\small C5-carnitine}
\+
\chemname{\chemfig{HO!x!x}}{\small n-butanol}
\arrow{-U>[][{\small \ce{H2O}}]}
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e)-[1]-[7]([6]=O)-[1,.6]O!y!y}}{\small C5-carnitine, butyl ester}
\schemestop

\vspace{2em}

\schemestart
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!d)-[1]-[7]([7]=O)([1]-O^{-})}}{\small C6DC-carnitine}
\+
\chemname{\chemfig{HO!x!x}}{\small n-butanol}
\arrow{-U>[][{\small \ce{2H2O}}]}
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e-[7,.6]O!x!x)-[1]-[7]([6]=O)-[1,.6]O!y!y}}{\small C6DC-carnitine, butyl ester}
\schemestop 

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{ac/ac/figures/ionization.png}
\caption{\label{fig:org3fc4700}Rationale for Derivatization}
\end{figure}

\subsubsection{FIA-MS/MS schematic}
\label{sec:org4dbe394}
\begin{center}
\begin{tikzpicture}[node distance=7em]
% nodes
\node(ms1)[ms]{MS1: Mass Filter};
\node(cc)[ms, right of=ms1]{Collision cell};
\node(ms2)[ms, right of=cc]{MS2: Mass Filter};
\node(ion)[ms, below of=ms1,yshift=3em]{Ionization};
\node(lc)[msw, below of=ion,yshift=3em]{Injection};
\node(detector)[ms, below of=ms2, yshift=3em]{Detector};
% arrows
\draw[arrow](lc) -- (ion);
\draw[arrow](ion) -- (ms1);
\draw[arrow](ms1) -- (cc);
\draw[arrow](cc) -- (ms2);
\draw[arrow](ms2) -- (detector);
\end{tikzpicture}
\end{center}

\subsubsection{Precursor Ion Scan}
\label{sec:orgec39340}
\begin{itemize}
\item electrospray ionization in positive mode
\item butylated acylcarnitines fragment to produce a characteristic ion with mass of 85 Da
\item precursor ion scan is used to identify molecules that fragment to form a 85 m/z molecule
\end{itemize}

\chemnameinit{}
\definesubmol{x}{-[1,.6]-[7,.6]}
 \chemname{\chemfig{H_{3}C-N^{+}([2]-CH_3)([6]-CH_{3})-CH_2-C([2]-H)([6]-O-C([0]=O)-{\color{red}R})-CH_2-C([2]=O)-O-CH_2-CH_2-CH_2-CH_3}}{\small acylcarnitine, butyl ester}

\vspace{2.5em}
\chemnameinit{}
 \chemname{\chemfig{H_{3}C-N([1]-CH_3)([7]-CH_3)}}{\small trimethylamine}
\hspace{2em}
\chemname{\chemfig{{\color{red}R}-C([1]=O)([7]-OH)}}{\small carboxylic acid}
\hspace{2em}
 \chemname{\chemfig{H!x!x}}{\small butyl group}
\hspace{2em}
 \chemname{\chemfig{H_{2}C^{+}-HC=CH-C([1]=O)([7]-OH)}}{\small 85 m/z}


\begin{center}
\begin{tikzpicture}
\node[box](ms1)[]{};
\node[label](ms1u)[above=of ms1,yshift=-3em]{MS1};
\node[label](ms1l)[below=of ms1,yshift=3em]{scanning};
\node[box](cc)[right= of ms1]{};
\node[label](ccu)[above=of cc,yshift=-3em]{Collision cell};
\node[label](ccl)[below=of cc,yshift=3em]{fragmentation};
\node[box](ms2)[right= of cc]{};
\node[label](ms2u)[above=of ms2,yshift=-3em]{MS2};
\node[label](ms2l)[below=of ms2,yshift=3em]{85 m/z};
\draw[->](ms1) -- (cc);
\draw[->](cc) -- (ms2);

%ms1
\draw [gray,->, decorate,decoration=snake] (-.8,0.5) -- (.8,0.5);
\draw [gray,->, decorate,decoration=snake] (-.8,0.25) -- (.8,0.25);
\draw [blue, ->, decorate,decoration=snake] (-.8, 0) -- (.8,0);
\draw [gray,->, decorate,decoration=snake] (-.8,-0.25) -- (.8,-0.25);
\draw [gray,->, decorate,decoration=snake] (-.8,-0.5) -- (.8,-0.5);

%cc
\draw [blue,->,decorate,decoration=snake] (2.1, 0) -- (2.4,0);
\fill (2.6,0) circle (0.1); 
\draw [gray,->,decorate,decoration=snake] (2.8, 0) -- (3.8,0.5);
\draw [red, ->,decorate,decoration=snake] (2.8, 0) -- (3.8,0);
\draw [gray,->,decorate,decoration=snake] (2.8, 0) -- (3.8,-0.5);

%ms2
\draw [red,->,decorate,decoration=snake] (5.1, 0) -- (6.8,0);
\end{tikzpicture}
\end{center}


\subsubsection{Overestimation of Free Carnitine}
\label{sec:org02e9d83}
\begin{itemize}
\item butylated acylcarnitines are converted to butylated carnitine in
n-butanol-3M HCl at 65\degree{}C \footnote{Johnson, D. W. (1999). Inaccurate measurement of free
carnitine by the electrospray tandem mass spectrometry screening
method for blood spots. Journal of Inherited Metabolic Disease, 22(2),
201–202.\label{org69eec90}}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org7523535}Overestimation of Free Carnitine}
\centering
\begin{tabular}{lr}
Acyl Carnitine & Half-life (min)\\
\hline
C2 & 31\\
C10 & 125\\
C18 & 172\\
\end{tabular}
\end{table}

\begin{itemize}
\item 65\degree{}C for 15 min.
\item NSO uses 60\degree{}C for 20 minutes.
\item IMD uses 55\degree{}C for 20 minutes.

\item in a sample with low free carnitine and high acetylcarnitine
\begin{itemize}
\item 30\% of the acetylcarnitine and smaller amounts of higher
molecular mass acylcarnitines are converted to carnitine
\item a low carnitine sample could appear to be normal
\end{itemize}
\item "the free carnitine results obtained by this screening method on
blood spots with high levels of acylcarnitines should therefore be
used with caution" \textsuperscript{\ref{org69eec90}}
\end{itemize}


\subsection{Free and Total Carnitine}
\label{sec:org60b6d15}
\begin{itemize}
\item acylcarnitine quantitation must be done using a non-derivatized FIA
or LC MS/MS method
\begin{itemize}
\item avoids hydrolysis of acylcarnitines to free carnitine (C0)
\end{itemize}
\end{itemize}

\[
\frac{free_{carnitine}}{total_{carnitine}} = \frac{C_0}{\sum_{0}^{18} C_n}
\]

\begin{itemize}
\item \(\Downarrow\) in CUD \textless{} 5-10\% of normal
\end{itemize}


\subsubsection{Fractional Tubular Re-absorption of Carnitine}
\label{sec:orgdc0cba7}

\begin{equation*}
FTR_{carnitine}\% = \left( 1 -  \frac{carnitine_{urine} \cdot creatinine_{plasma}}{carnitine_{plasma} \cdot creatinine_{urine}}\right) \cdot 100
\end{equation*}

\begin{itemize}
\item \(\Downarrow\) in CUD normally \textgreater{} 98\%
\end{itemize}
\section{AAAC}
\label{sec:org1e593f2}
\begin{itemize}
\item high-throughput method used in newborn screening
\end{itemize}
\subsection{Sample}
\label{sec:org2704a75}
\subsubsection{Dried Blood Spot}
\label{sec:org9036900}
\begin{itemize}
\item collected from free flowing blood spotted onto filter paper
\item used for NBS and monitoring
\item each DBS is assume to contain 3.2 \textmu{}L of blood
\item the quantity of blood present in the paper varies by
\begin{itemize}
\item hematocrit
\item degree of saturation
\item the cotton fiber paper
\item the environment  when applying blood (humidity and temperature)
\end{itemize}
\item because of these numerous factors, a dried blood spot is a highly
imprecise specimen compared with liquids such as urine, blood, and plasma
\end{itemize}

\subsubsection{Sample Preparation}
\label{sec:orgdc38b52}
\begin{itemize}
\item amino acids and acylcarnitines in the DBS eluate are esterified as butyl esters with butanol-hydrogen chloride
\item solvent delivery is via HPLC with no chromatography, called flow injection analysis
\item 10 \textmu{}L of sample extract is injected into a flowing stream operating at \textasciitilde{} 0.15 ml/min

\item typical injection rates between samples are 2 min, giving a
potential 400-600 sample capacity per instrument per day
\begin{itemize}
\item volume is typically 200-400 specimens per instrument per day
\item maintenance issues, repeat specimen analysis
\end{itemize}
\end{itemize}

\chemnameinit{}
\schemestart
\chemname{\chemfig{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]OH}}{\small \textalpha{}-amino acid}
\+
\chemname{\chemfig{HO-[::30]-[::-60]-[::60]-[::-60]}}{\small n-butanol}
\arrow{-U>[][{\small \ce{H2O}}]}
\chemname{\chemfig{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\small AA butyl ester}
\schemestop

\chemnameinit{}
\definesubmol{x}{-[1,.6]-[7,.6]}
\definesubmol{y}{-[7,.6]-[1,.6]}
\definesubmol{d}{!y!y-[7,.6]{\color{red}COOH}}
\definesubmol{e}{!y!y}
\schemestart
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e)-[1]-[7]([7]=O)([1]-O^{-})}}{\small C5-carnitine}
\+
\chemname{\chemfig{HO!x!x}}{\small n-butanol}
\arrow{-U>[][{\small \ce{H2O}}]}
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e)-[1]-[7]([6]=O)-[1,.6]O!y!y}}{\small C5-carnitine, butyl ester}
\schemestop

\vspace{2em}

\schemestart
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!d)-[1]-[7]([7]=O)([1]-O^{-})}}{\small C6DC-carnitine}
\+
\chemname{\chemfig{HO!x!x}}{\small n-butanol}
\arrow{-U>[][{\small \ce{2H2O}}]}
\chemname{\chemfig{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e-[7,.6]O!x!x)-[1]-[7]([6]=O)-[1,.6]O!y!y}}{\small C6DC-carnitine, butyl ester}
\schemestop 

\subsection{FIA-MS/MS}
\label{sec:org28ea37f}

\subsubsection{FIA-MS/MS Schematic}
\label{sec:orga30f75a}
\begin{center}
\begin{tikzpicture}[node distance=7em]
% nodes
\node(ms1)[ms]{MS1: Mass Filter};
\node(cc)[ms, right of=ms1]{Collision cell};
\node(ms2)[ms, right of=cc]{MS2: Mass Filter};
\node(ion)[ms, below of=ms1,yshift=3em]{Ionization};
\node(lc)[msw, below of=ion,yshift=3em]{Injection};
\node(detector)[ms, below of=ms2, yshift=3em]{Detector};
% arrows
\draw[arrow](lc) -- (ion);
\draw[arrow](ion) -- (ms1);
\draw[arrow](ms1) -- (cc);
\draw[arrow](cc) -- (ms2);
\draw[arrow](ms2) -- (detector);
\end{tikzpicture}
\end{center}

\subsubsection{Amino Acid NL Scan}
\label{sec:orgcd0edc4}
\begin{itemize}
\item electrospray ionization in positive mode
\item \(\alpha\)-amino acids fragment to produce the neutral butyl formate molecule (102 Da)
\item neutral loss scan is used to identify parent molecules with a NL of 102 Da
\item MRM is used to detected amino acids with basic functional groups
\begin{itemize}
\item arginine, ornithine and citrulline
\end{itemize}
\end{itemize}

\chemnameinit{}
\schemestart
\chemname{\chemfig{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\tiny AA butyl ester}
\arrow{->[{\tiny fragmentation}]}
\chemnameinit{}
\chemname{\chemfig{{\color{red}R}-[::60]=NH_2^{+}}}{\tiny fragment}
\+
\chemname{\chemfig{H-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\tiny butyl formate (102 Da)}
\schemestop
\subsubsection{Amino Acid MRM}
\label{sec:orgc46daa9}
\begin{itemize}
\item citrulline contains a labile amino group that fragments together with butyl formate
\item CID results in net neutral fragmentation of butyl formate (102 Da) plus \ce{NH3} (17 Da)
\item \href{https://en.wikipedia.org/wiki/Selected\_reaction\_monitoring}{SRM} Citrulline-Bu 232.15 Da \(\to\) 113 Da , loss of 119 Da
\end{itemize}

\chemnameinit{}
\schemestart
\chemname{\chemfig{H_2N-[::30,,2,](=[::60]O)-[::-60]NH-[::60]-[::-60]-[::60]-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]OH}}{\small citrulline 175 Da}
\+
\chemname{\chemfig{HO-[::30]-[::-60]-[::60]-[::-60]}}{\small n-butanol 74 Da}
\arrow{-U>[][{\small \ce{H2O}}]}
\chemname{\chemfig{H_2N-[::30,,2,](=[::60]O)-[::-60]NH-[::60]-[::-60]-[::60]-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\small 232 Da}
\schemestop

\schemestart
\chemname{\chemfig{H_2N-[::60]-[::-60]-[::60]-[::-60]-[::60]N=O=C}}{\small 113 Da}
\+
\chemname{\chemfig{H-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\small 102 Da}
\+
\chemname{\chemfig{NH_3}}{\small 17 Da}
\schemestop

\begin{table}[htbp]
\caption{\label{tab:org061c8a1}Quantified Amino Acids}
\centering
\begin{tabular}{ll}
glycine & tyrosine\\
alanine & ornithine\\
valine & citruline\\
leucine & arginine\\
methionine & \color{blue}succinylacetone\\
phenylalanine & \\
\end{tabular}
\end{table}

\subsubsection{Acylcarnitine Precursor Ion Scan}
\label{sec:org4c4425d}
\begin{itemize}
\item electrospray ionization in positive mode
\item butylated acylcarnitines fragment to produce a characteristic ion with mass of 85 Da
\item precursor ion scan is used to identify molecules that fragment to form a 85 m/z molecule
\end{itemize}


\chemnameinit{}
\definesubmol{x}{-[1,.6]-[7,.6]}
 \chemname{\chemfig{H_{3}C-N^{+}([2]-CH_3)([6]-CH_{3})-CH_2-C([2]-H)([6]-O-C([0]=O)-{\color{red}R})-CH_2-C([2]=O)-O-CH_2-CH_2-CH_2-CH_3}}{\small acylcarnitine, butyl ester}

\vspace{2.5em}

\chemnameinit{}
 \chemname{\chemfig{H_{3}C-N([1]-CH_3)([7]-CH_3)}}{\small trimethylamine}
\hspace{2em}
\chemname{\chemfig{{\color{red}R}-C([1]=O)([7]-OH)}}{\small carboxylic acid}
\hspace{2em}
 \chemname{\chemfig{H!x!x}}{\small butyl group}
\hspace{2em}
 \chemname{\chemfig{H_{2}C^{+}-HC=CH-C([1]=O)([7]-OH)}}{\small 85 m/z}

\begin{center}
\begin{tikzpicture}
\node[box](ms1)[]{};
\node[label](ms1u)[above=of ms1,yshift=-3em]{MS1};
\node[label](ms1l)[below=of ms1,yshift=3em]{scanning};
\node[box](cc)[right= of ms1]{};
\node[label](ccu)[above=of cc,yshift=-3em]{Collision cell};
\node[label](ccl)[below=of cc,yshift=3em]{fragmentation};
\node[box](ms2)[right= of cc]{};
\node[label](ms2u)[above=of ms2,yshift=-3em]{MS2};
\node[label](ms2l)[below=of ms2,yshift=3em]{85 m/z};
\draw[->](ms1) -- (cc);
\draw[->](cc) -- (ms2);

%ms1
\draw [gray,->, decorate,decoration=snake] (-.8,0.5) -- (.8,0.5);
\draw [gray,->, decorate,decoration=snake] (-.8,0.25) -- (.8,0.25);
\draw [blue, ->,decorate,decoration=snake] (-.8, 0) -- (.8,0);
\draw [gray,->, decorate,decoration=snake] (-.8,-0.25) -- (.8,-0.25);
\draw [gray,->,decorate,decoration=snake] (-.8,-0.5) -- (.8,-0.5);

%cc
\draw [blue,->,decorate,decoration=snake] (2.1, 0) -- (2.4,0);
\fill (2.6,0) circle (0.1); 
\draw [gray,->,decorate,decoration=snake] (2.8, 0) -- (3.8,0.5);
\draw [red, ->,decorate,decoration=snake] (2.8, 0) -- (3.8,0);
\draw [gray,->,decorate,decoration=snake] (2.8, 0) -- (3.8,-0.5);

%ms2
\draw [red,->,decorate,decoration=snake] (5.1, 0) -- (6.8,0);
\end{tikzpicture}
\end{center}

\subsubsection{Acylcarnitine MRM}
\label{sec:orgc41b63e}
\begin{itemize}
\item C0-Bu 218.1 Da \(\to\) 103 Da transition is optimal
\item all others benefit from the added sensitivity of MRM mode as
compared to parent ion scan
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org20a103e}MRM is used to detected selected acylcarnitines}
\centering
\begin{tabular}{ll}
Compound & Reaction\\
\hline
C0 & 218.10 > 103.00\\
C0 IS & 227.10 > 103.00\\
C2 & 260.20 > 85.00\\
C2 IS & 263.20 > 85.00\\
C3 & 274.20 > 85.00\\
C3 IS & 277.20 > 85.00\\
C3DC & 360.30 > 85.00\\
C4DC & 374.30 > 85.00\\
C5DC & 388.35 > 85.00\\
C5DC IS & 391.35 > 85.00\\
C6DC & 402.45 > 85.00\\
C8DC & 430.45 > 85.00\\
\end{tabular}
\end{table}

\begin{table}[htbp]
\caption{\label{tab:org2bbc53d}Quantified Acylcarnitines}
\centering
\begin{tabular}{lll}
C0 & C8 & C16\\
C2 & C8:1 & C16:1\\
C3 & C10 & C16:1-OH\\
C3DC & C10:1 & C16-OH\\
C4 & C12 & C18\\
C4DC & C12:1 & C18:1\\
C5 & C14 & C18:1-OH\\
C5:1 & C14:1 & C18:2\\
C5DC & C14:2 & C18-OH\\
C5-OH & C14-OH & \\
C6 &  & \\
C6DC &  & \\
\end{tabular}
\end{table}

\subsubsection{Pros and Cons of FIA-MS/MS using DBS}
\label{sec:org38bbb7e}
\begin{itemize}
\item as compared to AAA and LC-MS/MS
\end{itemize}
\begin{enumerate}
\item Pros
\label{sec:org69aed82}
\begin{itemize}
\item \textasciitilde{} 2 min analysis time
\item analyte specificity
\item ACs and AAs quantified simultaneously
\end{itemize}

\item Cons
\label{sec:org4aa3621}
\begin{itemize}
\item variability in DBS sample as described above
\item iso-baric compounds
\begin{itemize}
\item leucine, Isoleucine, Alloisoleucine
\item C5DC and C10-OH
\end{itemize}
\item overestimation of CO due to hydrolysis
\item fewer AA quantified
\begin{itemize}
\item homocystine (free)
\item glutamine
\end{itemize}
\end{itemize}
\end{enumerate}
\section{Organic Acids}
\label{sec:orga6a079f}
\subsection{Introduction}
\label{sec:org7af8cd2}
\begin{itemize}
\item organic acids are water soluble compounds containing \(\ge\) one
carboxyl group(s) and nonamino functional groups
\end{itemize}

\begin{center}
\chemnameinit{}
\chemname{\chemfig{X-C(-[2]X)(-[6]X)-C(-[2]X)(-[6]X)-C(-[7]OH)=[1]O}}{organic acid}
\end{center}


\begin{table}[htbp]
\caption{\label{tab:orge72ae44}Organic Acid Functional Groups}
\centering
\begin{tabular}{ll}
Functional group & Formula\\
\hline
hydrogen & -H\\
keto & .= O\\
hydroxyl & -OH\\
carboxyl & -COOH\\
side chain & -(CH\(_2\))\(_n\)\\
\end{tabular}
\end{table}



\begin{table}[htbp]
\caption{\label{tab:org2dbb35f}Organic Acid Side Chains}
\centering
\begin{tabular}{ll}
side chain & structure\\
\hline
methyl & \chemfig{CH_3-}\\
ethyl & \chemfig{CH_3-CH_2-}\\
propyl & \chemfig{CH_3-CH_2-CH_2-}\\
butyl & \chemfig{CH_3-CH_2-CH_2-CH_2-}\\
\end{tabular}
\end{table}

\begin{table}[htbp]
\caption{\label{tab:orgc231075}Organic Acid Nomenclature}
\centering
\begin{tabular}{lll}
length & monocarboxylic acid & dicarboxylic acid\\
\hline
C2 & acetic & oxalic\\
C3 & propionic & malonic\\
C4 & butyric & succinic\\
 & isobutyric & \\
C5 & valeric & glutaric\\
 & isovaleric & \\
 & 2-methylbutyric & \\
C6 & hexanoic (caprioc) & adipic\\
C7 & heptanoic (enanthic) & pimelic\\
C8 & octanoic (caprylic) & suberic\\
C9 & nonanoic (pelargonic) & azelaic\\
C10 & decanoic (capric) & sebacic\\
\end{tabular}
\end{table}

\begin{enumerate}
\item Acylglycines
\label{sec:org086ab96}
\begin{itemize}
\item acylglycines are also detected in UOA analysis
\begin{itemize}
\item conjugation of acyl-CoA species to glycine
\item catalysed by glycine N-acylase
\end{itemize}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:orgf9ad700}Acylglycines}
\centering
\begin{tabular}{llll}
carbons & acylglycine & precusor & disorder\\
\hline
C3 & propionylglycine & leu met & propionic acidemia\\
 &  & thr val & methylmalonic acidemias\\
C4 & butyrylglycine & butyryl-CoA & SCAD\\
 &  &  & GAII\\
C4 & isobutyrylglycine & val & isobutyryl-CoA dehydrogenase\\
 &  &  & GAII\\
 &  &  & ethylmalonic encephalopathy\\
C5:1 & tiglylglycine & ile & propionic acidemia\\
 &  &  & methylmalonic acidemias\\
 &  &  & ketothiolase deficiency\\
C5:1 & 3-methyl crotonylglycine & leu & 3-methylcrotonyl-CoA carboxylase\\
 &  &  & multiple carboxylase deficiency\\
C5 & isovalerylglycine & leu & isovaleric acidemia\\
C5 & 2-methylbutyrylglycine & ile & 2-methylbutyryl-CoA dehydrogenase\\
 &  &  & GAII\\
 &  &  & ethylmalonic encephalopathy\\
C6 & hexanoylglycine & hexanoyl-CoA & MCAD\\
 &  &  & GAII\\
C8 & octanoylglycine & octanoyl-CoA & MCAD\\
C8 & suberylglycine & suberyl-CoA & MCAD\\
 &  &  & GAII\\
C9 & phenylpropionylglycine & phe & MCAD\\
C9:1 & trans-cinnamoylglycine & phe & no known defect\\
\end{tabular}
\end{table}
\end{enumerate}

\subsubsection{Sources of Organic Acids}
\label{sec:orge408c20}
\begin{enumerate}
\item Endogenous
\label{sec:org953dd26}
\begin{itemize}
\item originate from the intermediate metabolism of all major groups of
organic cellular components
\begin{itemize}
\item amino acids
\item lipids
\item nucleotides
\item carbohydrates
\item nucleic acids
\item steroids
\end{itemize}
\end{itemize}

\item Exogenous
\label{sec:org60203d2}
\begin{itemize}
\item food
\item environment
\item medications
\end{itemize}
\end{enumerate}

\subsubsection{Urine organic acids detected in health}
\label{sec:org641ea64}
\begin{itemize}
\item tricarboxylic acid cycle acids
\begin{itemize}
\item citric acid
\end{itemize}
\item hydroxyaliphatic acids
\begin{itemize}
\item 3-hydroxybutyric acid
\end{itemize}
\item aliphatic keto acids
\begin{itemize}
\item pyruvic acid
\end{itemize}
\item aliphatic acids
\begin{itemize}
\item oxalic acid
\end{itemize}
\item aldonic and deoxyaldonic acids (sugar acids)
\item aromatic acids
\begin{itemize}
\item hippuric acid
\end{itemize}
\end{itemize}

\subsubsection{Abnormal Urine Organic acids profiles}
\label{sec:org114bdbf}
\begin{itemize}
\item elevated concentration of normal metabolites
\begin{itemize}
\item fumaric acid in fumarase deficiency
\item adipic, suberic, and sebacic acids in MCADD
\item ketones in fasting
\begin{itemize}
\item 3-hydroxybutyric acid
\item acetoacetic acid
\end{itemize}
\end{itemize}

\item pathological metabolites
\begin{itemize}
\item succinylacetone, methylcitric acid
\end{itemize}

\item food, medications, environment
\begin{itemize}
\item ethosuximide
\item adipic acid
\item cresol
\item 2-furaldehyde
\end{itemize}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org9403c87}Disorders of Organic Acid Metabolism}
\centering
\begin{tabular}{ll}
Disorder & Defective Enzyme\\
\hline
2-Keto adipic aciduria & 2-Keto adipic dehydrogenase\\
2-Keto glutaric aciduria & 2-Keto glutaric dehydrogenase\\
2-Ketothiolase deficiency & 2-Methylacetoacetyl-CoA thiolase\\
2-Methyl 3-hydroxy butyric aciduria & 2-Methyl 3-hydroxy butyryl-CoA dehydrogenase\\
2-Methylbutyrylglycinuria & 2-Methylbutyryl-CoA dehydrogenase\\
3-Hydroxy 3-methyl glutaric aciduria & 3-Hydroxy 3-methyl glutaryl-CoA lyase\\
3-Methylcrotonylglycinuria & 3-Methylcrotonyl-CoA carboxylase\\
3-Methylglutaconic aciduria & 3-Methyl glutaconyl-CoA hydratase\\
4-Hydroxy butyric aciduria & Succinic semialdehyde dehydrogenase\\
Alkaptonuria & Homogentisic dioxygenase\\
Canavan disease & N-Aspartoacylase\\
D-2-Hydroxy glutaric aciduria & D-2-Hydroxyglutaric dehydrogenase\\
Ethylmalonic encephalopathy & Unknown (ETHE1 gene)\\
Fumaric aciduria & Fumarase\\
Glutaric aciduria type I & Glutaryl-CoA dehydrogenase\\
Glyceroluria (X-linked) & Glycerol kinase\\
Hawkinsinuria (autosomal dominant) & 4-Hydroxy phenylpyruvic acid dioxygenase\\
Hyperoxaluria type I & Alanine:glyoxylate aminotransferase\\
Hyperoxaluria type II & D-Glyceric dehydrogenase\\
Isobutyrylglycinuria & Isobutyryl-CoA dehydrogenase\\
Isovaleric aciduria & Isovaleryl-CoA dehydrogenase\\
L-2-Hydroxy glutaric aciduria & L-2-Hydroxy dehydrogenase (Duranin)\\
Malonic aciduria & Malonyl-CoA decarboxylase\\
Methylmalonic acidurias & Methylmalonyl-CoA mutase, other defects\\
Mevalonic aciduria & Mevalonate kinase\\
Multiple carboxylase deficiency & Holocarboxylase synthase\\
Propionic aciduria & Propionyl-CoA carboxylase\\
Pyroglutamic aciduria & Glutathione synthase\\
\end{tabular}
\end{table}


\subsection{Urine Organic Acids by GC-MS}
\label{sec:orgc85042c}
\subsubsection{Oximation}
\label{sec:org9b52766}
\begin{itemize}
\item not always done, sometime a reflex when 2-keto acids present
\begin{itemize}
\item lactic acidemia, ketonuria
\end{itemize}
\item oximated with 10\% hydroxylamine-HCL
\begin{itemize}
\item avoids multiple TMS species due to keto-enol tautomerism
\end{itemize}
\end{itemize}

\schemestart
\chemname{\chemfig{R=[1](-[2]OH)-[7]R}}{\tiny enol}
\arrow{<=>}
\chemname{\chemfig{R-[1](=[2]O)-[7]R}}{\tiny ketone}
\+
\chemname{\chemfig{N(<:[::-160]H)(<[::-120]H)-O-[1]H}}{\tiny hydroxylamine}
\arrow{->}
\chemname{\chemfig{R-[1](=[2]N-[1]OH)-[7]R}}{\tiny ketoxime}
\schemestop
\subsubsection{BSTFA Derivatisation}
\label{sec:org660a43d}
\begin{itemize}
\item acidified and extracted twice with ethyl ether
\item derivatised with BSTFA (N,O-bis(trimethylsilyl)trifluoroacetamide) \footnote{Stalling DL, Gehrke CW, Zumwalt RW. A new silylation
reagent for amino acids bis(trimethylsilyl)trifluoroacetamide
(BSTFA). Biochemical and Biophysical Research Communications. 1968 May
23;31(4):616-22.}
\begin{itemize}
\item forms organic acid TMS esters
\end{itemize}
\end{itemize}

\schemestart
\chemname{\chemfig{F{_3}C-C(-[1]OTMS)=[7]NTMS}}{\tiny BSTFA}
\+
\chemname{\chemfig{R-C(=[1]O)-[7]OH}}{\tiny carboxylic acid}
\arrow{->}
\chemname{\chemfig{R-C(=[1]O)-[7]OTMS}}{\tiny TMS ester}
\+
\chemname{\chemfig{F{_3}C-C(=[1]O)-[7]NTMS}}{\tiny TMS amide}
\schemestop

\subsubsection{GC-MS}
\label{sec:org08a6d5d}
\begin{itemize}
\item detected by electron impact mass spectrometry performed in the scan mode
\item mass range between m/z 50 and 550
\item identification is achieved by comparison to published spectra of
bona fide compounds, or spectra generated by in-house analysis of
pure standard compounds
\item quantification is by comparison to calibration of pure standard
compounds in ratio to an internal standard
\end{itemize}
\section{Mitochondria}
\label{sec:org0fddfbd}
\begin{figure}[htbp]
\centering
\includegraphics[width=\textwidth]{./mito/figures/etc.pdf}
\caption{\label{fig:org36be31b}Flow of Electrons in the ETC}
\end{figure}

\subsection{Citrate Synthase}
\label{sec:org3fbc37c}
\subsubsection{Purpose}
\label{sec:orgf637489}
\begin{itemize}
\item citrate synthase (CS) is the initial enzyme of the TCA cycle
\item is an exclusive marker of the mitochondrial matrix
\end{itemize}
\subsubsection{Procedure}
\label{sec:orgfb00ab0}
\begin{itemize}
\item CS catalyzes the reaction between acetyl-CoA and oxaloacetic acid to
form citric acid and CoA-SH
\end{itemize}

\ce{acetyl-CoA + oxaloacetate ->[CS] Citrate + CoA-SH + H+ + H2O} 

\begin{itemize}
\item CoA-SH reacts with the 5,5′-Dithiobis-(2-nitrobenzoic acid)(DTNB) in
the reaction mixture to form 5-thio-2-nitrobenzoic acid (TNB)
\end{itemize}

\ce{CoA-SH + DTNB -> TNB + CoA-S-S-TNB}

\begin{itemize}
\item absorbance of the yellow product (TNB) is measured at 412 nm
\end{itemize}

\subsection{CI \& III assay}
\label{sec:org90002c5}
\begin{figure}[htbp]
\centering
\includegraphics[width=\textwidth]{./mito/figures/c1c3.pdf}
\caption{\label{fig:orgae8f513}CI \& III assay}
\end{figure}

\subsubsection{Purpose}
\label{sec:org479f2d9}
\begin{itemize}
\item determine the rate of cytochrome c reduction in mitochondria as a
result of electron transfer from NADH to cytochrome c (mitochondrial
complex I+III) activity
\item Complex I transfers electrons to ubiquinone (CoQ) through a
long series of redox groups
\item Complex III catalyzes electron transfer between ubiquinol(CoQH\textsubscript{2})
and cytochrome c and also translocates protons across the MIM
\end{itemize}

\subsubsection{Principle}
\label{sec:org1c0fc17}
\begin{itemize}
\item reduced cytochrome c absorbs light at 550 nm
\item increase of the absorption at 550 nm corresponding to the increased
formation of reduced cytochrome c by electrons derived from NADH,
which is rotenone sensitive
\item azide is added to inhibit CIV so there is no re-oxidation of reduced cytochrome c
\item rotenone is added to the reference cuvette to inhibit Complex I
\begin{description}
\item[{assay cuvette}] oxidized cyt c \& azide
\item[{reference cuvette}] oxidized cyt c \& azide \& rotenone
\end{description}
\end{itemize}

{\small\ce{4Fe3+ cytochrome c + NADH + 2H2O ->[CI + CIII] 4Fe2+ cytochrome c + NAD+ + 4H + O2}}
\ce{oxidized cyt c -> reduced cyt c} 

\begin{itemize}
\item spectrophotometer subtracts the activity seen in the reference cell
from the activity seen in the assay cell, the progress curve you see
on the computer screen reflects activity of cytochrome c reduction
by electrons passing ONLY through complex I
\end{itemize}

\subsection{CI assay}
\label{sec:org900e2fd}
\begin{figure}[htbp]
\centering
\includegraphics[width=.7\textwidth]{./mito/figures/c1.pdf}
\caption{\label{fig:org5e87c23}CI assay}
\end{figure}

\subsubsection{Purpose}
\label{sec:orgdf98fac}
\begin{itemize}
\item determining the rate of NADH oxidation in mitochondria as a result
of electron transfer from NADH to ubiquinone
\end{itemize}
\subsubsection{Principle}
\label{sec:org583378f}
\begin{itemize}
\item NADH absorbs light at 340 nm
\item the method follows the decrease of the absorption at 340 nm
corresponding to the decreased concentration of NADH, which has been
oxidized to NAD during the passage of electrons to ubiquinone
\item assay is rotenone sensitive
\item rotenone in the reference cuvette will specifically inhibit CI
therefore any oxidation of NADH from this cell does not include the
contribution of CI
\begin{description}
\item[{assay cuvette}] ubiquinone \&  antimycin A
\item[{reference cuvette}] ubiquinone \&  antimycin A \& rotenone
\end{description}
\end{itemize}

\ce{CoQ + NADH ->[CI] CoQH2 + NAD+} 

\begin{itemize}
\item spectrophotometer subtracts the activity seen in the reference cell
from the activity seen in the assay cell, the progress curve seen on
the computer screen reflects NADH oxidation ONLY through CI
\end{itemize}

\subsection{CII assay}
\label{sec:org7e88d5b}
\begin{figure}[htbp]
\centering
\includegraphics[width=.7\textwidth]{./mito/figures/c2.pdf}
\caption{\label{fig:orgbfd60d8}CII Assay}
\end{figure}

\subsubsection{Purpose}
\label{sec:orgf3c58ab}
\begin{itemize}
\item Complex II activity
\end{itemize}
\subsubsection{Principle}
\label{sec:org42f5692}
\begin{itemize}
\item secondary reduction of the dye 2,6-dichlorophenolindophenol (DCPIP)
by the CoQH\textsubscript{2} at 600nm
\item DCPIP assays are very prone to interference from NADPH
diaphorases
\item caution is recommended in interpreting results from non-muscle
tissue, rich in diaphorase
\begin{description}
\item[{assay cuvette}] succinate, ubiquinone \& DCPIP
\item[{reference cuvette}] ubiquinone \& DCPIP
\end{description}
\end{itemize}

\ce{CoQH2 + DCPIP_ox -> CoQ + DCPIP_red} 

\subsection{CII \& III assay}
\label{sec:org79d3131}
\begin{figure}[htbp]
\centering
\includegraphics[width=\textwidth]{./mito/figures/c2c3.pdf}
\caption{\label{fig:org8937176}CII \& CIII}
\end{figure}

\subsubsection{Purpose}
\label{sec:org0b844a9}
\begin{itemize}
\item measure rate of CII and III activity in mitochondria 
\begin{itemize}
\item complexes II and III are sometimes called succinate cytochrome c reductase (SCR)
\end{itemize}
\item Complex II performs a key step in the citric acid cycle in which
succinate is dehydrogenated to ubiquinone in the mitochondrial inner
membrane
\item Complex II is localized to the matrix side of the mitochondrial
inner membrane and it is the only respiratory chain enzyme of which
all 4 subunits are coded by the nuclear DNA
\item Complex III catalyzes electron transfer between ubiquinol and
cytochrome c and also translocates protons across the mitochondrial
inner membrane
\end{itemize}

\subsubsection{Principle}
\label{sec:orgb771498}
\begin{itemize}
\item reduced cytochrome c absorbs light at 550 nm
\item the increase of the absorption at 550 nm corresponds to the
increased formation of reduced cytochrome c by electrons derived
from succinate
\begin{description}
\item[{assay cuvette}] sample, oxidized cyt c, azide \& succinate
\item[{reference cuvette}] oxidized cyt c, azide \& succinate
\end{description}
\end{itemize}

{\small\ce{4Fe3+ cytochrome c + succinate + 2H2O ->[CII + CIII] 4Fe2+ cytochrome c + fumarate + 4H + O2}}
\\
\ce{oxidized cyt c -> reduced cyt c} 

\subsection{CIV assay}
\label{sec:org65d2ed0}
\begin{figure}[htbp]
\centering
\includegraphics[width=.7\textwidth]{./mito/figures/c4.pdf}
\caption{\label{fig:org82dc27c}CIV Assay}
\end{figure}

\subsubsection{Purpose}
\label{sec:orgb8d1765}
\begin{itemize}
\item determine the rate of cytochrome C oxidation in mitochondria as a
result of cytochrome C oxidase (mitochondrial complex IV, COX)
activity
\item COX is a multisubunit assembly in the inner mitochondrial membrane
responsible for the terminal event in electron transport in which
molecular oxygen is reduced
\item various phenotypic forms of COX deficiency have been recognized, the
major varieties involving the degeneration of the brain stem and
basal ganglia (Leigh syndrome) and lactic acidemia with or without
cardiomyopathy
\end{itemize}

\subsubsection{Principle}
\label{sec:orgc3bf54c}
\begin{itemize}
\item reduced cytochrome c absorbs light at 550nm
\item methods follows the decrease in absorbance of reduced cytochrome c
at 550 nm
\begin{description}
\item[{assay cuvette}] sample \& reduced cyt c
\item[{reference cuvette}] reduced cyt c
\end{description}
\end{itemize}

{\small\ce{4Fe3+ cytochrome c + 2H2O->[CIV] 4Fe2+ cytochrome c 4H + O2}}
\ce{reduced cyt c -> oxidized cyt c}

\subsection{CV assay}
\label{sec:org5d331f8}
\subsubsection{Purpose}
\label{sec:orga9980d6}
\begin{itemize}
\item determine the activity of respiratory chain CV in isolated
muscle and fibroblast mitochondria
\end{itemize}

\subsubsection{Principle}
\label{sec:orgd44d3f3}
\begin{itemize}
\item ATP hydrolysis via the ATPase activity of CV generates ADP which is
reconverted to ATP by the action of PK, thus maintaining a constant
concentration of ATP and a low steady state concentration of ADP
\item pyruvate production from PEP and ADP, catalysed by PK, is monitored
as a rate of oxidation of NADH by coupling to LDH
\item ATPase is oligomycyn sensitive
\begin{description}
\item[{assay cuvette}] LDH, PK, PEP \& rotenone
\item[{reference cuvette}] LDH, PK, PEP, rotenone \& oligomycin
\end{description}
\end{itemize}

\ce{ATP <->[CV] ADP}
\ce{PEP + ADP ->[PK] pyruvate}
\ce{pyruvate + NADH ->[LDH] lactate + NAD+}
\section{VLCFA}
\label{sec:org2b6b948}
\begin{itemize}
\item investigation of peroxisomal disorders (Table \ref{tab:org943cb5b})
\item C22, C24 and C26 fatty acids and their ratios are measured
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org943cb5b}VLCFAs in Peroxisomal Disease}
\centering
\begin{tabular}{llll}
Disease & VLCFA & phytanic & pristanic\\
\hline
ZS & \(\uparrow\) & N or \(\uparrow\) & N or \(\uparrow\)\\
RCDP & N & N or \(\uparrow\) & N or \(\downarrow\)\\
XALD & \(\uparrow\) & N & N\\
Refsum & N & N/\(\uparrow\) & N/\(\downarrow\)\\
\end{tabular}
\end{table}


\subsection{Method}
\label{sec:org49b5f45}
\begin{itemize}
\item gas chromatography-mass spectrometry (GC-MS) analysis
\item after derivatisation with N-methyl-N-(tert-butyldimethylsilyl)
trifluoroacetamide (MTBSTFA)
\item a robust and reliable method for the quantitative analysis of VLCFA,
pristanic acid and phytanic acid
\end{itemize}

\begin{enumerate}
\item Procedure
\label{sec:orgea9db14}
\begin{itemize}
\item to measure the total pool of VLCFA, pristanic acid and phytanic
acid, samples need to be subjected to both acidic and alkaline
hydrolysis, followed by extraction into hexane
\item after the hexane phase is washed once more, the sample is dried
under nitrogen followed by addition of pyridine and MTBSTFA and
heating of the samples at 80°C
\item the sample is dried again under nitrogen and taken up in hexane,
followed by GC-MS analysis
\end{itemize}
\end{enumerate}
\section{Glycosaminoglycans}
\label{sec:orgfb78f26}
\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{gags/figures/cs.png}
\caption{\label{fig:orgccbe033}Chondroitin Sulfate}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{gags/figures/ds.png}
\caption{\label{fig:org4fcf848}Dermatan Sulfate}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{gags/figures/hs.png}
\caption{\label{fig:org0304ed3}Heparan Sulfate}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{gags/figures/ks.png}
\caption{\label{fig:org8cfbc1f}Keratan Sulfate}
\end{figure}


\subsection{Berry Spot Test}
\label{sec:org7fe0a6f}
\begin{itemize}
\item provides a rapid qualitative evaluation of urine (Figure \ref{fig:orgd3759ee})
\item GAGs react with toluidine blue, a cationic dye, to yield a pink-colored compound
\item false-negatives and false-positives specimens
\item this fast procedure may pick up patients who have been referred to a
laboratory for other metabolic examinations
\item the
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{gags/figures/berryspot.png}
\caption{\label{fig:orgd3759ee}Positive (a) and Negative (b) Berry Spot Test}
\end{figure}


\subsection{Spectrophotometric}
\label{sec:orgc4989b9}
\begin{itemize}
\item urine sample
\item 1,9-dimethylene blue (DMB) forms a complex with sulfated GAGs
present in urine that can be measured at 520 nm
\end{itemize}


\subsection{Fractionation}
\label{sec:orgcd998a0}
\begin{itemize}
\item TLC or electrophoresis used to fractionate urine GAGs (Figures \ref{fig:org156cbcb} and \ref{fig:org68bdd6b})
\item stained with alcian blue
\end{itemize}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{gags/figures/tlc.png}
\caption{\label{fig:org156cbcb}TLC for urine GAGs}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{gags/figures/gel.png}
\caption{\label{fig:org68bdd6b}Electrophoresis for urine GAGs: LZ loading zone}
\end{figure}
\section{Oligosacarides}
\label{sec:org67f4f09}
\subsection{Fractionation}
\label{sec:org21e26e5}
\begin{itemize}
\item TLC is used to fractionate urine oligosaccharides (Figure \ref{fig:org895f642})
\item stained with orcinol
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{oligos/figures/tlc.png}
\caption{\label{fig:org895f642}TLC of urinary oligos:control, \(\alpha\)-mannosidosis, fucosidosis, Sandhoff and aspartylglucosaminuria}
\end{figure}
\section{Enzymes}
\label{sec:org7f4ff83}
\subsection{GALT}
\label{sec:orgcea9f89}
\subsubsection{Beutler Test}
\label{sec:org0d2fece}

\ce{Gal-1-P + UDP-Glu ->[GALT] Glu-1-P + UDP-Gal}

\ce{Glu-1-P ->[PGM] Glu-6-P}

\ce{Glu-6-P + NADP+ ->[G6PD] 6-glucuranate + NADPH}

\ce{6-glucuranate + NADP+ ->[6PGDH] ribulose-5-P + NADPH}

\begin{itemize}
\item fluorescence of NADPH is measured to determine GALT deficiency
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.4\textwidth]{enzymes/figures/beutler.jpg}
\caption{\label{fig:orgba54d15}Beutler Method}
\end{figure}


\subsubsection{Spotcheck}
\label{sec:orgbb09c27}

\ce{Gal-1-P + UDP-Glu ->[GALT] Glu-1-P + UDP-Gal}

\ce{Glu-1-P ->[PGM] Glu-6-P}

\ce{Glu-6-P + NADP+ ->[G6PD] 6-PG + NADPH}

\ce{NADPH + MTT ->[methoxy PMS] Coloured Formazan + NADP+}

\subsection{Biotinidase}
\label{sec:orgaf4d30b}

\ce{biotin-PAB ->[BTD][pH 6] biotin + PABA}

\ce{PABA ->[NO2, NH2SO3][NED] purple chromophore}

\subsection{Lysosomal Enzymes}
\label{sec:org67b6db9}
\begin{itemize}
\item activity assays report rates (umol/L/h)
\end{itemize}

\subsubsection{4MU Iduronidase Assay}
\label{sec:orgf3677a7}
\begin{itemize}
\item 4-MU-\(\alpha\)-L-iduronide is a fluorogenic substrate
of \(\alpha\)-iduronidase is incubated with whole blood or leukocytes for \textasciitilde{} 20 hours
\begin{itemize}
\item 5 hours possible
\end{itemize}
\item 4-MU is quenched when linked to \(\alpha\)-L-iduronide
\begin{itemize}
\item ex=360 and em=445
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.4\textwidth]{enzymes/figures/M337643.png}
\caption[4MUI]{\label{fig:org73c57fd}4-Methylumbelliferyl-\(\alpha\)-L-Iduronide}
\end{figure}

\begin{enumerate}
\item Known Issues
\label{sec:orge5f6ffc}
\begin{enumerate}
\item Psuedodeficiency
\label{sec:org93d4009}
\begin{itemize}
\item Low \emph{in vitro} \(\alpha\)-iduronidase activity with native and 4-MU
substrates
\item Known PD alleles:
\begin{itemize}
\item p.A300T, p.A79T, p.H82Q, p.D223N, p.V322E
\end{itemize}
\end{itemize}

\item Sample quality
\label{sec:org541ee81}
\begin{itemize}
\item Poor sample quality \(\to\) low enzyme activity
\begin{itemize}
\item measurement of control enzymes useful
\end{itemize}
\end{itemize}
\end{enumerate}
\end{enumerate}

\subsubsection{Multiplex DBS Assay}
\label{sec:org353e519}
\begin{itemize}
\item a single 3-mm DBS punch, which is incubated in a single-assay
cocktail with all substrates and internal standards for \textasciitilde{} 20 hours
\item after incubation and liquid-liquid extraction, samples are analyzed by flow injection MS/MS
\item all deuterated internal standards correspond to enzymatically generated products
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org56009c7}Neo-LSD Lysosomal enzyme activities}
\centering
\begin{tabular}{lll}
enzyme & disease & abbreviation\\
\hline
\(\alpha\)-iduronidase & MPS 1 & IDUA\\
\(\beta\)-glucocerebrosidase & Gaucher & ABG\\
acid \(\alpha\)-glucosidase & Pompe & GAA\\
galactocerebrosidase & Krabbe & GALC\\
acid sphingomylinase & NPA/B & ASM\\
\(\alpha\)-galactosidase & Fabry & GLA\\
\end{tabular}
\end{table}

\subsubsection{Hexosamindase}
\label{sec:orga25aca4}
\begin{itemize}
\item total hexosaminidases (A+B) using a synthetic fluorogenic substrate
allows the diagnosis of Sandhoff disease
\item differential assay of HexA using heat or acid
inactivation does not identify patients with the B1 variant
\item HexA is heat-labile while Hex B is not and total hexosaminidase
\item an artificial substrate is most commonly used
\item total hexosaminidase activity is quantified
\begin{itemize}
\item following this, heat inactivation of HexA occurs with a second
measurement of the total enzyme level
\item from this, the percent HexA is calculated
\begin{itemize}
\item HexA = [Prior to Heat] - [After Heat\}
\item HexA = Total - HexB
\end{itemize}
\end{itemize}
\item Tay-Sachs disease is characterized by normal total hexosaminidase
with a very low percent HexA (Table \ref{tab:org1fa1e2b})
\item carriers of Tay-Sachs disease are asymptomatic, but have
intermediate percent HexA in serum and leukocytes
\item follow-up molecular testing is recommended for all individuals with
enzyme results in the carrier or possible carrier ranges to
differentiate carriers of a pseudodeficiency allele from those with
a disease-causing mutation
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org1fa1e2b}Biochemical Results in GM2 Gangliosidosis}
\centering
\begin{tabular}{lllll}
Disorder & Enzyme & HexA + HexB & Hex A & HexA\%\\
\hline
Tay-Sachs & HexA & N & \(\Downarrow\) & \(\Downarrow\)\\
TS carrier & HexA & \dowarrow & \(\downarrow\) & \(\downarrow\)\\
Sandhoff & HexA/B & \(\Downarrow\) & \(\Downarrow\) & \(\uparrow\)\\
\end{tabular}
\end{table}


\begin{enumerate}
\item HexA assay
\label{sec:org109f6f6}
\begin{itemize}
\item the direct assay of hexosaminidase A using the sulfated synthetic
substrate (4-MU-6-sulfo-/beta-glucosaminide) specific for the
\(\alpha\)-subunit is the method of choice
\item \emph{in vitro} hexoaminidase activity: leukocytes, fibroblasts
\begin{itemize}
\item 4-MU-6-sulfo-\(\beta\)-glucosaminide
\item specific for the \(\alpha\) subunit
\end{itemize}
\item falsely normal results in Tay-Sachs female carriers
\end{itemize}

\item B1 Variant
\label{sec:org021ee92}
\begin{itemize}
\item very small group of patients affected with Tay-Sachs disease have
mutations referred to as the B1 variant of Hex A
\item in the presence of an artificial substrate, the B1 variant allows
for a heterodimer formation of HexA and exhibits
activity
\begin{itemize}
\item \emph{in vivo} the B1 variant HexA is inactive on the natural
substrate
\item with the artificial substrate, these patients appear to be
unaffected
\item B1 variant of Tay-Sachs disease must be distinguished using a
natural substrate assay
\item patients with at least one B1 variant typically become symptomatic
beyond the infantile period
\end{itemize}
\end{itemize}
\end{enumerate}
\section{Glycosylation}
\label{sec:orga1cc498}
\subsection{Transferrin IEF}
\label{sec:org5980018}
\begin{itemize}
\item serum transferrin IEF is the screening method of choice
\begin{itemize}
\item can detect nearly all known CDG-I types as well as most CDG-II types and many CDG-X cases
\item N-glycosylation disorders associated with sialic acid deficiency
\item does not detect CDG-IIb or IIc
\end{itemize}
\item normal serum transferrin is mainly composed of:
\begin{itemize}
\item tetrasialotransferrin and small amounts of mono, di, tri, penta
and hex-asialotransferrins
\end{itemize}
\item partial deficiency of sialic acid (-ve charge) causes a
cathodal shift
\item two main types of cathodal shift can be recognized:
\begin{itemize}
\item type 1 or 2 patterns
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=1\textwidth]{glyc/figures/trans1or2.png}
\caption{\label{fig:org38b528f}Transferrin IEF}
\end{figure}

\subsubsection{Type 1 pattern}
\label{sec:org066b18d}
\begin{itemize}
\item \(\uparrow\) disialo- and asialotransferrin
\item \(\downarrow\)  tetra-, penta-and hexasialotransferrins
\item defects in the assembly of the dolichol lipid-linked
oligosaccharide chain and transfer to the nascent protein
\item CDG-Ia or CDG-Ib should be considered first
\item also seen in secondary glycosylation disorders such as:
\begin{itemize}
\item chronic alcoholism, hereditary fructose intolerance and galactosaemia
\end{itemize}
\end{itemize}

\subsubsection{Type 2 pattern}
\label{sec:org6227ce5}
\begin{itemize}
\item Type 1 pattern with additional \(\uparrow\) tri- \textpm{}
monosialotransferrin bands
\item defects in the trimming and processing of the protein-bound
glycans either late in the endoplasmic reticulum or the Golgi
compartments
\end{itemize}

\subsubsection{Transferrin IEF limitations}
\label{sec:org07d6166}
\begin{itemize}
\item deficiencies of ER-glucosidase I (CDG-IIb) and Golgi GDP-fucose
transporter (CDG-IIc) are missed
\item prenatal diagnostics by IEF analysis from fetal blood is not
reliable
\item IEF of serum from children \textless{} 2 weeks may be false-positive
\item heavy alcohol consumption can also result in serum transferrin
deficiency in carbohydrate moieties, leading to an abnormal
IEF-pattern
\item mutations in the protein backbone of transferrin
\begin{itemize}
\item desialylation of transferrin by neuraminidase treatment or IEF of
an alternative glycoprotein like \(\alpha\) 1-antitrypsin should be
performed
\end{itemize}
\item galactosemia and HFI may cause false positive patterns
\end{itemize}
\subsection{Additional Laboratory Investigations}
\label{sec:org7b2735b}
\begin{itemize}
\item protein-linked glycan analysis can be performed to identify the defective step
\begin{itemize}
\item MALDI-TOF analysis of released N-linked oligosaccharides
\end{itemize}
\item CDG gene panel analysis or WES

\item capillary zone electrophoresis of total serum is a rapid screening
test for CDG
\begin{itemize}
\item an abnormal result should be further investigated by serum
transferrin IEF
\end{itemize}
\item HPLC-UV/Vis
\end{itemize}
\section{NIET}
\label{sec:orgaa2d238}
\begin{itemize}
\item used to identify a GSD in patients with myopathy (Table
\ref{tab:orgbefad0a} and Figure \ref{fig:org6c70861})
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:orgbefad0a}NIET in Myopathy}
\centering
\begin{tabular}{lll}
condition & lactate & ammonia\\
\hline
GSD I & N & N\\
GSD III (L\&M) & \(\Downarrow\) & N/\(\uparrow\)\\
GSD V & \(\Downarrow\) & N/\(\uparrow\)\\
GSD VII & \(\Downarrow\) & N/\(\uparrow\)\\
GSD IX & \(\Downarrow\) & N/\(\uparrow\)\\
GSD X & \(\downarrow\) & N/\(\uparrow\)\\
alcoholic myopathy & N & N\\
chronic fatigue & N & N\\
poor effort & N/\(\downarrow\) & N/\(\downarrow\)\\
mito myopathy & variable & N\\
\end{tabular}
\end{table}

\subsection{Exercising Muscle}
\label{sec:org05b13ff}
\subsubsection{Lactate}
\label{sec:org1b21baf}
\begin{itemize}
\item lactate, ammonia and purine compounds are generated by exercising muscle
\item exercising muscle generates lactic acid from the anaerobic breakdown
of glycogen to pyruvate
\begin{itemize}
\item pyruvate \(\to\) lactate
\end{itemize}
\item lactate enters the circulation and is converted back to pyruvate in
the liver by LDH (Figure \ref{fig:orgca9b249})
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.4\textwidth]{niet/figures/Lactate_dehydrogenase_mechanism.png}
\caption{\label{fig:orgca9b249}LDH}
\end{figure}

\subsubsection{ATP}
\label{sec:orgd5f2e0e}
\begin{itemize}
\item some ATP regeneration is provided by glycolytic metabolism of fuels,
but this is relatively slow
\item most ATP regeneration relys on creatine kinase catalysed transfer of
phosphate from phosphocreatine
\end{itemize}

\ce{phosphocreatine + ADP ->[CK] creatine + ATP}

\begin{itemize}
\item adenylatekinase transphosphorylates ATP to be regenerated with the formation
of AMP

\ce{2ADP ->[ADK] ATP + AMP}

\item AMP deaminase
\end{itemize}

\ce{AMP ->[AMPD] IMP + NH4+}

\begin{itemize}
\item IMP degraded to hypoxanthine
\item recycled back to AMP in the purine nucleotide cycle
\end{itemize}

\subsubsection{Ammonia}
\label{sec:org772468f}
\begin{itemize}
\item most ammonia produced by exercising muscle removed by formation of glutamine (Figure \ref{fig:orgd6a40ab})
\begin{itemize}
\item ultimately excreted as urea
\end{itemize}
\item some ammonia is released by exercising skeletal muscle directly into the circulation
\begin{itemize}
\item removed with a half-life of 20\textpm{}30 min
\end{itemize}
\item in resting skeletal muscle ammonia is consumed rather than produced
\item \textasciitilde{}50\% of arterial ammonia can be taken up and metabolized by skeletal muscle
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.6\textwidth]{niet/figures/nitrogen_glutamine.png}
\caption[gln]{\label{fig:orgd6a40ab}Glutamine and Ammonia}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=.6\textheight]{niet/figures/niet_results.png}
\caption[interp]{\label{fig:org6c70861}NIET Results}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{niet/figures/niet_method.png}
\caption{\label{fig:org673a388}NIET Method}
\end{figure}
\section{Porphyrins}
\label{sec:orgff1037b}
\subsection{Methods for Metabolites}
\label{sec:org9ab0ba3}
\begin{enumerate}
\item ALA
\item PBG
\item Urinary Porphyrins
\item Fecal Porphyrins
\item Blood Porphyrins
\end{enumerate}
\subsubsection{Specimen Collection and Stability}
\label{sec:org0cddc9a}
\begin{itemize}
\item protect from light
\item urinary porphyrins and PBG best collected in fresh random urine
without preservative. Very dilute urine (creatinine <2 mmol/L) is
not suitable
\end{itemize}
\subsubsection{Methods for Porphyrin Precursors}
\label{sec:org65714b8}
\begin{enumerate}
\item Porphobilinogen
\label{sec:org16a785e}
\begin{itemize}
\item ehrlich's reagent
\begin{itemize}
\item inhibited by urobilinogen \(\therefore\) ion-exchange prior to remove
\end{itemize}
\end{itemize}
\item 5-Aminolevulinic Acid
\label{sec:org70358ce}
\begin{itemize}
\item converted to Ehrlich's reacting pyrrole via condensation with
acetylacetone
\end{itemize}
\item Analysis of Porphyrins in Urine and Feces
\label{sec:orgcaa2bb5}
\begin{itemize}
\item screening via spectrophotometric scanning of acidified urine or
fecal extracts for the soret band
\end{itemize}
\item Semiquantitative Method for Total Porphyrin in Urine and Feces
\label{sec:orge2f9d63}
\begin{itemize}
\item done as a screen
\end{itemize}
\item HPLC Fractionation of Porphyrins in Urine and Feces
\label{sec:org2f38276}
\begin{itemize}
\item fluorometric detection
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{porphyrins/figures/urine.pdf}
\caption{\label{fig:org88b052e}Urine Porphyrins}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{porphyrins/figures/fecal.pdf}
\caption{\label{fig:orgc91fafd}Fecal Porphyrins}
\end{figure}
\end{enumerate}

\subsubsection{Methods for Blood Porphyrins}
\label{sec:org18ad402}
\begin{enumerate}
\item Determination of Erythrocyte Total Porphyrin
\label{sec:orgc5b135a}
\begin{itemize}
\item increased in:
\begin{itemize}
\item EPP
\item CEP
\item homozygous variants
\item iron deficiency
\item hemolytic anemia
\item some other forms of anemia
\item lead poisoning
\end{itemize}
\item total porphyrin concentration within RI excludes EPP
\begin{itemize}
\item distinction between EPP and other causes requires differentiation
between protoporphyrin and ZN-protoporphyrin
\end{itemize}
\end{itemize}
\item Qualitative Determination of ZN-protoporphyrin and Protoporphyrin
\label{sec:org0b27332}
\begin{itemize}
\item emission peaks for ZPP = 587 nm and free protoporphyrin = 630 nm
\item in EPP the concentration of free protoporphyrin >> ZPP
\begin{itemize}
\item may be 60\% of total porphyrin
\end{itemize}
\item in lead poisoning, iron deficiency and other anemias ZPP is the main
component
\item limitation of method is that extraction of ZPP is \textasciitilde{}50\%
\end{itemize}
\end{enumerate}

\subsubsection{Analysis of Plasma Porphyrins}
\label{sec:org9bceb8e}
\begin{enumerate}
\item Fluorescence emision Spectroscopy of Plasma Porphyrins
\label{sec:org1b4f866}
\begin{itemize}
\item emission spectra of saline diluted plasma excited at 405 nm
\item in VP the plasma contains porphyrin covalently bound to protein with
Em\textsubscript{max} at 624 to 628 nm
\item other porphyrias contain porphyrin non-covalently bound to albumin
and hemopexin
\begin{itemize}
\item fresh plasma protoporphyrin Em\textsubscript{max} = 632 nm
\item older sample: binding to globulin released from red cells Em\textsubscript{max} =
626 nm
\end{itemize}
\item 2\degree causes of increase include: impaired excretion, renal
failure, cholestasis
\end{itemize}
\end{enumerate}

\subsection{Enzyme Measurements}
\label{sec:org403ea36}
\begin{itemize}
\item rarely required for patients with symptoms
\item can be used for family studies, DNA is better
\end{itemize}
\subsubsection{Assay of Etythrocyte Hydroxymethylbilane Synthase Activity}
\label{sec:org16b6f55}
\begin{itemize}
\item measures rate of formation of porphyrinogens from PBG by hemolysed erythrocytes
\item discriminates between AIP and unaffected relatives
\item interferences:
\begin{itemize}
\item HMBS activity declines sharply with erythrocyte age
\item affected by \(\propto\) of retics, and young cells in peripheral blood
\item \(\uparrow\) in acute illness, ie acute porphyria
\item \(\sim\) 1:800 low HMBS activity in France
\end{itemize}
\end{itemize}
\end{document}