% Created 2020-04-07 Tue 13:20
% Intended LaTeX compiler: pdflatex
\documentclass{scrartcl}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{longtable}
\usepackage{wrapfig}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{amsmath}
\usepackage{textcomp}
\usepackage{amssymb}
\usepackage{capt-of}
\usepackage{hyperref}
\hypersetup{colorlinks,linkcolor=black,urlcolor=blue}
\usepackage{textpos}
\usepackage{textgreek}
\usepackage[version=4]{mhchem}
\usepackage{chemfig}
\usepackage{siunitx}
\usepackage{gensymb}
\usepackage[usenames,dvipsnames]{xcolor}
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage{verbatim}
\usepackage{tikz}
\usepackage{wasysym}
\usetikzlibrary{shapes.geometric,arrows,decorations.pathmorphing,backgrounds,positioning,fit,petri}
\usepackage{fancyhdr}
\pagestyle{fancy}
\author{Matthew Henderson, PhD, FCACB}
\date{\today}
\title{Methods}
\hypersetup{
 pdfauthor={Matthew Henderson, PhD, FCACB},
 pdftitle={Methods},
 pdfkeywords={},
 pdfsubject={},
 pdfcreator={Emacs 26.3 (Org mode 9.1.9)}, 
 pdflang={English}}
\begin{document}

\maketitle
\tableofcontents


\tikzstyle{chemical} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=black, fill=none]
\tikzstyle{hardware} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=black, fill=gray!30]
\tikzstyle{ms} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=orange, fill=none]
\tikzstyle{msw} = [rectangle, rounded corners, text width=7em, minimum height=1em,text centered, draw=orange, fill=none]
\tikzstyle{label} = [rectangle,text width=8em, minimum height=1em, text centered, draw=none, fill=none]
\tikzstyle{hl} = [rectangle, rounded corners, text width=5em, minimum height=1em,text centered, draw=black, fill=red!30]
\tikzstyle{box} = [rectangle, rounded corners, text width=5em, minimum height=5em,text centered, draw=black, fill=none]
\tikzstyle{arrow} = [thick,->,>=stealth]
\tikzstyle{hl-arrow} = [ultra thick,->,>=stealth,draw=red]



\section{NBS}
\label{sec:org33b70a3}
\subsection{Dried Blood Spot}
\label{sec:orgcebe4ba}
\begin{itemize}
\item collected from free flowing blood spotted onto filter paper
\item used for NBS and monitoring
\item each DBS is assume to contain 3.2 \textmu{}L of blood
\item the quantity of blood present in the paper varies by
\begin{itemize}
\item hematocrit
\item degree of saturation
\item the cotton fiber paper
\item the environment  when applying blood (humidity and temperature)
\end{itemize}
\item because of these numerous factors, a dried blood spot is a highly
imprecise specimen compared with liquids such as urine, blood, and plasma
\end{itemize}

\subsection{AAAC}
\label{sec:org46459ed}
\begin{enumerate}
\item FIA-MS/MS schematic
\label{sec:orgb84ec0b}
\begin{center}
\begin{tikzpicture}[node distance=7em]
% nodes
\node(ms1)[ms]{MS1: Mass Filter};
\node(cc)[ms, right of=ms1]{Collision cell};
\node(ms2)[ms, right of=cc]{MS2: Mass Filter};
\node(ion)[ms, below of=ms1,yshift=3em]{Ionization};
\node(lc)[msw, below of=ion,yshift=3em]{Injection};
\node(detector)[ms, below of=ms2, yshift=3em]{Detector};
% arrows
\draw[arrow](lc) -- (ion);
\draw[arrow](ion) -- (ms1);
\draw[arrow](ms1) -- (cc);
\draw[arrow](cc) -- (ms2);
\draw[arrow](ms2) -- (detector);
\end{tikzpicture}
\end{center}

\item FIA-MS/MS sample
\label{sec:orga73067b}

\begin{itemize}
\item amino acids and acylcarnitines in the DBS eluate are esterified as butyl esters with butanol-hydrogen chloride
\item solvent delivery is via HPLC with no chromatography, called flow injection analysis
\item 10 \textmu{}L of sample extract is injected into a flowing stream operating at \textasciitilde{} 0.15 ml/min

\item typical injection rates between samples are 2 min, giving a
potential 400-600 sample capacity per instrument per day
\begin{itemize}
\item volume is typically 200-400 specimens per instrument per day
\item maintenance issues, repeat specimen analysis
\end{itemize}
\end{itemize}

\centering
%\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.33]{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]OH}}{\tiny \textalpha{}-amino acid}
\+
\chemname{\chemfig[][scale=.33]{HO-[::30]-[::-60]-[::60]-[::-60]}}{\tiny n-butanol}
\arrow{-U>[][{\tiny \ce{H2O}}]}
\chemname{\chemfig[][scale=.33]{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\tiny AA butyl ester}
\schemestop

\definesubmol{x}{-[1,.6]-[7,.6]}
\definesubmol{y}{-[7,.6]-[1,.6]}
\definesubmol{d}{!y!y-[7,.6]{\color{red}COOH}}
\definesubmol{e}{!y!y}
\centering
\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.33]{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e)-[1]-[7]([7]=O)([1]-O^{-})}}{\tiny C5-carnitine}
\+
\chemname{\chemfig[][scale=.33]{HO!x!x}}{\tiny n-butanol}
\arrow{-U>[][{\tiny \ce{H2O}}]}
\chemname{\chemfig[][scale=.33]{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e)-[1]-[7]([6]=O)-[1,.6]O!y!y}}{\tiny C5-carnitine, butyl ester}
\schemestop
\vspace{2em}
\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.33]{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!d)-[1]-[7]([7]=O)([1]-O^{-})}}{\tiny C6DC-carnitine}
\+
\chemname{\chemfig[][scale=.33]{HO!x!x}}{\tiny n-butanol}
\arrow{-U>[][{\tiny \ce{2H2O}}]}
\chemname{\chemfig[][scale=.33]{-N^{+}([2]-)([6]-)-[1]-[7]([6]-O-([5]=O)!e-[7,.6]O!x!x)-[1]-[7]([6]=O)-[1,.6]O!y!y}}{\tiny C6DC-carnitine, butyl ester}
\schemestop 

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./nbs/figures/outletmethod.pdf}
\caption{\label{fig:org4e26b20}
FIA-MS/MS Method}
\end{figure}

\item AA NL Scan
\label{sec:org61f9464}
\begin{itemize}
\item electrospray ionization in positive mode
\item \(\alpha\)-amino acids fragment to produce the neutral butyl formate molecule (102 Da)
\item neutral loss scan is used to identify parent molecules with a NL of 102 Da
\item MRM is used to detected amino acids with basic functional groups
\begin{itemize}
\item arginine, ornithine and citrulline
\end{itemize}
\end{itemize}

\centering
\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.33]{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\tiny AA butyl ester}
\arrow{->[{\tiny fragmentation}]}
\chemname{\chemfig[][scale=.33]{{\color{red}R}-[::60]=NH_2^{+}}}{\tiny fragment}
\+
\chemname{\chemfig[][scale=.33]{H-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\tiny butyl formate (102 Da)}
\schemestop
\item AA MRM
\label{sec:org099a4ee}
\begin{itemize}
\item citrulline contains a labile amino group that fragments together with butyl formate
\item CID results in net neutral fragmentation of butyl formate (102 Da) plus \ce{NH3} (17 Da)
\item \href{https://en.wikipedia.org/wiki/Selected\_reaction\_monitoring}{SRM} Citrulline-Bu 232.15 Da \(\to\) 113 Da , loss of 119 Da
\end{itemize}

\centering
\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.33]{H_2N-[::30,,2,](=[::60]O)-[::-60]NH-[::60]-[::-60]-[::60]-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]OH}}{\tiny citrulline 175 Da}
\+
\chemname{\chemfig[][scale=.33]{HO-[::30]-[::-60]-[::60]-[::-60]}}{\tiny n-butanol 74 Da}
\arrow{-U>[][{\tiny \ce{H2O}}]}
\chemname{\chemfig[][scale=.33]{H_2N-[::30,,2,](=[::60]O)-[::-60]NH-[::60]-[::-60]-[::60]-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\tiny 232 Da}
\schemestop

\centering
\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.33]{H_2N-[::60]-[::-60]-[::60]-[::-60]-[::60]N=O=C}}{\tiny 113 Da}
\+
\chemname{\chemfig[][scale=.33]{H-[::60](=[::60]O)-[::-60]O-[::60]-[::-60]-[::60]-[::-60]}}{\tiny 102 Da}
\+
\chemname{\chemfig[][scale=.43]{NH_3}}{\tiny 17 Da}
\schemestop

\begin{table}[htbp]
\caption{\label{tab:orgbafdba3}
Quantified Amino Acids}
\centering
\begin{tabular}{ll}
Glycine & Tyrosine\\
Alanine & Ornithine\\
Valine & Citruline\\
Leucine & Arginine\\
Methionine & \color{blue}Succinylacetone\\
Phenylalanine & \\
\end{tabular}
\end{table}

\item AC Precursor Ion Scan
\label{sec:org8beca2d}
\begin{itemize}
\item electrospray ionization in positive mode
\item butylated acylcarnitines fragment to produce a characteristic ion with mass of 85 Da
\item precursor ion scan is used to identify molecules that fragment to form a 85 m/z molecule
\end{itemize}

\definesubmol{x}{-[1,.6]-[7,.6]}
\centering
 \chemname{\chemfig[][scale=.33]{H_{3}C-N^{+}([2]-CH_3)([6]-CH_{3})-CH_2-C([2]-H)([6]-O-C([0]=O)-{\color{red}R})-CH_2-C([2]=O)-O-CH_2-CH_2-CH_2-CH_3}}{\tiny acylcarnitine, butyl ester}

\vspace{2.5em}

 \chemname{\chemfig[][scale=.33]{H_{3}C-N([1]-CH_3)([7]-CH_3)}}{\tiny trimethylamine}
\hspace{2em}
\chemname{\chemfig[][scale=.33]{{\color{red}R}-C([1]=O)([7]-OH)}}{\tiny carboxylic acid}
\hspace{2em}
 \chemname{\chemfig[][scale=.33]{H!x!x}}{\tiny butyl group}
\hspace{2em}
 \chemname{\chemfig[][scale=.33]{H_{2}C^{+}-HC=CH-C([1]=O)([7]-OH)}}{\tiny 85 m/z}

\begin{center}
\begin{tikzpicture}
\node[box](ms1)[]{};
\node[label](ms1u)[above=of ms1,yshift=-3em]{MS1};
\node[label](ms1l)[below=of ms1,yshift=3em]{scanning};
\node[box](cc)[right= of ms1]{};
\node[label](ccu)[above=of cc,yshift=-3em]{Collision cell};
\node[label](ccl)[below=of cc,yshift=3em]{fragmentation};
\node[box](ms2)[right= of cc]{};
\node[label](ms2u)[above=of ms2,yshift=-3em]{MS2};
\node[label](ms2l)[below=of ms2,yshift=3em]{85 m/z};
\draw[->](ms1) -- (cc);
\draw[->](cc) -- (ms2);

%ms1
\draw [gray,->, decorate,decoration=snake] (-.8,0.5) -- (.8,0.5);
\draw [gray,->, decorate,decoration=snake] (-.8,0.25) -- (.8,0.25);
\draw [blue, ->,decorate,decoration=snake] (-.8, 0) -- (.8,0);
\draw [gray,->, decorate,decoration=snake] (-.8,-0.25) -- (.8,-0.25);
\draw [gray,->,decorate,decoration=snake] (-.8,-0.5) -- (.8,-0.5);

%cc
\draw [blue,->,decorate,decoration=snake] (2.1, 0) -- (2.4,0);
\fill (2.6,0) circle (0.1); 
\draw [gray,->,decorate,decoration=snake] (2.8, 0) -- (3.8,0.5);
\draw [red, ->,decorate,decoration=snake] (2.8, 0) -- (3.8,0);
\draw [gray,->,decorate,decoration=snake] (2.8, 0) -- (3.8,-0.5);

%ms2
\draw [red,->,decorate,decoration=snake] (5.1, 0) -- (6.8,0);
\end{tikzpicture}
\end{center}

\item AC MRM
\label{sec:org0eed321}
\begin{itemize}
\item C0-Bu 218.1 Da \(\to\) 103 Da transition is optimal
\item all others benefit from the added sensitivity of MRM mode as
compared to parent ion scan
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:orgc9abc1c}
MRM is used to detected selected acylcarnitines}
\centering
\begin{tabular}{ll}
Compound & Reaction\\
\hline
C0 & 218.10 > 103.00\\
C0 IS & 227.10 > 103.00\\
C2 & 260.20 > 85.00\\
C2 IS & 263.20 > 85.00\\
C3 & 274.20 > 85.00\\
C3 IS & 277.20 > 85.00\\
C3DC & 360.30 > 85.00\\
C4DC & 374.30 > 85.00\\
C5DC & 388.35 > 85.00\\
C5DC IS & 391.35 > 85.00\\
C6DC & 402.45 > 85.00\\
C8DC & 430.45 > 85.00\\
\end{tabular}
\end{table}

\begin{table}[htbp]
\caption{\label{tab:org68009f5}
Quantified Acylcarnitines}
\centering
\begin{tabular}{lll}
C0 & C8 & C16\\
C2 & C8:1 & C16:1\\
C3 & C10 & C16:1-OH\\
C3DC & C10:1 & C16-OH\\
C4 & C12 & C18\\
C4DC & C12:1 & C18:1\\
C5 & C14 & C18:1-OH\\
C5:1 & C14:1 & C18:2\\
C5DC & C14:2 & C18-OH\\
C5-OH & C14-OH & \\
C6 &  & \\
C6DC &  & \\
\end{tabular}
\end{table}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./nbs/figures/ionization.png}
\caption{\label{fig:org0c299d4}
Rationale for Derivatization}
\end{figure}

\item Overestimation of Free Carnitine
\label{sec:org1be71b0}
\begin{itemize}
\item butylated acylcarnitines are converted to butylated carnitine in
n-butanol-3M HCl at 65\degree{}C. \footnote{Johnson, D. W. (1999). Inaccurate measurement of free
carnitine by the electrospray tandem mass spectrometry screening
method for blood spots. Journal of Inherited Metabolic Disease, 22(2),
201–202.\label{orga80e51c}}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:orgec170f0}
Overestimation of Free Carnitine}
\centering
\begin{tabular}{lr}
Acyl Carnitine & Half-life (min)\\
\hline
C2 & 31\\
C10 & 125\\
C18 & 172\\
\end{tabular}
\end{table}

\begin{itemize}
\item 65\degree{}C for 15 min.
\item NSO uses 60\degree{}C for 20 minutes.
\item IMD uses 55\degree{}C for 20 minutes.

\item in a sample with low free carnitine and high acetylcarnitine
\begin{itemize}
\item 30\% of the acetylcarnitine and smaller amounts of higher
molecular mass acylcarnitines are converted to carnitine
\item a low carnitine sample could appear to be normal
\end{itemize}
\item "the free carnitine results obtained by this screening method on
blood spots with high levels of acylcarnitines should therefore be
used with caution." \textsuperscript{\ref{orga80e51c}}
\end{itemize}

\item Pros and Cons of FIA-MS/MS using DBS
\label{sec:orgd295fd8}
\begin{itemize}
\item as compared to AAA and LC-MS/MS
\end{itemize}
\begin{enumerate}
\item Pros
\label{sec:org0d83a61}
\begin{itemize}
\item \textasciitilde{} 2 min analysis time
\item analyte specificity
\item ACs and AAs quantified simultaneously
\end{itemize}

\item Cons
\label{sec:org4fab69e}
\begin{itemize}
\item variability in DBS sample as described above
\item iso-baric compounds
\begin{itemize}
\item leucine, Isoleucine, Alloisoleucine
\item C5DC and C10-OH
\end{itemize}
\item overestimation of CO due to hydrolysis
\item fewer AA quantified
\begin{itemize}
\item homocystine (free)
\item glutamine
\end{itemize}
\end{itemize}
\end{enumerate}
\end{enumerate}
\section{Amino Acids}
\label{sec:org140f420}
\subsection{Introduction}
\label{sec:org7c91ced}
\begin{enumerate}
\item Amino Acids: A Very Short Introduction
\label{sec:org1c700a9}
\begin{itemize}
\item amino acids are mono or dicarboxylic acids with one or more amino groups.
\begin{itemize}
\item zwitterion at ph 7.45
\end{itemize}

\item proteinogenic amino acids (22)
\begin{itemize}
\item 21 amino acids naturally incorportated into polypeptides in humans
\item 20 genetically encoded
\item selenocysteine
\end{itemize}

\item non-proteinogenic
\begin{itemize}
\item post-translational modification
\begin{itemize}
\item hydroxylation of proline \(\to\) hydroxyproline
\end{itemize}
\item not found in proteins
\begin{itemize}
\item gamma-aminobutryic acid
\item ornithine, citrulline
\end{itemize}
\end{itemize}

\item 76 amino acids of biological interest in humans
\end{itemize}

\item Indications for Measurement of Amino Acids
\label{sec:org1f337f3}
\begin{itemize}
\item Diagnosis of inborn errors of amino acid metabolism and transport
\item Diet monitoring in patients with known IEM
\item Nutritional assessment of patients with non-metabolic conditions [e.g. short bowel syndrome]
\item Signs and symptoms:
\begin{enumerate}
\item Lethargy, coma, seizures or vomiting in a neonate
\item Hyperammonaemia
\item Ketosis
\item Metabolic acidosis or lactic acidaemia
\item Alkalosis
\item Metabolic decompensation
\item Unexplained developmental delay or developmental regression
\item Polyuria, polydipsia and dehydration
\item Unexplained liver dysfunction
\item Unexplained neurological symptoms
\item Abnormal amino acid results on newborn screening programme
\item Previous sibling with similar clinical presentation
\item Clinical presentation specific to an amino acid disorder
\item Monitoring treatment and diet.
\end{enumerate}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org83f1e72}
Recommended Plasma AA Profile for Diagnosis of Amino Acid Disorders}
\centering
\begin{tabular}{lll}
Alanine & Glutamine & Ornithine\\
Alloisoleucine & Glycine & Phenylalanine\\
Arginine & Histidine & Proline\\
Argininosuccinic acid & Homocysteine \footnotemark & Serine\\
Cystine & Isoleucine & Sulphocysteine \footnotemark\\
Citrulline & Leucine & Taurine\\
Glutamic acid & Lysine & Threonine\\
Valine & Methionine & Tyrosine\\
\end{tabular}
\end{table}\footnotetext[2]{\label{org5b207df}Plasma total homocysteine is not detected by routine methods, plasma free homocystine analysis has poor clinical sensitivity.}\footnotetext[3]{\label{org4af4883}Sulphocysteine may not be detectable in plasma using routine methods}

\item Plasma
\label{sec:orgba26eac}
\begin{itemize}
\item Patient prep
\begin{itemize}
\item Fasting (overnight preferred, 4 hours minimum). Infants and children should be
drawn just before next feeding (2-3 hours without TPN if possible).
\end{itemize}
\item Sample
\begin{itemize}
\item LiHeparin venous plasma
\item Prompt separation and deproteinisation is essential for accurate measurement of (free) sulphur containing amino acids.
\item Protein binding: cystine and homocystine
\item release of arginase from RBCs
\item store at -20\degree{}C to limit glutamine decomposition
\end{itemize}
\end{itemize}

\item Plasma - Pre-analytical
\label{sec:org8dde097}
\begin{itemize}
\item Serum should not be used because there may be deamination (asparagine to
aspartic acid and glutamine to glutamic acid), loss of sulphur
containing amino acids and release of oligopeptides.
\item EDTA plasma is recommended in some labs as the specimen of
choice. The older literature reports ninhydrin positive artefacts in
EDTA plasma but modern tubes do not seem to have this problem.
\item Haemolysis will cause increases in serine, glycine, taurine,
phosphoethanolamine, aspartic acid, glutamic acid, ornithine and
decreased arginine.
\item Delayed separation or leucocyte and platelet contamination will
cause increased serine, glycine, taurine, phosphoethanolamine,
ornithine, glutamic acid and decreased arginine, homocystine,
cystine.
\item Phenylalanine and tyrosine increase if specimen separation is
delayed
\item Amino acids are more stable in frozen deproteinised plasma than
in frozen native plasma.
\item Capillary blood may be used with careful cleaning of the skin prior
to specimen collection provided the blood is flowing freely.
\item Free tryptophan may be lost when using sulphosalicylic acid as
deproteinising agent.
\begin{itemize}
\item trichloroacetic acid is the deproteinising agent of choice for
this amino acid.
\end{itemize}
\item Sodium metabisulphite, found in some intravenous preparations as a
preservative, can cause the conversion of cystine to sulphocysteine.
\end{itemize}

\item Urine
\label{sec:org5116e5d}
\begin{itemize}
\item Urine
\begin{itemize}
\item 24 hour or random urine
\item preservative free bottle
\item Specimen quality is checked by testing for nitrite and pH.
\begin{itemize}
\item Specimen deterioration causes:
\begin{itemize}
\item \(\downarrow\) serine
\item \(\uparrow\) \(\downarrow\) alanine
\item increased glycine
\item decarboxylation of glutamic acid \(\to\) \(\gamma\)-aminobutyric acid
\item breakdown of phosphoethanolamine \(\to\) ethanolamine and phosphate
\item breakdown of cystathionine \(\to\) homocystine
\item hydrolysis of peptides causing \(\uparrow\) proline
\end{itemize}
\end{itemize}
\item Fecal contamination causes increased proline, glutamic acid, branched chain amino acids but not hydroxyproline.
\item Faecal bacteria can produce \(\gamma\)-aminobutyric acid from glutamic acid and b-alanine from aspartic acid
\item reported in \textmu{}mol/g creatinine
\item Aminoaciduria due to overflow and amino acid transport defects.
\end{itemize}
\end{itemize}

\item Cerebral Spinal Fluid
\label{sec:org9a77e95}
\begin{itemize}
\item free of blood contamination
\item which tube is used?
\item investigation of neurological disorders and essential for the
diagnosis of non-ketotic hyperglycinaemia.
\item CSF/Plasma ratio of amino acids is more informative than an isolated CSF sample.
\begin{itemize}
\item A paired plasma sample should be obtained within two hours.
\end{itemize}
\end{itemize}
\item Sample prep
\label{sec:orge573962}
\begin{itemize}
\item Hydrophillic amino acids require deprotonization with acetonitrile or alcohol
\item Deproteination to release cysteine, homocysteine and tryptophan.
\end{itemize}
\end{enumerate}

\subsection{Amino Acid Analyser}
\label{sec:orgb7458e6}
\begin{itemize}
\item Cation-exchange chromatography using a lithium buffer system
followed by post-column derivatization with ninhydrin
\item Samples are de-proteinized with sulfosalicylic acid prior to
injection
\item Utilizes a lithium-based cation-exchange column
\item Eluting amino acids undergo post column reaction with ninhydrin
\item Optical detection in the visible spectrum
\begin{itemize}
\item amino acids: 570nm
\item imino acids 440 nm
\end{itemize}
\end{itemize}

\begin{center}
\begin{tikzpicture}[node distance=9em]
% nodes
\node(column)[msw, right of=extraction]{Chromatography};
\node(derivatization)[msw, right of=column]{Ninhyrin Derivatization};
\node(detector)[ms, right of=derivatization]{UV Detector};
% arrows
\draw[arrow](column) -- (derivatization);
\draw[arrow](derivatization) -- (detector);
\end{tikzpicture}

\vspace{2em}

\schemedebug{false}
\schemestart
\chemfig[][scale=.33]{*6(=*5(-(=O)-(-[6]OH)(-[8]OH)-(=O)-)-=-=-)}
\+
\chemfig[][scale=.5]{{\color{red}R}-[::-60](<[::-60]NH_3^+)-[::60](=[::60]O)-[::-60]OH}
\arrow{-U>[][{\tiny \ce{2H2O}}]}
\chemfig[][scale=.33]{*6(=*5(-(=O)-(=N-[::-60](-[::-60]{\color{red}R})-[::60](=[::60]O)-[::-60]OH)-(=O)-)-=-=-)}
\arrow{->[][]}
\chemfig[][scale=.33]{*6(=*5(-(=O)-(-N=*5(-(=O)-(*6(-=-=-))--(=O)-))-(=O)-)-=-=-)}
\schemestop

\schemedebug{false}
\schemestart
\chemfig[][scale=.33]{*6(=*5(-(=O)-(-[6]OH)(-[8]OH)-(=O)-)-=-=-)}
\+
\chemfig[][scale=.33]{H-*5(N----(-COOH)-)}
\arrow{->[][]}
\chemfig[][scale=.33]{*6(=*5(-(=O)-(-*5(N-----))-(=O)-)-=-=-)}
\schemestop
\end{center}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./aa/figures/aaachrom.png}
\caption{\label{fig:org00c60e5}
Amino Acids Analyser}
\end{figure}

\begin{enumerate}
\item Pros and Cons of Amino Acid Analysers
\label{sec:org495c6c5}
\begin{enumerate}
\item Pros
\label{sec:org01ebfab}
\begin{itemize}
\item Standardized and established technology
\item Interpretive experience
\end{itemize}
\item Cons
\label{sec:org2da5d78}
\begin{itemize}
\item Long run time (90 – 150 minutes)
\item Lack of analyte specificity (identification by retention time)
\begin{itemize}
\item antibiotics (ampicillin, amoxicillin, and gentamicin), acetaminophen
\end{itemize}
\item Co-eluting substances cannot be separated and distinguished on a standard IEC chromatogram
\begin{itemize}
\item Homocitrulline co-elutes with methionine
\item ASA co-elutes with leucine
\item Alloisoleucine co-elutes with cystathionine
\item Tryptophan co-elutes with histidine
\end{itemize}
\end{itemize}
\end{enumerate}
\end{enumerate}

\subsection{LC-MS/MS}
\label{sec:org7def4f2}
\begin{enumerate}
\item LC-MS/MS schematic
\label{sec:org2ab04f1}
\begin{center}
\begin{tikzpicture}[node distance=7em]
% nodes
\node(ms1)[ms]{MS1: Mass Filter};
\node(cc)[ms, right of=ms1]{Collision cell};
\node(ms2)[ms, right of=cc]{MS2: Mass Filter};
\node(ion)[ms, below of=ms1,yshift=3em]{Ionization};
\node(lc)[msw, below of=ion,yshift=3em]{Chromatography};
\node(detector)[ms, below of=ms2, yshift=3em]{Detector};
% arrows
\draw[arrow](lc) -- (ion);
\draw[arrow](ion) -- (ms1);
\draw[arrow](ms1) -- (cc);
\draw[arrow](cc) -- (ms2);
\draw[arrow](ms2) -- (detector);
\end{tikzpicture}
\end{center}


\item LC-MS/MS sample prep
\label{sec:org1bd7a92}
\begin{itemize}
\item 10 \textmu{}L of sample is mixed with 990 \textmu{}L of IS in 0.5 mM perfluoroheptanoic acid and centrifuge to deproteinize.
\item 200 \textmu{}L of supernatant is removed
\item 7.5 \textmu{}L is injected onto an octadecylsilyl (C18) stationary phase.
\end{itemize}


\begin{center}
\includegraphics[width=0.7\textwidth]{./aa/figures/outletmethod.png}
\end{center}

\item Ion-Pairing Chromatography
\label{sec:org5b555e5}
\chemfig[][scale=.70]{CF_3-{(CF_2)_4}-CF_2-[::30](=[::60]O)-[::-60]OH}
\vspace{3em}
\ce{AA+ + PFHA-  <=> AA+ PFHA-}

\item LC- MS/MS transitions
\label{sec:orgf38814b}
\begin{itemize}
\item ESI in positive mode
\begin{itemize}
\item MRM
\end{itemize}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org141a962}
AA Quantified by LC-MS/MS}
\centering
\begin{tabular}{lll}
phosphoserine & alanine & phenylalanine\\
taurine & citulline & aminoisobutyric\\
pphosphoethanolamine & 2-aminobutyric & \(\gamma\)-aminobutryic\\
aspartate & valine & ethanolamine\\
hydroxyproline & cystine & hydroxylysine\\
threonine & saccharopine & ornithine\\
serine & methionine & lysine\\
asparagine & alloisoleucine & 1-methylhistidine\\
glutamate & cystathionine & histidine\\
glutamine & isoleucine & tryptophan\\
sarcosine & leucine & 3-methylhistidine\\
aminoadipic & arginosuccinic acid & anserine\\
proline & tyrosine & carnosine\\
glycine & \(\beta\)-alanine & arginine\\
 &  & s-sulfocyteine*\\
\end{tabular}
\end{table}

\item Pros and Cons of LC-MS/MS vs FIA-MS/MS
\label{sec:orgc01e561}
\begin{enumerate}
\item Pros
\label{sec:org3310797}
\begin{itemize}
\item 43 vs 11 amino acids quantified
\begin{itemize}
\item Leu/Ile/Allo
\end{itemize}
\item Iso-baric compounds resolved
\begin{itemize}
\item Leucine, Isoleucine, Alloisoleucine
\end{itemize}
\end{itemize}
\item Cons
\label{sec:orgae052fb}
\begin{itemize}
\item Too slow for NBS
\item Manual peak integration
\end{itemize}
\end{enumerate}
\item Pros and Cons of LC-MS/MS vs AAA
\label{sec:org314adc4}
\begin{enumerate}
\item Pros
\label{sec:org092fbb1}
\begin{itemize}
\item \textasciitilde{} 30 min shorter analysis time
\item Analyte specificity
\begin{itemize}
\item Based on MRM rather than RT and ninhydrin reactivity
\begin{itemize}
\item gentamycin, acetaminophen, dopamine analogs
\end{itemize}
\item Co-eluting substances cannot be separated and distinguished on a
standard IEC chromatogram
\begin{itemize}
\item Homocitrulline co-elutes with methionine
\item ASA co-elutes with leucine
\item Alloisoleucine co-elutes with cystathionine
\item Tryptophan co-elutes with histidine
\end{itemize}
\end{itemize}
\item Long term reagent expense
\end{itemize}

\item Cons
\label{sec:orgc1029e7}
\begin{itemize}
\item Upfront hardware expense
\item Manual peak integration
\item Lab developed test - not standardized
\item Changing LOQ with equipment age
\end{itemize}
\end{enumerate}
\end{enumerate}
\section{Acylcarnitines}
\label{sec:org57ff7be}
\subsection{Introduction}
\label{sec:orgd4b40c2}
\begin{itemize}
\item carnitine (\(\beta\)-hydroxy-\(\gamma\)-N-trimethylaminobutyric acid) is
an endogenous quaternary ammonium compound synthesized from lysine
and methionine.
\item L-Carnitine has been described as a "conditionally essential"
nutrient for humans.
\item populations with an exogenous carnitine requirement include:
\begin{itemize}
\item infants (premature and full-term),
\item patients on long-term parenteral nutrition,
\item perhaps children.
\end{itemize}
\item exogenous carnitine is required to maintain "normal" (in the
epidemiologic sense) plasma or serum carnitine concentrations in
humans of all ages.
\item primary function is to shuttle long chain fatty acids to the
mitochondrial matrix, for \(\beta\)-oxidation.
\item acylcarnitines are markers for FAODs and OAs
\end{itemize}

\vspace{2em}

\chemname{\chemfig[][scale=.5]{H3C-N^{+}([2]-CH3)([6]-CH3)-CH2-C([2]-H)([6]-OH)-CH_2-C([1]=O)([7]-O^{-})}}{\tiny Carnitine}
\hspace{3em}
\chemname{\chemfig[][scale=.5]{H3C-N^{+}([2]-CH3)([6]-CH3)-CH2-C([2]-H)([6]-O-C([0]=O)-{\color{red}R})-CH_2-C([1]=O)([7]-O^{-})}}{\tiny Acylcarnitine}
%\chemname{\chemfig[][scale=.5]{H3C-N^{+}([2]-CH3)([6]-CH3)-CH2-C([2]-H)([6]-O-C([0]=O)-{\color{red}R})-CH_2-C([2]=O)-O-CH_2-CH_2-CH_2-CH_3}}{\tiny Acylcarnitine, butyl ester}

\subsection{Diagnostic FIA-MS/MS}
\label{sec:org45b3346}
\begin{itemize}
\item see \ref{sec:org46459ed}
\end{itemize}

\begin{enumerate}
\item Typical Panel
\label{sec:orgfb463f3}
\begin{itemize}
\item acetylcarnitine (C2)
\item propionylcarnitine (C3)
\item butyryl-/isobutyrylcarnitine (C4)
\item tiglylcarnitine (C5:1)
\item isovaleryl-/2-methylbutyryl-/pivalylcarnitine (C5)
\item hexanoylcarnitine (C6)
\item 3-hydroxyisovaleryl-/2-methyl-3-hydroxybutyrylcarnitine (C5-OH)
\item octenoylcarnitine (C8:1)
\item octanoylcarnitine (C8)
\item malonylcarnitine (C3-DC)
\item decadienoylcarnitine (C10:2)
\item decenoylcarnitine (C10:1)
\item decanoylcarnitine (C10)
\item methylmalonyl-/succinylcarnitine (C4-DC)
\item glutarylcarnitine (C5-DC)
\item dodecenoylcarnitine (C12:1)
\item dodecanoylcarnitine (C12)
\item adipyl-/methylglutarylcarnitine (C6-DC)
\item tetradecadienoylcarnitine (C14:2)
\item tetradecenoylcarnitine (C14:1)
\item myristoylcarnitine (C14)
\item suberylcarnitine (C8-DC)
\item 3-hydroxymyristoylcarnitine (C14-OH)
\item palmitoleylcarnitine (C16:1)
\item palmitoylcarnitine (C16)
\item sebacylcarnitine (C10-DC)
\item 3-hydroxypalmitoleylcarnitine (C16:1-OH)
\item 3-hydroxypalmitoylcarnitine (C16)
\item linoleylcarnitine (C18:2)
\item oleylcarnitine (C18:1)
\item stearoylcarnitine (C18)
\item 3-hydroxylinoleylcarnitine (C18:2-OH)
\item 3-hydroxyoleylcarnitine (C18:1-OH)
\item 3-hydroxystearoylcarnitine (C18-OH)
\item hexadecanedioylcarnitine (C16-DC).
\end{itemize}
\end{enumerate}


\subsection{{\bfseries\sffamily TODO} Free and Total Carnitine}
\label{sec:orgc776266}
\begin{itemize}
\item Method?
\end{itemize}
\begin{enumerate}
\item Fractional Tubular Re-absorption of Carnitine
\label{sec:orgaaa7911}

\begin{equation*}
FTR_{carnitine}\% = \left( 1 -  \frac{carnitine_{urine} \cdot creatinine_{plasma}}{carnitine_{plasma} \cdot creatinine_{urine}}\right) \cdot 100
\end{equation*}

\begin{itemize}
\item normally >98\%, \(\Downarrow\) in CUD
\end{itemize}

\item Free/Total Carnitine
\label{sec:org95ce6bd}

\[
\frac{Free_{carnitine}}{Total_{carnitine}} = \frac{C_0}{\sum_{0}^{18} C_n}
\]

\begin{itemize}
\item \(\Downarrow\) in CUD, < 5-10\% of normal
\end{itemize}
\end{enumerate}
\section{Organic Acids}
\label{sec:org581bac6}
\subsection{Background}
\label{sec:org8ec2447}
\begin{enumerate}
\item What are urine organic acids?
\label{sec:org6833388}
\begin{itemize}
\item water soluble compounds containing \(\ge\) one carboxyl group(s) and
nonamino functional groups
\end{itemize}

\chemfig{X-C(-[2]X)(-[6]X)-C(-[2]X)(-[6]X)-C(-[7]OH)=[1]O}
\begin{enumerate}
\item Acylglycines
\label{sec:org056538c}
\begin{itemize}
\item acylglycines are also detected in UOA analysis
\begin{itemize}
\item conjugation of acyl-CoA species to glycine
\item catalysed by glycine N-acylase
\end{itemize}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org3f3154b}
Organic Acid Nomenclature}
\centering
\begin{tabular}{lll}
Length & Monocarboxylic acid & Dicarboxylic acid\\
\hline
C2 & Acetic & Oxalic\\
C3 & Propionic & Malonic\\
C4 & Butyric & Succinic\\
 & Isobutyric & \\
C5 & Valeric & Glutaric\\
 & Isovaleric & \\
 & 2-Methylbutyric & \\
C6 & Hexanoic (caprioc) & Adipic\\
C7 & Heptanoic (enanthic) & Pimelic\\
C8 & Octanoic (caprylic) & Suberic\\
C9 & Nonanoic (pelargonic) & Azelaic\\
C10 & Decanoic (capric) & Sebacic\\
\end{tabular}
\end{table}
\end{enumerate}

\item Functional Groups
\label{sec:org7e69740}

\chemfig{X-C(-[2]X)(-[6]X)-C(-[2]X)(-[6]X)-C(-[7]OH)=[1]O}

\begin{table}[htbp]
\caption{\label{tab:org2cc2b9b}
Organic Acid Functional Groups}
\centering
\begin{tabular}{ll}
Functional group & Formula\\
\hline
hydrogen & -H\\
keto & .= O\\
hydroxyl & -OH\\
carboxyl & -COOH\\
side chain & -(CH\(_2\))\(_n\)\\
\end{tabular}
\end{table}

\item Side Chains
\label{sec:org420b828}

\chemfig{X-C(-[2]X)(-[6]X)-C(-[2]X)(-[6]X)-C(-[7]OH)=[1]O}

\begin{table}[htbp]
\caption{\label{tab:orgcd4a765}
Organic Acid Side Chains}
\centering
\begin{tabular}{ll}
Side chain & Structure\\
\hline
Methyl & \chemfig{CH_3-}\\
Ethyl & \chemfig{CH_3-CH_2-}\\
Propyl & \chemfig{CH_3-CH_2-CH_2-}\\
Butyl & \chemfig{CH_3-CH_2-CH_2-CH_2-}\\
\end{tabular}
\end{table}

\item Where do they come from?
\label{sec:org27d8755}
\begin{enumerate}
\item Endogenous Sources
\label{sec:orgf1febaa}
\begin{itemize}
\item originate from the intermediate metabolism of all major groups of
organic cellular components
\begin{itemize}
\item amino acids
\item lipids
\item nucleotides
\item carbohydrates
\item nucleic acids
\item steroids
\end{itemize}
\end{itemize}

\item Exogenous
\label{sec:orgef744d4}
\begin{itemize}
\item food
\item environment
\item medications
\end{itemize}
\end{enumerate}

\item Urine organic acids detected in health
\label{sec:org81f1864}
\begin{itemize}
\item tricarboxylic acid cycle acids
\begin{itemize}
\item citric
\end{itemize}
\item hydroxyaliphatic acids
\begin{itemize}
\item 3-hydroxybutyric
\end{itemize}
\item aliphatic keto acids
\begin{itemize}
\item pyrvic
\end{itemize}
\item aliphatic acids
\begin{itemize}
\item oxalic
\end{itemize}
\item aldonic and deoxyaldonic acids (sugar acids)
\item aromatic acids
\begin{itemize}
\item hippuric
\end{itemize}
\end{itemize}

\item Abnormal Urine Organic acids profiles
\label{sec:org7ab8def}
\begin{itemize}
\item elevated concentration of normal metabolites
\begin{itemize}
\item fumaric acid in fumarase deficiency
\item adipic, suberic, and sebacic acids in MCADD
\item ketones in fasting
\begin{itemize}
\item 3-hydroxybutyric
\item acetoacetic
\end{itemize}
\end{itemize}

\item pathological metabolites
\begin{itemize}
\item succinylacetone, methylcitric acid
\end{itemize}

\item food, medications, environment
\begin{itemize}
\item ethosuximide
\item adipic acid
\item cresol
\item 2-furaldehyde
\end{itemize}
\end{itemize}
\end{enumerate}

\subsection{Urine Organic Acids by GC-MS}
\label{sec:org8e31517}
\begin{enumerate}
\item Oximation
\label{sec:orge77d06d}
\begin{itemize}
\item oximated with 10\% hydroxylamine-HCL
\begin{itemize}
\item avoids multiple TMS species due to keto-enol tautomerism
\end{itemize}
\end{itemize}

\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.5]{R=[1](-[2]OH)-[7]R}}{\tiny enol}
\arrow{<=>}
\chemname{\chemfig[][scale=.5]{R-[1](=[2]O)-[7]R}}{\tiny ketone}
\+
\chemname{\chemfig[][scale=.5]{N(<:[::-160]H)(<[::-120]H)-O-[1]H}}{\tiny hydroxylamine}
\arrow{->}
\chemname{\chemfig[][scale=.5]{R-[1](=[2]N-[1]OH)-[7]R}}{\tiny ketoxime}
\schemestop
\item BSTFA Derivatisation
\label{sec:org0bc8c71}
\begin{itemize}
\item acidified and extracted twice with ethyl ether
\item derivatised with BSTFA (N,O-bis(trimethylsilyl)trifluoroacetamide) \footnote{Stalling DL, Gehrke CW, Zumwalt RW. A new silylation
reagent for amino acids bis(trimethylsilyl)trifluoroacetamide
(BSTFA). Biochemical and Biophysical Research Communications. 1968 May
23;31(4):616-22.}
\begin{itemize}
\item forms organic acid TMS esters
\end{itemize}
\end{itemize}

\schemedebug{false}
\schemestart
\chemname{\chemfig[][scale=.5]{F{_3}C-C(-[1]OTMS)=[7]NTMS}}{\tiny BSTFA}
\+
\chemname{\chemfig[][scale=.5]{R-C(=[1]O)-[7]OH}}{\tiny carboxylic acid}
\arrow{->}
\chemname{\chemfig[][scale=.5]{R-C(=[1]O)-[7]OTMS}}{\tiny TMS ester}
\+
\chemname{\chemfig[][scale=.5]{F{_3}C-C(=[1]O)-[7]NTMS}}{\tiny TMS amide}
\schemestop

\item Gas Chromatography
\label{sec:org71889b6}

\item Mass-spectroscopy
\label{sec:org38c965c}

\item Reporting
\label{sec:org791ac04}
\end{enumerate}
\section{Mitochondria}
\label{sec:org73dbb8b}
\subsection{Introduction}
\label{sec:org90cd945}
\begin{figure}[htbp]
\centering
\includegraphics[width=\textwidth]{./mito/figures/etc.pdf}
\caption{\label{fig:orgc98fba3}
Electron Transport Chain}
\end{figure}

\begin{enumerate}
\item Inhibitors
\label{sec:org61cbf61}
\begin{enumerate}
\item CI
\label{sec:orge2860ee}
\begin{itemize}
\item the best-known inhibitor of complex I is rotenone commonly used as
an organic pesticide
\begin{itemize}
\item rotenone binds to the ubiquinone binding site of complex I
\end{itemize}
\item piericidin A is a potent inhibitor and structural homologue to
ubiquinone
\item hydrophobic inhibitors like rotenone or piericidin likely disrupt
electron transfer between FeS cluster N2 and ubiquinone
\item bullatacin is the most potent known inhibitor of NADH dehydrogenase
(ubiquinone)
\item Complex I is also blocked by adenosine diphosphate
ribose
\begin{itemize}
\item a reversible competitive inhibitor of NADH oxidation
\end{itemize}
\end{itemize}

\item CII
\label{sec:org73cf3dc}
\begin{itemize}
\item there are two distinct classes of inhibitors of complex II:
\begin{itemize}
\item those that bind in the succinate pocket and those that bind in the ubiquinone pocket
\end{itemize}
\item ubiquinone type inhibitors include carboxin and thenoyltrifluoroacetone
\item succinate-analogue inhibitors include the synthetic compound malonate as well as the TCA cycle intermediates, malate and oxaloacetate
\begin{itemize}
\item oxaloacetate is one of the most potent inhibitors of Complex II
\end{itemize}
\end{itemize}
\item CIII
\label{sec:org6399307}
\begin{itemize}
\item there are three distinct groups of Complex III inhibitors:
\begin{itemize}
\item antimycin A binds to the Q\(_{\text{i}}\) site and inhibits the transfer of electrons in Complex III from heme b\(_{\text{H}}\) to oxidized Q (Q\(_{\text{i}}\) site inhibitor)
\item myxothiazol and stigmatellin bind to distinct but overlapping pockets within the Q\(_{\text{o}}\) site
\begin{itemize}
\item myxothiazol binds nearer to cytochrome bL (hence termed a "proximal" inhibitor)
\item stigmatellin binds farther from heme bL and nearer the Rieske Iron sulfur protein
\item both inhibit the transfer of electrons from reduced QH\(_{\text{2}}\) to the Rieske Iron sulfur protein
\end{itemize}
\end{itemize}
\end{itemize}

\item CIV
\label{sec:org157078f}
\begin{itemize}
\item cyanide, azide, and carbon monoxide all bind to cytochrome c
oxidase
\item nitric oxide and hydrogen sulfide, can also inhibit COX by
binding to regulatory sites on the enzyme
\end{itemize}
\item CV
\label{sec:org6d3189e}
\begin{itemize}
\item Oligomycin A inhibits ATP synthase by blocking its proton channel
(F\(_{\text{0}}\) subunit), which is necessary for oxidative phosphorylation of
ADP to ATP (energy production)
\item The inhibition of ATP synthesis by oligomycin A will significantly
reduce electron flow through the electron transport chain; however,
electron flow is not stopped completely due to a process known as
proton leak or mitochondrial uncoupling
\begin{itemize}
\item This process is due to facilitated diffusion of protons into the
mitochondrial matrix through an uncoupling protein such as
thermogenin, or UCP1
\end{itemize}

\item Administering oligomycin to an individual can result in very high
levels of lactate accumulating in the blood and urine
\end{itemize}
\end{enumerate}
\end{enumerate}

\subsection{Citrate Synthase}
\label{sec:org647bc81}
\subsection{CI+III assay}
\label{sec:orgeeae2ec}
\begin{enumerate}
\item Purpose
\label{sec:org1808145}
\begin{itemize}
\item determine the rate of cytochrome c reduction in mitochondria as a
result of electron transfer from NADH to cytochrome c (mitochondrial
complex I+III) activity
\item Complex I transfers electrons to ubiquinone (coenzyme Q10) through a
long series of redox groups
\item Complex III catalyzes electron transfer between ubiquinol and
cytochrome c and also translocates protons across the mitochondrial
inner membrane
\end{itemize}

\item Principle
\label{sec:org0787f96}
\begin{itemize}
\item reduced cytochrome c absorbs light at 550 nm
\item increase of the absorption at 550 nm corresponding to the increased
formation of reduced cytochrome c by electrons derived from NADH,
which is rotenone sensitive
\item azide is added to inhibit CIV so there is no re-oxidation of reduced cytochrome c
\item rotenone is added to the reference cuvette to inhibit Complex I
\begin{description}
\item[{assay cuvette}] oxidized cyt c \& azide
\item[{reference cuvette}] oxidized cyt c \& azide \& rotenone
\end{description}
\end{itemize}

{\small\ce{4Fe3+ cytochrome c + NADH + 2H2O ->[CI + CIII] 4Fe2+ cytochrome c + NAD+ + 4H + O2}}
\ce{oxidized cyt c -> reduced cyt c} 

\begin{itemize}
\item spectrophotometer subtracts the activity seen in the reference cell
from the activity seen in the assay cell, the progress curve you see
on the computer screen reflects activity of cytochrome c reduction
by electrons passing ONLY through complex I
\end{itemize}
\end{enumerate}

\subsection{CI assay}
\label{sec:org3084140}
\begin{enumerate}
\item Purpose
\label{sec:org326edf5}
\begin{itemize}
\item determining the rate of NADH oxidation in mitochondria as a result
of electron transfer from NADH to ubiquinone
\end{itemize}
\item Principle
\label{sec:org1b673ca}
\begin{itemize}
\item NADH absorbs light at 340 nm
\item the method follows the decrease of the absorption at 340 nm
corresponding to the decreased concentration of NADH, which has been
oxidized to NAD during the passage of electrons to ubiquinone
\item assay is rotenone sensitive
\item rotenone in the reference cuvette will specifically inhibit Complex
I therefore any oxidation of NADH from this cell does not include
the contribution of Complex I
\begin{description}
\item[{assay cuvette}] ubiquinone \&  antimycin A
\item[{reference cuvette}] ubiquinone \&  antimycin A \& rotenone
\end{description}
\end{itemize}

\ce{ubiquinone(CoQ) + NADH ->[CI] ubiquinol(CoQH2) + NAD+} 
\\
\ce{oxidized CoQ -> reduced CoQ} 

\begin{itemize}
\item spectrophotometer subtracts the activity seen in the reference cell
from the activity seen in the assay cell, the progress curve seen on
the computer screen reflects activity of NADH oxidation ONLY through
Complex I
\end{itemize}
\end{enumerate}

\subsection{CII assay}
\label{sec:org41965b6}
\begin{enumerate}
\item Purpose
\label{sec:orga0655fa}
\begin{itemize}
\item Complex II activity
\end{itemize}
\item Principle
\label{sec:org34b5923}
\begin{itemize}
\item secondary reduction of the dye 2,6-dichlorophenolindophenol (DCPIP)
by the ubiquinol formed at 600nm
\item DCPIP assays are very prone to interference from NAD(P)H
diaphorases
\item caution is recommended in interpreting results from non-muscle
tissue, rich in diaphorase
\begin{description}
\item[{assay cuvette}] succinate, ubiquinone \& DCPIP
\item[{reference cuvette}] ubiquinone \& DCPIP
\end{description}
\end{itemize}

\ce{oxidized DCPIP -> reduced DCPIP}
\end{enumerate}

\subsection{CII + III assay}
\label{sec:orge57c042}
\begin{enumerate}
\item Purpose
\label{sec:org724abb8}
\begin{itemize}
\item measure rate of SCR in mitochondria as a result of mitochondrial
respiratory complexes II and III activity
\item Complex II performs a key step in the citric acid cycle, in which
succinate is dehydrogenated to ubiquinone in the mitochondrial inner
membrane
\item Complex II is localized to the matrix side of the mitochondrial
inner membrane and it is the only respiratory chain enzyme of which
all 4 subunits are coded by the nuclear DNA
\item Complex III catalyzes electron transfer between ubiquinol and
cytochrome c and also translocates protons across the mitochondrial
inner membrane
\end{itemize}

\item Principle
\label{sec:org5022c2f}
\begin{itemize}
\item reduced cytochrome c absorbs light at 550 nm
\item the increase of the absorption at 550 nm corresponds to the
increased formation of reduced cytochrome c by electrons derived
from succinate
\begin{description}
\item[{assay cuvette}] sample, oxidized cyt c, azide \& succinate
\item[{reference cuvette}] oxidized cyt c, azide \& succinate
\end{description}
\end{itemize}

{\tiny\ce{4Fe3+ cytochrome c + succinate + 2H2O ->[CII + CIII] 4Fe2+ cytochrome c + fumarate + 4H + O2}}
\\
\ce{oxidized cyt c -> reduced cyt c}
\end{enumerate}

\subsection{CIV assay}
\label{sec:org1882f4e}
\begin{enumerate}
\item Purpose
\label{sec:orgb6ae69e}
\begin{itemize}
\item determine the rate of cytochrome C oxidation in mitochondria as a
result of cytochrome C oxidase (mitochondrial complex IV, COX)
activity
\item COX is a multisubunit assembly in the inner mitochondrial membrane
responsible for the terminal event in electron transport in which
molecular oxygen is reduced
\item various phenotypic forms of COX deficiency have been recognized, the
major varieties involving the degeneration of the brain stem and
basal ganglia (Leigh syndrome) and lactic acidemia with or without
cardiomyopathy
\end{itemize}

\item Principle
\label{sec:org45dbadc}
\begin{itemize}
\item reduced cytochrome c absorbs light at 550nm
\item methods follows the decrease in absorbance of reduced cytochrome c
at 550 nm
\begin{description}
\item[{assay cuvette}] sample \& reduced cyt c
\item[{reference cuvette}] reduced cyt c
\end{description}
\end{itemize}

{\small\ce{4Fe2+ cytochrome c + NAD+ + 4H + O2 ->[CI + CIII] 4Fe3+ cytochrome c + NADH + 2H2O}}
\ce{reduced cyt c -> oxidized cyt c}
\end{enumerate}

\subsection{CV assay}
\label{sec:orgfa19196}
\begin{enumerate}
\item Purpose
\label{sec:org1f604a1}
\begin{itemize}
\item determine the activity of respiratory chain complex V in isolated
muscle and fibroblast mitochondria
\end{itemize}

\item Principle
\label{sec:orgbae144b}
\begin{itemize}
\item ATP hydrolysis by ATPase liberates ADP which is reconverted to ATP
by the action of PK, thus maintaining a constant concentration of
ATP and a low steady state concentration of ADP
\item pyruvate production from PEP and ADP, catalysed by PK, is monitored
as a rate of oxidation of NADH by coupling to LDH
\item ATPase is oligomycyn sensitive
\begin{description}
\item[{assay cuvette}] LDH, PK, PEP \& rotenone
\item[{reference cuvette}] LDH, PK, PEP, rotenone \& oligomycin
\end{description}
\end{itemize}


\ce{ATP <->[ATPase] ADP}
\ce{PEP + ADP ->[PK] pyruvate}
\ce{pyruvate + NADH ->[LDH] lactate + NAD+}
\end{enumerate}
\section{{\bfseries\sffamily TODO} VLCFA}
\label{sec:org28a33f0}
\section{{\bfseries\sffamily TODO} GAGs and Oligosacarides}
\label{sec:orga9224a4}
\section{Enzymes}
\label{sec:orgf805637}
\subsection{GALT}
\label{sec:orgc045cc2}
\begin{enumerate}
\item Beutler Test
\label{sec:org1379ade}

\ce{Gal-1-P + UDP-Glu ->[GALT] Glu-1-P + UDP-Gal}

\ce{Glu-1-P ->[PGM] Glu-6-P}

\ce{Glu-6-P + NADP+ ->[G6PD] 6-glucuranate + NADPH}

\ce{6-glucuranate + NADP+ ->[6PGDH] ribulose-5-P + NADPH}

\begin{itemize}
\item fluorescence of NADPH is measured to determine GALT deficiency
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./enzymes/figures/beutler.jpg}
\caption{\label{fig:orgc821ecc}
Beutler Method}
\end{figure}


\item Spotcheck
\label{sec:org98237a1}

\ce{Gal-1-P + UDP-Glu ->[GALT] Glu-1-P + UDP-Gal}

\ce{Glu-1-P ->[PGM] Glu-6-P}

\ce{Glu-6-P + NADP+ ->[G6PD] 6-PG + NADPH}

\ce{NADPH + MTT ->[methoxy PMS] Coloured Formazan + NADP+}
\end{enumerate}

\subsection{Biotinidase}
\label{sec:org909a353}

\ce{biotin-PAB ->[BTD][ph 6] biotin + PABA}

\ce{PABA ->[NO2, NH2SO3][NED] purple chromophore}

\subsection{Lysosomal Enzymes}
\label{sec:orgf1973f6}
\begin{enumerate}
\item Multiplex DBS Assay
\label{sec:org6f067ff}
\begin{itemize}
\item the DBS screening assay tests for:
\begin{itemize}
\item Gaucher
\item Krabbe
\item Niemann-Pick-A/B
\item Pompe
\item Fabry
\item MPS-I
\end{itemize}
\item a single 3-mm DBS punch, which is incubated in a single-assay
cocktail with all substrates and internal standards
\item after incubation and liquid-liquid extraction, samples are analyzed by flow injection MS/MS
\item all deuterated internal standards correspond to enzymatically generated products
\end{itemize}


\item Hexosamindase
\label{sec:org884473b}
\begin{itemize}
\item testing for Tay-Sachs and Sandhoff diseases occurs by analysis of hexosaminidase A, a heat-labile enzyme, and total hexosaminidase (hexosaminidase A plus hexosaminidase B)
\item an artificial substrate is most commonly used
\item total hexosaminidase is quantified
\begin{itemize}
\item following this, heat inactivation of hexosaminidase A occurs with a second measurement of the total enzyme level
\item from this, the percent hexosaminidase A is calculated
\end{itemize}
\item Tay-Sachs disease is characterized by normal total hexosaminidase with a very low percent hexosaminidase A
\item carriers of Tay-Sachs disease are asymptomatic, but have intermediate percent hexosaminidase A in serum and leukocytes
\item follow-up molecular testing is recommended for all individuals with
enzyme results in the carrier or possible carrier ranges to
differentiate carriers of a pseudodeficiency allele from those with
a disease-causing mutation
\item this allows for the facilitation of prenatal diagnosis for at-risk
pregnancies
\end{itemize}

\begin{center}
\begin{tabular}{lllll}
Disorder & Enzyme & Total Hexosamindase & Hex A & Percent\\
\hline
Tay-Sachs & HexA & N & \(\Downarrow\) & \(\Downarrow\)\\
TS carrier & Hex & \dowarrow & \(\downarrow\) & \(\downarrow\)\\
Sandhoff & HexA/B & \(\Downarrow\) & \(\Downarrow\) & \(\uparrow\)\\
\end{tabular}
\end{center}


\begin{enumerate}
\item B1 Variant
\label{sec:orgd799c7b}
\begin{itemize}
\item very small group of patients affected with Tay-Sachs disease have
mutations referred to as the B1 variant

\item in the presence of an artificial substrate, the B1 variant allows
for a heterodimer formation of hexosaminidase A and exhibits
activity

\begin{itemize}
\item in vivo the B1 variant hexosaminidase A is inactive on the natural
substrate

\item with the artificial substrate, these patients appear to be
unaffected.

\item B1 variant of Tay-Sachs disease must be distinguished using a
natural substrate assay.

\item patients with at least one B1 variant typically become symptomatic
beyond the infantile period
\end{itemize}
\end{itemize}
\end{enumerate}
\end{enumerate}
\section{Glycosylation}
\label{sec:org065ebfc}
\subsection{Transferrin IEF}
\label{sec:orge5eb20a}
\begin{itemize}
\item serum transferrin IEF is the screening method of choice
\begin{itemize}
\item can detect nearly all known CDG-I types as well as most CDG-II types and many CDG-X cases
\item N-glycosylation disorders associated with sialic acid deficiency
\end{itemize}
\item normal serum transferrin is mainly composed of:
\begin{itemize}
\item tetrasialotransferrin and small amounts of mono, di, tri, penta
and hex-asialotransferrins
\end{itemize}
\item partial deficiency of sialic acid (-ve charge) causes a
cathodal shift
\item two main types of cathodal shift can be recognized:
\begin{itemize}
\item type 1 or 2 patterns
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=1\textwidth]{./glyc/figures/trans1or2.png}
\caption{\label{fig:org51c7843}
Transferrin IEF}
\end{figure}

\begin{enumerate}
\item Type 1 pattern
\label{sec:org48d5556}
\begin{itemize}
\item \(\uparrow\) disialo- and asialotransferrin
\item \(\downarrow\)  tetra-, penta-and hexasialotransferrins
\item defects in the assembly of the dolichol lipid-linked
oligosaccharide chain and transfer to the nascent protein
\item PMM2-CDG or MPI-CDG should be considered first
\item also seen in secondary glycosylation disorders such as:
\begin{itemize}
\item chronic alcoholism, hereditary fructose intolerance and galactosaemia
\end{itemize}
\end{itemize}

\item Type 2 pattern
\label{sec:org666bb40}
\begin{itemize}
\item Type 1 pattern with additional \(\uparrow\) tri- \textpm{}
monosialotransferrin bands
\item defects in the trimming and processing of the protein-bound
glycans either late in the endoplasmic reticulum or the Golgi
compartments
\end{itemize}

\item Transferrin IEF limitations
\label{sec:orgf466f7d}
\begin{itemize}
\item deficiencies of ER-glucosidase I (CDG-IIb) and Golgi GDP-fucose
transporter (CDG-IIc) are missed
\item prenatal diagnostics by IEF analysis from fetal blood is not
reliable
\item IEF of serum from children \textless{} 2 weeks may be false-positive
\item heavy alcohol consumption can also result in serum transferrin
deficiency in carbohydrate moieties, leading to an abnormal
IEF-pattern
\item mutations in the protein backbone of transferrin
\begin{itemize}
\item desialylation of transferrin by neuraminidase treatment or IEF of
an alternative glycoprotein like \(\alpha\) 1-antitrypsin should be
performed
\end{itemize}
\item galactosemia and HFI may cause false positive patterns
\end{itemize}
\end{enumerate}
\subsection{Additional Laboratory Investigations}
\label{sec:org6e3c5e0}
\begin{itemize}
\item protein-linked glycan analysis can be performed to identify the defective step
\begin{itemize}
\item MALDI-TOF analysis of released N-linked oligosaccharides
\end{itemize}
\item CDG gene panel analysis or WES

\item capillary zone electrophoresis of total serum is a rapid screening
test for CDG
\begin{itemize}
\item An abnormal result should be further investigated by serum
transferrin IEF
\end{itemize}
\item HPLC-UV/Vis
\end{itemize}
\section{NIET}
\label{sec:org0e3a01a}
\subsection{Exercising Muscle}
\label{sec:org8586d7b}
\begin{enumerate}
\item Lactate
\label{sec:orgc099696}
\begin{itemize}
\item lactate, ammonia and purine compounds are generated by exercising muscle
\item exercising muscle generates lactic acid from the anaerobic breakdown
of glycogen to pyruvate
\begin{itemize}
\item pyruvate \(\to\) lactate
\end{itemize}
\item lactate enters the circulation and is converted back to pyruvate in the liver
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.4\textwidth]{./niet/figures/Lactate_dehydrogenase_mechanism.png}
\caption{\label{fig:org034a0de}
LDH}
\end{figure}

\item ATP
\label{sec:org045f705}
\begin{itemize}
\item some ATP regeneration is provided by glycolytic metabolism of fuels,
but this is relatively slow
\item most ATP regeneration relys on creatine kinase catalysed transfer of
phosphate from phosphocreatine

\begin{itemize}
\item \ce{phosphocreatine + ADP ->[CK] creatine + ATP}
\end{itemize}

\item adenylatekinase transphosphorylates ATP to be regenerated with the formation
of AMP

\begin{itemize}
\item \ce{2ADP ->[ADK] ATP + AMP}
\end{itemize}

\item AMP deaminase
\begin{itemize}
\item \ce{AMP ->[AMPD] IMP + NH4+}
\end{itemize}

\item IMP degraded to hypoxanthine
\item recycled back to AMP in the purine nucleotide cycle
\end{itemize}

\item Ammonia
\label{sec:org08d2905}
\begin{itemize}
\item most ammonia produced by exercising muscle removed by formation of glutamine
\begin{itemize}
\item ultimately excreted as urea
\end{itemize}
\item some ammonia is released by exercising skeletal muscle directly into the circulation
\begin{itemize}
\item removed with a half-life of 20\textpm{}30 min
\end{itemize}
\item in resting skeletal muscle ammonia is consumed rather than produced
\item \textasciitilde{}50\% of arterial ammonia can be taken up and metabolized by skeletal muscle
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.6\textwidth]{./niet/figures/nitrogen_glutamine.png}
\caption[gln]{\label{fig:orgb0eee13}
Glutamine and Ammonia}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=.6\textheight]{./niet/figures/niet_results.png}
\caption[interp]{\label{fig:org0789710}
NIET Results}
\end{figure}
\end{enumerate}


\subsection{The non-ischemic forearm exercise test}
\label{sec:orge5e2781}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./niet/figures/niet_method.png}
\caption{\label{fig:org6e80666}
NIET Method}
\end{figure}


\begin{table}[htbp]
\caption{\label{tab:org431c38f}
NIET in Myopathy}
\centering
\begin{tabular}{lll}
 & Lactate & Ammonia\\
\hline
GSD I & N & N\\
GSD III (L\&M) & \(\downarrow\) \(\downarrow\) & N/\(\uparrow\)\\
GSD V & \(\downarrow\) \(\downarrow\) & N/\(\uparrow\)\\
\textbf{GSD VII (PFK)} & \(\downarrow\) \(\downarrow\) & N/\(\uparrow\)\\
\textbf{GSD IX (PGK)} & \(\downarrow\) \(\downarrow\) & N/\(\uparrow\)\\
\textbf{GSD X (PGAM)} & \(\downarrow\) & N/\(\uparrow\)\\
Alcoholic myopathy & N & N\\
CFS & N & N\\
Poor effort & N/\(\downarrow\) & N/\(\downarrow\)\\
\end{tabular}
\end{table}
\section{Porphyrins}
\label{sec:orgcd4b7b6}
\subsection{Methods for Metabolites}
\label{sec:org63b6036}
\begin{enumerate}
\item ALA
\item PBG
\item Urinary Porphyrins
\item Fecal Porphyrins
\item Blood Porphyrins
\end{enumerate}
\begin{enumerate}
\item Specimen Collection and Stability
\label{sec:orga46d2a2}
\begin{itemize}
\item Protect from light
\item Urinary porphyrins and PBG best collected in fresh random urine
without preservative. Very dilute urine (creatinine <2 mmol/L) is
not suitable
\item 24 hour urine offer little advantage
\begin{enumerate}
\item delay diagnosis
\item increased risk of degradation
\end{enumerate}
\end{itemize}
\item Methods for Porphyrin Precursors
\label{sec:org14ab5de}
\begin{enumerate}
\item Porphobilinogen
\label{sec:org004ad92}
\begin{itemize}
\item Ehrlich's reagent
\begin{itemize}
\item urobilinogen inhibition, therefore ion-exchange
\end{itemize}
\end{itemize}
\item 5-Aminolevulinic Acid
\label{sec:org265af1b}
\begin{itemize}
\item Converted to Ehrlich's reacting pyrrole via condensation with
acetylacetone
\end{itemize}
\item Analysis of Porphyrins in Urine and Feces
\label{sec:orga2b35b1}
\begin{itemize}
\item Screening via spectrophotometric scanning of acidified urine or
fecal extracts for the Soret band
\end{itemize}
\item Semiquantitative Method for Total Porphyrin in Urine and Feces
\label{sec:orgff7aa9c}
\begin{itemize}
\item we do this as a screen
\end{itemize}
\item HPLC Fractionation of Porphyrins in Urine and Feces
\label{sec:org4c6b5ab}
\begin{itemize}
\item fluormetric detection recommended
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./porphyrins/figures/urine.pdf}
\caption{\label{fig:orgb263e83}
Urine Porphyrins}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./porphyrins/figures/fecal.pdf}
\caption{\label{fig:orga096ab9}
Fecal Porphyrins}
\end{figure}
\end{enumerate}

\item Methods for Blood Porphyrins
\label{sec:org748b3cf}
\begin{enumerate}
\item Determination of Erythrocyte Total Porphyrin
\label{sec:orgdaaaa4c}
\begin{itemize}
\item increased in:
\begin{itemize}
\item EPP
\item CEP
\item homozygous variants
\item iron deficiency
\item hemolytic anemia
\item some other forms of anemia
\item lead poisoning
\end{itemize}
\item total porphyrin concentration within RI excludes EPP
\begin{itemize}
\item distinction between EPP and other causes requires diffentiation
between protoporphyrin and ZN-protoporphyrin
\end{itemize}
\end{itemize}
\item Qualitative Determination of ZN-protoporphyrin and Protoporphyrin
\label{sec:org46932ce}
\begin{itemize}
\item Emission peaks for ZPP = 587 nm and free protoporphyrin = 630 nm
\item In EPP the concentration of free protoporphyrin >> ZPP
\begin{itemize}
\item may be 60\% of total porphyrin
\end{itemize}
\item Lead poisoning, iron deficiency and other anemias ZPP is the main
component.
\item Limitation of method is that extraction of ZPP is \textasciitilde{}50\%
\end{itemize}
\end{enumerate}

\item Analysis of Plasma Porphyrins
\label{sec:org727c5d6}
\begin{enumerate}
\item Fluorescence emision Spectroscopy of Plasma Porphyrins
\label{sec:orgdae378c}
\begin{itemize}
\item Emission spectra of saline diluted plasma excited at 405 nm
\item In VP the plasma contains porphyrin covalently bound to protein with
E\(_{\text{max}}\) at 624 to 628 nm
\item Other porphyrias contain porphyrin non-covalently bound to albumin
and hemopexin.
\begin{itemize}
\item Fresh plasma protoporphyrin E\(_{\text{max}}\) = 632 nm
\item Older sample: binding to globulin released from red cells E\(_{\text{max}}\) =
626 nm
\end{itemize}
\item 2\degree causes of increase include: impaired excretion, renal
failure, cholestasis
\end{itemize}
\end{enumerate}
\end{enumerate}

\subsection{Enzyme Measurements}
\label{sec:orga99ea85}
\begin{itemize}
\item rarely required for patients with symptoms
\item can be used for family studies, DNA is better
\end{itemize}
\begin{enumerate}
\item Assay of Etythrocyte Hydroxymethylbilane Synthase Activity
\label{sec:org96851c3}
\begin{itemize}
\item Measures rate of formation of porphyrinogens from PBG by hemolysed erythrocytes
\item Discriminates between AIP and unaffected relatives
\item Interferences:
\begin{itemize}
\item HMBS activity declines sharply with erythrocyte age
\item affected by \(\propto\) of retics, and young cells in peripheral blood
\item \(\uparrow\) in acute illness, ie acute porphyria
\item \(\sim\) 1:800 low HMBS activity in France
\end{itemize}
\end{itemize}
\end{enumerate}
\end{document}