#+TITLE: Thompson & Thompson
#+AUTHOR: Matthew Henderson, PhD, FCACB
#+DATE: \today

:PROPERTIES:
#+DRAWERS: PROPERTIES
#+LaTeX_CLASS: koma-article
#+OPTIONS: H:2 toc:1 num:t ^:t
#+PROPERTY: header-args:R :session *R*
#+PROPERTY: header-args :cache no
#+PROPERTY: header-args :tangle yes
#+STARTUP: overview
#+STARTUP: indent
#+SEQ_TODO: TODO(t) STARTED(s) | DONE(d) 
#+latex_header: \hypersetup{colorlinks,linkcolor=black,urlcolor=blue}
#+LaTeX_header: \usepackage{textpos}
#+LaTeX_header: \usepackage{textgreek}
#+LaTeX_header: \usepackage[version=4]{mhchem}
#+LaTeX_header: \usepackage{chemfig}
#+LaTeX_header: \usepackage{siunitx}
#+LaTeX_header: \usepackage{gensymb}
#+LaTex_HEADER: \usepackage[usenames,dvipsnames]{xcolor}
#+LaTeX_HEADER: \usepackage[T1]{fontenc}
#+LaTeX_HEADER: \usepackage{lmodern}
#+LaTeX_HEADER: \usepackage{verbatim}
#+LaTeX_HEADER: \usepackage{tikz}
#+LaTeX_HEADER: \usepackage{wasysym}
#+LaTeX_HEADER: \usetikzlibrary{shapes.geometric,arrows,decorations.pathmorphing,backgrounds,positioning,fit,petri}
#+LaTeX_HEADER: \usepackage{fancyhdr}
#+LaTeX_HEADER: \pagestyle{fancy}
:END:

* Chapter 2: Human Genome
** Mitosis
- Prophase :: condensation of the chromosomes
  - formation of the mitotic spindle,
  - formation of a pair of centrosomes, from which microtubules
    radiate and eventually take up positions at the poles of the cell.
- Prometaphase :: nuclear membrane dissolves
  - chromosomes disperse within the cell
  - attach, by their kinetochores, to microtubules of the mitotic
   spindle.
- Metaphase :: the chromosomes condensed and line up at the equatorial
               plane.
- Anaphase ::  chromosomes separate at the centromere
  - sister chromatids of each chromosome now become independent
    daughter chromosomes, which move to opposite poles of the cell.
- Telophase :: chromosomes begin to decondense
  - nuclear membrane begins to re-form around each of the two daughter
    nuclei
  - cytoplasm cleaves = cytokinesis

#+CAPTION[]:Mitosis
#+NAME: fig:mitosis
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch2_mitosis.png]]


#+CAPTION[]:Karyotype
#+NAME: fig:karyotype
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch2_karyotype.png]]

** Meiosis

- Meiosis I is also known as the reduction division because it is the
  division in which the chromosome number is reduced by half through
  the pairing of homologues in prophase and by their segregation to
  different cells at anaphase of meiosis I.
- the stage at which genetic recombination (also called meiotic crossing over) occurs.
  - homologous segments of DNA are exchanged between non-sister
    chromatids of each pair of homologous chromosomes, thus ensuring
    that none of the gametes produced by meiosis will be identical to
    another.
- Theconceptual and practical consequences of recombination for many
aspects of human gen

#+CAPTION[]:Meiosis
#+NAME: fig:meiosis
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch2_meiosis.png]]


** Homologous Recombination

- the genetic content of each gamete is unique, because of random
  assortment of the parental chromosomes to shuffle the combination of
  sequence variants between chromosomes and because of homologous
  recombination to shuffle the combination of sequence variants within
  each and every chromosome.
- This has significant consequences for patterns of genomic variation
  among and between different populations around the globe and for
  diagnosis and counseling of many common conditions with complex patterns of inheritance.
- The amounts and patterns of meiotic recombination are determined by
  sequence variants in specific genes and at specific “hot spots” and
  differ between individuals, between the sexes, between families, and
  between populations
- Because recombination involves the physical intertwining of the two
  homologues until the appropriate point during meiosis I, it is also
  critical for ensuring proper chromosome segregation during
  meiosis.
- Failure to recombine properly can lead to chromosome missegregation
  (nondisjunction) in meiosis I and is a frequent cause of pregnancy
  loss and of chromosome abnormalities like Down syndrome
- Although homologous recombination is normally precise, areas of
  repetitive DNA in the genome and genes of variable copy number in
  the population are prone to occasional unequal crossing over during
  meiosis, leading to variations in clinically relevant traits such as
  drug response, to common disorders such as the thalassemias or
  autism, or to abnormalities of sexual differentiation
- Although homologous recombination is a normal and essential part of
  meiosis, it also occurs, albeit more rarely, in somatic
  cells.
- Anomalies in somatic recombination are one of the causes of genome
  instability in cancer

* Chapter 3: Gene Structure and Function
** Allelic Imbalance in Gene Expression
- Monoallelic gene expression
  - Somatic rearrangement - T-cell receptors
  - random monoallelic expression
  - parent-of-origin imprinting
- X chromosome inactivation
  - random, X inactivation center
  - ncRNA called XIST

* Chapter 4: Human Genetic Diversity
** Inherited variation and polymorphism
- SNP
- Indels
- microsatelite
- mobile element insertion polymorphism
  - retrotranspostion: Alu, LINE
- CNVs
  - related to indels and microsatellites but variation in
    the number of copies of larger segments of the genome
  - 1000 bp to many hundreds of kilobase pairs.
- Inversion polymorphism
  - few base pairs up to several megabase pairs
  - can be present in either of two orientations in the genomes of different individuals
#+CAPTION[]:Polymorphism
#+NAME: fig:poly
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch4_polymorphism.png]]


** Origins and Freq of Mutation types
- germline
- somatic
- Mutations:
  - Chromosome number
  - Regional: affecting the structure or regional organization of chromosomes
  - Gene: base pair substitutions, insertions, and deletions

#+CAPTION[]:Mutation
#+NAME: fig:mutation
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch4_mutation.png]]

* Chapter 5: Cytogenetics
** Clinical Indications for Chromosome and Genome Analysis
*** Problems of early growth and development
  - failure to thrive, developmental delay,dysmorphic facies, multiple
    malformations, short stature, ambiguous genitalia, and
    intellectual disability are frequent findings in children with
    chromosome abnormalities
*** Stillbirth and neonatal death
- incidence of chromosome abnormalities is much higher among
  stillbirths (\sim10%) than among live births (\sim0.7%)
- also elevated among infants who die in the neonatal period (~10%)
- karyotyping (or other comprehensive ways of scanning the genome) is
  essential for accurate genetic counseling
*** Fertility problems
- chromosome studies are indicated for women presenting with
  amenorrhea and for couples with a history of infertility or recurrent miscarriage
- chromosome abnormality is seen in one or the other parent in 3% to
  6% of cases in which there is infertility or two or more
  miscarriages
*** Family history
- known or suspected chromosome or genome abnormality in a first
  degree relative is an indication for chromosome and genome analysis
*** Neoplasia
- virtually all cancers are associated with one or more chromosome
  abnormalities
- chromosome and genome evaluation in the tumor itself, or in bone
  marrow in the case of hematological malignant neoplasms, can offer
  diagnostic or prognostic information
*** Pregnancy
- a higher risk for chromosome abnormality in fetuses conceived by
  women of increased age, typically defined as \gt 35 years
- fetal chromosome and genome analysis should be offered as a routine
  part of prenatal care in such pregnancies
- NIPT is a screening approach for the most common chromosome
  disorders and is now available to pregnant women of all ages

** Chromosome Identification
- G-banding (Giemsa) is the gold standard for detection and
  characterization of structural and numerical genomic abnormalities
  - both constitutional (postnatal or prenatal) and acquired (cancer)
  - detection of deletions and duplications \gt 5 to 10 Mb 
than approximately 5 to 10 Mb anywhere in the genome
- three types of chromosomes:
  - metacentric :: central centromere
  - submetacentric :: off center centromere
  - acrocentric :: centromere at one end
    - 13,14,15,21,22

** Fluorescence In Situ Hybridization
- detecting the presence or absence of a particular DNA sequence or
  for evaluating the number or organization of a chromosome or
  chromosomal region /in situ/
- uses ordered collections of recombinant DNA clones containing DNA
  from around the entire genome
- limited by the need to target a specific genomic region based
on a clinical diagnosis or suspicion

** Microarrays
- comparative genome hybridization (CGH)
  - detects relative copy number gains and losses genome-wide by
    hybridizing two samples:
    - control genome
    - patient
  - excess of sequences from one or the other genome indicates an
    overrepresentation or underrepresentation of those sequences in the
    patient genome relative to the control
- SNP arrays
  - relative representation and intensity of alleles in different
    regions of the genome indicate if a chromosome or chromosomal
    region is present at the appropriate dosage
- probe spacing provides a resolution as high as 250 kb
** Chromosome Abnormalities
- numerical or structural
- incidence of 1/154 live births
- aneuploidy is most common
  - associated with physical and/or mental abnormalities
- structural abnormalities are also common

#+CAPTION[]:Incidence of chromosomal abnormalities
#+NAME: fig:freq
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_freq.png]]

#+CAPTION[]:ISCN for common cytogenetic aberration
#+NAME: fig:nom
#+ATTR_LaTeX: :width 1.2\textwidth
[[file:./figures/ch5_nom.png]]

#+CAPTION[]:ISCN (continued)
#+NAME: fig:nom
#+ATTR_LaTeX: :width 1.2\textwidth
[[file:./figures/ch5_nom2.png]]

** Gene Dosage, Balance and Imbalance
- for chromosome and genomic disorders, it is the quantitative aspects
  of gene expression that underlie disease, in contrast to single-gene
  disorders, in which pathogenesis often reflects qualitative aspects
  of a gene's function
*** monosomies are more deleterious than trisomies
- complete monosomies are generally not viable, except for monosomy
  for the X chromosome
- complete trisomies are viable for chromosomes 13, 18, 21, X, and Y

*** phenotype in partial aneuploidy depends on a number of factors
- size of the unbalanced segment
- which regions of the genome are affected
- which genes are involved
- whether the imbalance is monosomic or trisomic
*** risk in cases of inversions depends on the location of the inversion with respect to the centromere and on the size of the inverted segment
- paracentric inversions do not involve the centromere
  - very low risk for an abnormal phenotype in the next generation
- pericentric inversions do involve the centromere
  - risk for birth defects in offspring may be significant and
    increases with the size of the inverted segment

*** mosaic karyotype involving any chromosome abnormality, all bets are off!
- the degree of mosaicism in relevant tissues or relevant stages of
  development is generally unknown
- there is uncertainty about the severity of the phenotype

** Abnormalities of Chromosome Number
- heteroploid :: chromosome complement other than 46 is 
- euploid :: exact multiple of n
- aneuploid :: non-multiple of n

*** Triploidy and tetraploidy
- most result from fertilization of an egg by two sperm (dispermy)
- also failure of one of the meiotic divisions in either sex,
  resulting in a diploid egg or sperm
- maternal source are aborted
- paternal source \to degenerative placenta (parital hydatidiform
  mole) w small fetus

*** Aneuploidy
- most common cause is meiotic nondisjunction in meiosis I or II (Figure [[fig:nondys]])
  - trisomy :: 21,18,13
  - monosomy :: X (Turner syndrome)

#+CAPTION[]:Nondisjunction
#+NAME: fig:nondys 
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_nondys.png]]

** Abnormalities of Chromosome Structure
- present in 1/375 newborns
- balances or unbalanced

#+CAPTION[]:Structural rearrangements of chromosomes
#+NAME: fig:str
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_struct.png]]

*** Unbalanced rearrangements
- Delections and Duplications
- Marker and Ring Chromosomes
  - very small, unidentified chromosomes
- Isochromosomes
  - one arm is missing and the other duplicated in a mirror-image
- Dicentric
  - two chromosome segments, each with a centromere, fuse end to end
*** Balanced rearrangements
- "balanced" depends on resolution
- Translocations
  - Reciprocal translocations
  - Robertsonian translocations
  - Insertions
- Inversions
  - paracentric - outside the centromere
  - pericentric - includes the centromere

#+CAPTION[]:Balanced translocations
#+NAME: fig:trans
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_trans.png]]

* STARTED Chapter 6: Chromosomal and Genomic basis of Disease

- Disorders due to:
  - abnormal chromosome segregation (nondisjunction)
  - recurrent chromosomal syndromes, involving
    deletions or duplications at genomic hot spots
  - idiopathic chromosomal abnormalities, typically de novo
  - unbalanced familial chromosomal abnormalities
  - chromosomal and genomic events that reveal regions
    of genomic imprinting

#+CAPTION[]:Mechanisms of chromosome abnormalities and genomic imbalance
#+NAME: fig:mech 
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch6_mech.png]]

** Lessons From Genomic Disorders
- altered gene dosage for any extensive chromosomal or genomic region
  is likely to result in a clinical abnormality, the phenotype of
  which will, in principle, reflect haploinsufficiency for or
  overexpression of one or more genes encoded within the region.
  - in some cases, the clinical presentation appears to be accounted
    for by dosage imbalance for just a single gene; in other
    syndromes, however, the phenotype appears to reflect imbalance for
    multiple genes across the region
- the distribution of these duplication/deletion disorders is not random,
  - segmental duplications in pericentromeric and subtelomeric
    regions, predisposes particular regions to the unequal
    recombination events that underlie these syndromes
- patients carrying what appears to be the same chromosomal deletion
  or duplication can present with a range of variable phenotypes

** Aneuploidy
- common mutation in our species involves errors in chromosome segregation
- only three well-defined nonmosaic chromosome disorders compatible
  with postnatal survival in which there is an abnormal dose of an
  entire autosome:
  1) trisomy 21 (Down syndrome)
  2) trisomy 18
  3) trisomy 13
- also have the \downarrow genes among autosomes

*** Down Syndrome
- see conditions
**** Robertsonian Translocation 
- \sim4% of Down syndrome patients have 46 chromosomes
- one of which is a Robertsonian translocation between chromosome
  21q and the long arm of one of the other acrocentric chromosomes
  (usually chromosome 14 or 22)
- 46,XX or XY,rob(14;21)(q10;q10),+21

#+CAPTION[]:Chromosomes of gametes that theoretically can be produced by a carrier of a Robertsonian translocation, rob(14;21)
#+NAME: fig:ft
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch6_rtgam.png]]

*** Uniparental Disomy
- nondisjunction \to both copies of a chromosome derive from the same
  parent
  - called uniparental disomy
  - defined as the presence of a disomic cell line containing two
    chromosomes, or portions thereof, that are inherited from only one
    parent
- isodisomy if  two chromosomes are derived from identical sister chromatids
- heterodisomy if if both homologues from one parent are present
- common explanation for uniparental disomy is trisomy “rescue” due to
  chromosome nondisjunction in cells of a trisomic conceptus to
  restore a disomic state

*** Contiguous Gene Syndrome
- segmental aneusomy is a form of genetic imbalance due to recurrent
  subchromosomal or regional abnormalities
  - typically detected by microarray
  - called contiguous gene syndrome

** Idiopathic Chromosome Abnormalities 
- Autosomal deletion syndromes
  - cri du chat syndrome, there is either a terminal or interstitial
    deletion of part of the short arm of chromosome 5
- Balanced translocations with developmental phenotypes
  
** Disorders Associated with Genomic Imprinting
- Prader-Willi
- Angelman syndrome
- Beckwith-Wiedemann syndrome

#+CAPTION[]:Mechanism causing Prader-Willi and Angelman Syndrome
#+NAME: fig:pw_as
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch6_pw_as.png]]

* TODO Chapter 7: Patterns of Single Gene Inheritance
* TODO Chapter 8: Complex Traits
* STARTED Chapter 9: Genetic Variation in Populations
** Hardy-Weinberg
- The Hardy-Weinberg law states that the frequency of the three
  genotypes AA, Aa, and aa is given by the terms of the binomial
  expansion of (p + q)^2 = p^2 + 2pq + q^2
- p = the frequency of the dominant allele (represented here by A)
- q = the frequency of the recessive allele (represented here by a)

- For a population in genetic equilibrium:
  - p + q = 1.0 (The sum of the frequencies of both alleles is 100%)
  - (p + q)^2 = 1
  - p^2 + 2pq + q^2 = 1
- The three terms of this binomial expansion indicate the frequencies
  of the three genotypes:
  - p2 = frequency of AA (homozygous dominant)
  - 2pq = frequency of Aa (heterozygous)
  - q2 = frequency of aa (homozygous recessive)

- The Hardy-Weinberg law rests on these assumptions:
  - The population under study is large, and matings are random with
    respect to the locus in question
  - Allele frequencies remain constant over time because of the
    following:
    - There is no appreciable rate of new mutation.• Individuals with
      all genotypes are equally capable of mating and passing on their
      genes; that is, there is no selection against any particular
      genotype
    - There has been no significant immigration of individuals from a
      population with allele frequencies very different from the
      endogenous population
- A population that reasonably appears to meet these assumptions is
  considered to be in Hardy-Weinberg equilibrium
- Then population genotype frequencies from generation to generation
  will remain constant, at equilibrium, if the allele frequencies p
  and q remain constant.

| Inheritance | Population Freq |                  |
|-------------+-----------------+------------------|
| AR          | q^2             | homozygote       |
| AD          | 2pq             | heterozygotes    |
| XL          | 1/2 \cdot q^2   | hemizygote males |

** Factors That Disturb Hardy-Weinberg Equilibrium
*** Exceptions to Large Populations with Random Mating
- stratification
- assortative mating
- consaguinity
*** Exceptions to Constant Allele Frequencies
- mutation
- selection and fitness
  - selection in dominant disease
  - selection in recessive disease - less effect
- Mutation and Selection Balance in Dominant Disease
  - \mu = sq
    - q = mutant allele freq
    - s = coefficient of selection
    - \mu = mutation rate per generation

*** Mutation and Selection Balance in X-Linked Recessive Mutations.
- males have one X chromosome and females two, the pool of X-linked
  alleles in the entire population’s gene pool is partitioned at any
  given time, with one third of mutant alleles present in males and
  two thirds in females
- if the incidence of a serious X-linked disease is not changing and
  selection is operating against (and only against) hemizygous males,
  the mutation rate then µ, must equal the coefficient of selection, s
  times q, the allele frequency, adjusted by a factor of 3 because
  selection is operating only on the third of the mutant alleles in
  the population that are present in males at any time. Thus,
  - \mu = sq/3


* TODO Chapter 10: Identifying the Genetic Basis for Human Disease
* TODO Chapter 11: The Molecular Basis of Genetic Disease
* TODO Chapter 12: Molecular, Biochemical, and Cellular Basis of Genetic Disease
* TODO Chapter 13: Treatment
* TODO Chapter 14: Developmental Genetics
* Chapter 15: Cancer
- cancer is not a single disease but rather comes in many forms and
  degrees of malignancy
- there are three main classes of cancer:
  - sarcomas :: tumor has arisen in mesenchymal tissue, such as bone,
                muscle, or connective tissue, or in nervous system
                tissue
  - carcinomas :: originate in epithelial tissue, such as the cells
                  lining the intestine, bronchi, or mammary ducts
  - hematopoietic and lymphoid :: leukemia and lymphoma which spread
       throughout the bone marrow, lymphatic system, and peripheral
       blood
** Genetic Basis of Cancer
- driver and passenger mutations
- particular translocations are sometimes highly specific for certain
  types of cancer and involve specific genes
  - BCR-ABL translocation in chronic myelogenous leukemia
- cancers can also show complex rearrangements in which chromosomes
  break into numerous pieces and rejoin, forming novel and complex
  combinations (a process known as “chromosome shattering”)
- large genomic alterations involving many kilobases of DNA can form
  the basis for loss of function or increased function of one or more
  driver genes

- genes in which mutations cause cancer are referred to as *driver*
  *genes*, and the cancer causing mutations in these genes are *driver*
  *mutations*
- driver genes fall into two distinct categories
  1) activated oncogenes
  2) tumor suppressor genes (TSGs)

- an activated oncogene is a mutant allele of a proto-oncogene, a
  class of normal cellular protein-coding genes that promotes growth
  and survival of cells
- oncogenes facilitate malignant transformation by stimulating
  proliferation or inhibiting apoptosis
- oncogenes encode proteins such as the following:
  - proteins in signaling pathways for cell proliferation
  - transcription factors that control the expression of growth-promoting genes
  - inhibitors of programmed cell death machinery
- A TSG is a gene in which loss of function through mutation or
  epigenomic silencing directly removes normal regulatory controls on
  cell growth or leads indirectly to such losses through an increased
  mutation rate or aberrant gene expression
- TSGs encode proteins involved in many aspects of cellular function including:
  - maintenance of correct chromosome number and structure
  - DNA repair proteins
  - proteins involved in regulating the cell cycle, cellular
    proliferation or contact inhibition

- tumor initiation can be caused by different types of genetic
  alterations:
  - activating or gain-of-function mutations
  - ectopic and heterochronic mutations of protooncogenes
  - chromosome translocations that cause misexpression of genes or chimeric genes
  - LOF of both alleles, or a dominant negative mutation of one allele, of TSGs

- tumor progression occurs as a result of accumulating additional
  genetic damage,through mutations or epigenetic silencing, of driver
  genes that encode the machinery that repairs damaged DNA and
  maintains cytogenetic normality

** Cancer in Families

- germline mutation - inherited
- second-hit
  - somatic mutation
  - loss of heterozygozity around locus


** Sporadic Cancer
- activation of oncogenes by point mutation
- activation of oncogenes by chromosome translocation
  - best-known example is the translocation between chromosomes 9 and
    22 (Philadelphia chromosome) that is seen in CML
  - moves the protooncogene ABL1, a tyrosine kinase, from its normal
    position on chromosome 9q to a gene of unknown function, BCR, on
    chromosome 22q
  - results in the synthesis of a novel, chimeric protein, BCR-ABL1 w
    enhanced tyrosine kinase activity


#+CAPTION[]: Characteristic Chromosome Translocations in Selected Human Malignant Neoplasms
#+NAME: fig:trans 
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch15_neo.png]]

** Cytogenetic Changes in Cancer

*** Aneuploidy and Aneusomy
- when CML, with the 9;22 Philadelphia chromosome, evolves from the
  typically indolent chronic phase to a severe, life-threatening blast
  crisis, there may be several additional cytogenetic abnormalities,
  including numerical or structural changes, such as a second copy of
  the 9;22 translocation chromosome or an isochromosome for 17q

- another cytogenetic aberration seen in many cancers is gene
  amplification, a phenomenon in which many additional copies of a
  segment of the genome are present in the cell
  - *double minutes* (very small accessory chromosomes)
  - *homogeneously staining regions* that do not band normally and
    contain multiple, amplified copies of a particular DNA segment
- known to include extra copies of proto-oncogenes such as the genes
  encoding Myc, Ras, and epithelial growth factor receptor, which
  stimulate cell growth, block apoptosis, or both

** Targeted Cancer Therapy
- The proof of principle for this approach was established with the
  development of imatinib, a highly effective inhibitor of a number of
  tyrosine kinases, including the ABL1 kinase in CML
- Prolonged remissions of this disease have been seen, in some cases
  with apparently indefinite postponement of the transformation into a
  virulent acute leukemia (blast crisis) that so often meant the end
  of a CML patient’s life
- Additional kinase inhibitors have been developed to target other
  activated oncogene driver genes in a variety of tumor types
- initial results with targeted therapies, although very promising in
  some cases, have not led to permanent cures in most patients because
  tumors develop resistance to the targeted therapy
- The outgrowth of resistant tumors because cancer cells are highly
  mutable, and their genomes undergo recurrent mutation


#+CAPTION[]:Targeted cancer treatment
#+NAME: fig:target
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch15_target.png]]

* STARTED Chapter 16: Risk
** X linked 
*** Transmission
- Daughters always inherit Dad's X
- Moms transmit one X
- Sons have phenotype
- Daughters are carriers

*** Prior probability that female is a carrier of X linked lethal disorder 
- H is the population frequency of female carriers
- There are there ways a female can be a carrier:
  1) inherits from a carrier mother
     - 1/2  H
  2) receives a new mutant allele on X from mom
     - \mu
  3) receives a new mutant allele on X from dad
     - \mu
\[H = (1/2 \cdot H) + \mu + \mu \]
\[H = H/2 +2\mu \]
\[H = 4\mu \]

- The incidence of carrier females in next generation (C_{n+1}) will
  be 1/2 the previous generation (C_{n}) plus the mutation rate in
  females (\mu) plus the mutation rate in males (\nu)

\[C_{n+1} = 1/2 \cdot C_n + \mu + \nu \]

- same as above

| I-2         | C    | NC  |
|-------------+------+-----|
| prior       | 4\mu | 1   |
| cond [fn:1] | 1/2  | \mu |
| joint       | 2\mu | \mu |
| post        | 2/3  | 1/3 |


- 2/3 inherited from mom
- 1/3 /de novo/



[fn:1] Prop of affected son
** AR
*** Carrier risk for unaffected sibs of patient w AR disease
- 2/3

* TODO Chapter 17: Prenatal
* TODO Chapter 18: Application
* TODO Chapter 19: Ethics




* 1/3 Rule
 If the reproductive fitness of a male affected with an X-linked
 recessive disorder is low or nil, then in a population *one-third of*
 *all affected X chromosomes will be removed from the gene pool every
 generation*. If the incidence of the disease is constant, then
 one-third of cases must be due to mutations arising /de novo/ in a
 family.


 \[
 p^2 + 2pq + q^2 = 1 
 \]
 \[
 q^2 \sim 0
 \]

 |   | X  | Y    |
 |---+----+------|
 | X | XX | XY   |
 |---+----+------|
 | x | xX | *xY* |


*** The Haldane Hypothesis 
 - Applies to X-linked recessive traits
   - A study of fertility rates in hemophillia

 - In a large population of 2N (N \male{} and N \female)
 - (1 - f)xN genes removed per generation
   - x = proportion of affected males in the polulation
   - f = effective fertility

 - Each of N \female{} has 2X/cell, and each of N \male{} has 1X/cell
 - The mean mutation rate per X-chromosome per generation is: [fn:haldane]

 \[
 u = 1/3(1 - f)x  
 \]

[fn:haldane] Haldane JB. The rate of spontaneous mutation of a human gene. 1935. J Genet 2004;83:235-44.
