#+TITLE: Thompson & Thompson
#+AUTHOR: Matthew Henderson
#+DATE: \today

:PROPERTIES:
#+DRAWERS: PROPERTIES
#+LaTeX_CLASS: koma-article
#+OPTIONS: H:3 toc:1 num:t ^:t
#+STARTUP: overview
#+SEQ_TODO: TODO(t) STARTED(s) | DONE(d) 
#+latex_header: \hypersetup{colorlinks,linkcolor=black,urlcolor=blue}
#+LaTeX_header: \usepackage{textpos}
#+LaTeX_header: \usepackage{textgreek}
#+LaTeX_header: \usepackage[version=4]{mhchem}
#+LaTeX_header: \usepackage{chemfig}
#+LaTeX_header: \usepackage{siunitx}
#+LaTeX_header: \usepackage{gensymb}
#+LaTex_HEADER: \usepackage[usenames,dvipsnames]{xcolor}
#+LaTeX_HEADER: \usepackage{lmodern}
#+LaTeX_HEADER: \usepackage{verbatim}
#+LaTeX_HEADER: \usepackage{tikz}
#+LaTeX_HEADER: \usepackage{wasysym}
#+LaTeX_HEADER: \usetikzlibrary{shapes.geometric,arrows,decorations.pathmorphing,backgrounds,positioning,fit,petri}
#+LaTeX_HEADER: \usepackage[automark, autooneside=false, headsepline]{scrlayer-scrpage}
#+LaTeX_HEADER: \clearpairofpagestyles
#+LaTeX_HEADER: \ihead{\leftmark}% section on the inner (oneside: right) side
#+LaTeX_HEADER: \ohead{\rightmark}% subsection on the outer (oneside: left) side
#+LaTeX_HEADER: \addtokomafont{pagehead}{\upshape}% header upshape instead of italic
#+LaTeX_HEADER: \ofoot*{\pagemark}% the pagenumber in the center of the foot, also on plain pages
#+LaTeX_HEADER: \pagestyle{scrheadings}
:END:

* Human Genome
** Mitosis
- Prophase :: condensation of the chromosomes
  - formation of the mitotic spindle
  - formation of a pair of centrosomes, from which microtubules
    radiate and eventually take up positions at the poles of the cell
- Prometaphase :: nuclear membrane dissolves
  - chromosomes disperse within the cell
  - attach by their kinetochores to microtubules of the mitotic
   spindle
- Metaphase :: the chromosomes condensed and line up at the equatorial
               plane
- Anaphase ::  chromosomes separate at the centromere
  - sister chromatids of each chromosome now become independent
    daughter chromosomes, which move to opposite poles of the cell
- Telophase :: chromosomes begin to decondense
  - nuclear membrane begins to re-form around each of the two daughter
    nuclei
  - cytoplasm cleaves = cytokinesis

#+CAPTION[]:Mitosis
#+NAME: fig:mitosis
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch2_mitosis.png]]


#+CAPTION[]:Karyotype
#+NAME: fig:karyotype
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch2_karyotype.png]]

** Meiosis
*** Meiosis I 
- meiosis I is also known as the reduction division because it is the
  division in which the chromosome number is reduced by half through
  the pairing of homologues in prophase and by their segregation to
  different cells at anaphase of meiosis I
- the stage at which genetic recombination (also called meiotic crossing over) occurs
  - homologous segments of DNA are exchanged between non-sister
    chromatids of each pair of homologous chromosomes, thus ensuring
    that none of the gametes produced by meiosis will be identical
*** Meiosis II 
- meiosis II is the second meiotic division, and involves equational
  segregation, or separation of sister chromatids
- mechanically, the process is similar to mitosis, though its genetic
  results are fundamentally different
- the end result is production of four haploid cells (n chromosomes,
  23 in humans) from the two haploid cells (with n chromosomes, each
  consisting of two sister chromatids) produced in meiosis I

#+CAPTION[]:Meiosis
#+NAME: fig:meiosis
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch2_meiosis.png]]

*** Homologous Recombination
 - occurs during meiosis I
 - the genetic content of each gamete is unique
 - \because of random assortment of the parental chromosomes to
   shuffle the combination of sequence variants between chromosomes
   and homologous recombination to shuffle the combination of sequence
   variants within each and every chromosome
 - this has significant consequences for patterns of genomic variation
   among and between different populations around the globe and for
   diagnosis and counselling of many common conditions with complex
   patterns of inheritance
 - the amounts and patterns of meiotic recombination are determined by
   sequence variants in specific genes and at specific “hot spots” and
   differ between individuals, between the sexes, between families, and
   between populations
 - \because recombination involves the physical intertwining of the two
   homologues until the appropriate point during meiosis I, it is also
   critical for ensuring proper chromosome segregation during
   meiosis
 - failure to recombine properly can lead to chromosome missegregation
   (nondisjunction) in meiosis I and is a frequent cause of pregnancy
   loss and of chromosome abnormalities like Down syndrome
 - although homologous recombination is normally precise, areas of
   repetitive DNA in the genome and genes of variable copy number in
   the population are prone to occasional unequal crossing over during
   meiosis, leading to variations in clinically relevant traits such as
   drug response, to common disorders such as the thalassemias or
   autism, or to abnormalities of sexual differentiation
 - although homologous recombination is a normal and essential part of
   meiosis, it also occurs, albeit more rarely, in somatic
   cells
 - anomalies in somatic recombination are one of the causes of genome
   instability in cancer

* Gene Structure and Function
** Allelic Imbalance in Gene Expression
- monoallelic gene expression
  - somatic rearrangement - T-cell receptors
  - random monoallelic expression
  - parent-of-origin imprinting
- X chromosome inactivation
  - random, X inactivation center
  - ncRNA called XIST

#+CAPTION[]:X Inactivation
#+NAME: fig:xist
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch3_xist.png]]

* Human Genetic Diversity
** Inherited variation and polymorphism
- SNPs
- indels
- microsatelite
- mobile element insertion polymorphism
  - retrotranspostion: Alu elements, long interspersed nucleotide elements (LINE)
- CNVs
  - related to indels and microsatellites but variation in
    the number of copies of larger segments of the genome
  - 1000 bp to many hundreds of kilobase pairs.
- inversion polymorphism
  - few base pairs up to several megabase pairs
  - can be present in either of two orientations in the genomes of different individuals

#+CAPTION[]:Polymorphism
#+NAME: fig:poly
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch4_polymorphism.png]]


** Mutation Types
- cell type:
  - germline
  - somatic
- mutations:
  - chromosome number
  - regional: affecting the structure or regional organization of chromosomes
  - gene: base pair substitutions, insertions, and deletions

#+CAPTION[]:Mutation Types and Frequency
#+NAME: fig:mutation
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch4_mutation.png]]

* Cytogenetics
** Clinical Indications for Chromosome and Genome Analysis
*** Problems of Early Growth and Development
  - failure to thrive, developmental delay, dysmorphic facies, multiple
    malformations, short stature, ambiguous genitalia, and
    intellectual disability are frequent findings in children with
    chromosome abnormalities
*** Stillbirth and Neonatal Death
- incidence of chromosome abnormalities is much higher among
  stillbirths (\sim10%) than among live births (\sim0.7%)
- also elevated among infants who die in the neonatal period (~10%)
- karyotyping (or other comprehensive ways of scanning the genome) is
  essential for accurate genetic counselling
*** Fertility
- chromosome studies are indicated for women presenting with
  amenorrhea and for couples with a history of infertility or recurrent miscarriage
- chromosome abnormality is seen in one or the other parent in 3% to
  6% of cases in which there is infertility or two or more
  miscarriages
*** Family History
- known or suspected chromosome or genome abnormality in a first
  degree relative is an indication for chromosome and genome analysis
*** Neoplasia
- Virtually all cancers are associated with one or more chromosome
  abnormalities
- chromosome and genome evaluation in the tumour itself, or in bone
  marrow in the case of hematological malignant neoplasms, can offer
  diagnostic or prognostic information
*** Pregnancy
- a higher risk for chromosome abnormality in fetuses conceived by
  women of increased age, typically defined as \gt 35 years
- fetal chromosome and genome analysis should be offered as a routine
  part of prenatal care in such pregnancies
- NIPT is a screening approach for the most common chromosome
  disorders and is now available to pregnant women of all ages

** Chromosome Identification
- G-banding (Giemsa) is the gold standard for detection and
  characterization of structural and numerical genomic abnormalities
  - both constitutional (postnatal or prenatal) and acquired (cancer)
  - detection of deletions and duplications \ge 5 to 10 Mb anywhere in
    the genome
- three types of chromosomes:
  - metacentric :: central centromere
  - submetacentric :: off center centromere
  - acrocentric :: centromere at one end
    - 13, 14, 15, 21, 22

** Fluorescence In Situ Hybridization
- detecting the presence or absence of a particular DNA sequence or
  for evaluating the number or organization of a chromosome or
  chromosomal region /in situ/
- uses ordered collections of recombinant DNA clones containing DNA
  from around the entire genome
- limited by the need to target a specific genomic region based on a
  clinical diagnosis or suspicion

** Microarrays
- comparative genome hybridization (CGH)
  - detects relative copy number gains and losses genome-wide by
    hybridizing two samples:
    - control genome
    - patient
  - excess of sequences from one or the other genome indicates an
    overrepresentation or underrepresentation of those sequences in the
    patient genome relative to the control
- SNP arrays
  - relative representation and intensity of alleles in different
    regions of the genome indicate if a chromosome or chromosomal
    region is present at the appropriate dosage
  - loss of heterozygozity
- probe spacing provides a resolution as high as 250 kb
** Chromosome Abnormalities
- numerical or structural
- incidence of 1/154 live births
- aneuploidy is most common
  - associated with physical and/or mental abnormalities
- structural abnormalities are also common

#+CAPTION[]:Incidence of Chromosomal Abnormalities
#+NAME: fig:freq
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_freq.png]]

#+CAPTION[]:ISCN for Common Cytogenetic Aberration
#+NAME: fig:nom
#+ATTR_LaTeX: :width 1.2\textwidth
[[file:./figures/ch5_nom.png]]

#+CAPTION[]:ISCN (continued)
#+NAME: fig:nom
#+ATTR_LaTeX: :width 1.2\textwidth
[[file:./figures/ch5_nom2.png]]

** Gene Dosage, Balance and Imbalance
- for chromosome and genomic disorders, it is the quantitative aspects
  of gene expression that underlie disease, in contrast to single-gene
  disorders, in which pathogenesis often reflects qualitative aspects
  of a gene's function
*** monosomies are more deleterious than trisomies
- complete monosomies are generally not viable, except for monosomy
  for the X chromosome
- complete trisomies are viable for chromosomes 13, 18, 21, X, and Y

*** phenotype in partial aneuploidy depends on a number of factors
- size of the unbalanced segment
- which regions of the genome are affected
- which genes are involved
- whether the imbalance is monosomic or trisomic
*** risk in cases of inversions depends on the location of the inversion with respect to the centromere and on the size of the inverted segment
- paracentric inversions do not involve the centromere
  - very low risk for an abnormal phenotype in the next generation
- pericentric inversions do involve the centromere
  - risk for birth defects in offspring may be significant and
    increases with the size of the inverted segment

*** mosaic karyotype involving any chromosome abnormality, all bets are off!
- the degree of mosaicism in relevant tissues or relevant stages of
  development is generally unknown
- there is uncertainty about the severity of the phenotype

** Abnormalities of Chromosome Number
- heteroploid :: chromosome number that is neither haploid (n=23) or diploid(2n=46)
- euploid :: exact multiple of n (e.g. triploid)
  - triploidy and tetraploidy
    - most result from fertilization of an egg by two sperm (dispermy)
    - also failure of one of the meiotic divisions in either sex,
      resulting in a diploid egg or sperm
    - maternal source are aborted
    - paternal source \to degenerative placenta (parital hydatidiform
      mole) w small fetus
- aneuploid :: non-multiple of n (e.g. trisomy 21)
  - most common cause is meiotic nondisjunction in meiosis I or II (Figure [[fig:nondys]])
    - trisomy :: 21, 18, 13
    - monosomy :: X (Turner syndrome)




#+CAPTION[]:Nondisjunction
#+NAME: fig:nondys 
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_nondys.png]]

** Abnormalities of Chromosome Structure
- present in 1/375 newborns
- balanced or unbalanced

#+CAPTION[]:Structural Rearrangements of Chromosomes
#+NAME: fig:str
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_struct.png]]

*** Unbalanced Rearrangements
- delections and duplications
- marker and ring chromosomes
  - very small, unidentified chromosomes
- isochromosomes
  - one arm is missing and the other duplicated in a mirror-image
- dicentric
  - two chromosome segments, each with a centromere, fuse end to end
*** Balanced Rearrangements
- "balanced" depends on resolution
- translocations
  - reciprocal translocations
  - robertsonian translocations
  - insertions
- inversions
  - paracentric - outside the centromere
  - pericentric - includes the centromere

#+CAPTION[]:Balanced Translocations
#+NAME: fig:trans
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch5_trans.png]]

* Chromosomal and Genomic basis of Disease
- disorders due to:
  - abnormal chromosome segregation (nondisjunction)
  - recurrent chromosomal syndromes, involving
    deletions or duplications at genomic hot spots
  - idiopathic chromosomal abnormalities, typically /de novo/
  - unbalanced familial chromosomal abnormalities
  - chromosomal and genomic events that reveal regions
    of genomic imprinting

#+CAPTION[]:Mechanisms of Chromosome Abnormalities and Genomic Imbalance
#+NAME: fig:mech 
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch6_mech.png]]

** Lessons from Genomic Disorders
- altered gene dosage for any extensive chromosomal or genomic region
  is likely to result in a clinical abnormality, the phenotype of
  which will, in principle, reflect haploinsufficiency for or
  overexpression of one or more genes encoded within the region.
  - in some cases the clinical presentation appears to be accounted
    for by dosage imbalance for just a single gene; in other
    syndromes, however, the phenotype appears to reflect imbalance for
    multiple genes across the region
- the distribution of these duplication/deletion disorders is not random
  - segmental duplications in pericentromeric and subtelomeric
    regions, predisposes particular regions to the unequal
    recombination events that underlie these syndromes
- patients carrying what appears to be the same chromosomal deletion
  or duplication can present with a range of variable phenotypes

** Aneuploidy
- common mutation in our species involves errors in chromosome segregation
- only three well-defined non-mosaic chromosome disorders compatible
  with postnatal survival in which there is an abnormal dose of an
  entire autosome:
  1) trisomy 21 (Down syndrome)
  2) trisomy 18
  3) trisomy 13
- also have the lowest number of genes among autosomes

*** Down Syndrome
- see Conditions
**** Robertsonian Translocation 
- \sim4% of Down syndrome patients have 46 chromosomes
- one of which is a Robertsonian translocation between chromosome
  21q and the long arm of one of the other acrocentric chromosomes
  (usually chromosome 14 or 22)
- 46,XX or XY,rob(14;21)(q10;q10),+21

#+CAPTION[]:Chromosomes of Gametes that Theoretically can be Produced by a Carrier of a Robertsonian Translocation, rob(14;21)
#+NAME: fig:ft
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch6_rtgam.png]]

*** Uniparental Disomy
- nondisjunction \to both copies of a chromosome derive from the same
  parent
  - called uniparental disomy
  - defined as the presence of a disomic cell line containing two
    chromosomes, or portions thereof, that are inherited from only one
    parent
- isodisomy if  two chromosomes are derived from identical sister chromatids
- heterodisomy if if both homologues from one parent are present
- common explanation for uniparental disomy is trisomy “rescue” due to
  chromosome nondisjunction in cells of a trisomic conceptus to
  restore a disomic state

*** Contiguous Gene Syndrome
- segmental aneusomy is a form of genetic imbalance due to recurrent
  subchromosomal or regional abnormalities
  - typically detected by microarray
  - called contiguous gene syndrome

** Idiopathic Chromosome Abnormalities 
- autosomal deletion syndromes
  - Cri du Chat syndrome is either a terminal or interstitial deletion
    of part of the short arm of chromosome 5
- balanced translocations with developmental phenotypes
  
** Disorders Associated with Genomic Imprinting
- Prader-Willi
- Angelman syndrome
- Beckwith-Wiedemann syndrome

#+CAPTION[]:Mechanism causing Prader-Willi and Angelman Syndrome
#+NAME: fig:pw_as
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch6_pw_as.png]]

- see Conditions
** Sex Chromosomes and Their Abnormalities
- SRY found on the Y chromosome is the major testis-determining gene
- inactivation of an X chromosome depends on the presence of the X
  inactivation center region (XIC)
- in females structurally abnormal X chromosomes are almost always inactive
  - nonrandom inactivation observed in X;autosome translocations
  - if balanced the normal X chromosome is preferentially inactivated
  - the two parts of the translocated chromosome remain active
  - reflecting selection against cells in which critical autosomal
    genes have been inactivated

#+CAPTION[]:Sex Chromosome Constitution and X Inactiviation
#+NAME: tab:xinactive
| phenotype | karyotype                     | active X | inactive X |
|-----------+-------------------------------+----------+------------|
| \male     | 46,XY; 47,Xyy                 |        1 |          0 |
|           | 47,XXY (Klinefelter); 48,XXYY |        1 |          1 |
|           | 48,XXXY; 49,XXXYY             |        1 |          2 |
|           | 49,XXXXY                      |        1 |          3 |
|-----------+-------------------------------+----------+------------|
| \female   | 45,X (Turner)                 |        1 |          0 |
|           | 46,XX                         |        1 |          1 |
|           | 47,XXX                        |        1 |          2 |
|           | 48,XXXX                       |        1 |          3 |
|           | 49,XXXXX                      |        1 |          4 |

* Patterns of Single Gene Inheritance
- penetrance :: the probabilty that a mutant allele(s) will have a
  phenotypic expression
  - penetrance is all or nothing (affected vs unaffected)
- expressivity :: the severity of the phenotype among those with the
  disease causing genotype

#+CAPTION[]:Pedigree Symbols
#+NAME: fig:pedigree
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch7_pedigree.png]]

** Autosomal Dominant Inheritance
  - disorder occurs in every generation
  - affected person has affected parents
    - except in /de novo/ mutation
    - non-penetrant
  - 50% risk of trait in child of affected parent
  - significant number of cases due to /de novo/ mutations

  - pure dominant inheritance is rare
    - generally homozygotes are more severe
  - achondroplasia is an example of incomplete dominant inheritance
  - sex limited phenotype in some disorders

** X-linked Inheritance
- X-linked dominant or recessive depends on phenotype in \female
  - \sim 1/3 of XL disorders are penetrant in het \female 

** Mosaicism
- confined placental :: discrepancy between the chromosomal makeup of
  the cells in the placenta and the cells in the fetus (Figure [[fig:pmosaic]])
- somatic :: somatic cells
- germline :: germline cells
- segmental :: mutation affecting morphogenesis, occuring during
  embryonic development

#+CAPTION[]:Genotypes and Phenotypes in X-linked Disease
#+NAME: tab:xlinked
|         | genotype                   | phenotype            |
|---------+----------------------------+----------------------|
| \male   | hemizygous X_H             | unaffected           |
|         | hemizygous X_h             | affected             |
|---------+----------------------------+----------------------|
| \female | homozygous X_H/X_H         | unaffected           |
|         | heterozygous X_H/X_h       | carrier \pm affected |
|         | homozygous X_h/X_h         | affected             |
|         | compound het X_{h1}/X_{h2} | affected             |
** Mitochondrial DNA
*** Maternal Inheritance
 - mitochondria and therefore the mtDNA, usually come from the egg
   - the egg cell contains relatively few mitochondria
   - these mitochondria divide to populate the cells
 - sperm mitochondria enter the egg, but do not contribute genetic
   information to the embryo.
   - paternal mitochondria are marked with ubiquitin for destruction
     inside the embryo.
 - mitochondria are randomly distributed to the daughter cells during
   the division of the cytoplasm.

*** Heteroplasmy
 - heteroplasmy is the presence of more than one type of organellar
   genome within a cell or individual
 - it is an important factor in considering the severity of
   mitochondrial diseases
   - can also be beneficial
 - microheteroplasmy is present in most individuals
   - hundreds of independent mutations, with each mutation found in
     about 1–2% of all mitochondrial genomes

** Genotype - Phenotype
- allelic heterogeneity :: different mutations in a gene cause the same phenotype
- locus heterogeneity :: mutations in different genes cause the same phenotype
- phenotypic heterogeneity :: different mutations in the same gene
  result in different phenotypes

* Complex Traits
- *qualitative trait* trait is present or absent
- *quantitative trait* measurable physiological or biochemical quantity
- relative risk ratio \lambda_r is used to measure famillial
  aggregation of disease in qualitative traits
- \lambda_r = prevalence of disease in relatives/prevalence in population
- H^{2} is heritability
  - fraction of phenotypic variance of a quantitative trait that is
    due to allelic variation

* Genetic Variation in Populations
** Hardy-Weinberg
- the Hardy-Weinberg law states that the frequency of the three
  genotypes AA, Aa, and aa is given by the terms of the binomial
  expansion of
  - (p + q)^2 = p^2 + 2pq + q^2
    - p = the frequency of the dominant allele (represented here by A)
    - q = the frequency of the recessive allele (represented here by a)

- for a population in genetic equilibrium:
  - p + q = 1 (sum of the frequencies of both alleles is 100%)
  - (p + q)^2 = 1
  - p^2 + 2pq + q^2 = 1
- the three terms of this binomial expansion indicate the frequencies
  of the three genotypes (Table [[tab:hwfreq]])
  - p^2 = frequency of AA 
  - 2pq = frequency of Aa 
  - q^2 = frequency of aa
  - q = frequency of affected males in XL disease



#+CAPTION[]:Population Frequency in Modes of Inheritance
#+NAME: tab:hwfreq
| inheritance | frequency     | genotype |                  |
|-------------+---------------+----------+------------------|
| AR          | q^2           | aa       | homozygote       |
| AD          | 2pq           | Aa       | heterozygotes    |
| XL          | q[fn:xl]      | X_{M}Y   | hemizygote \male |


[fn:xl] q = frequency of affected \male{} in XL disease


- the Hardy-Weinberg law rests on these assumptions:
  - the population under study is large and matings are random with
    respect to the locus in question
  - allele frequencies remain constant over time because of the
    following:
    - there is no appreciable rate of new mutation
    - individuals with all genotypes are equally capable of mating and
      passing on their genes
      - \therefore no selection against	any particular genotype
    - there has been no significant immigration of individuals from a
      population with allele frequencies very different from the
      endogenous population
- a population that reasonably appears to meet these assumptions is
  considered to be in Hardy-Weinberg equilibrium
  - population genotype frequencies from generation to generation will
    remain constant if the allele frequencies p and q remain constant

** Factors That Disturb Hardy-Weinberg Equilibrium
*** exceptions to large populations with random mating
- stratification :: isolation by distance
- assortative mating :: social factors
- consaguinity ::
*** exceptions to constant allele frequencies
- mutation
- selection and fitness
  - selection in dominant disease
  - selection in recessive disease - less effect
- mutation and selection balance in dominant disease:
  - \mu = sq
    - \mu = mutation rate per generation
    - q = mutant allele freq
    - s = coefficient of selection
- mutation and selection balance in X-linked recessive mutations
  - 1/3 rule

** 1/3 Rule
  - \male{} have one X chromosome and \female{} two, the pool of X-linked
    alleles in the entire population’s gene pool is partitioned at any
    given time, with one third of mutant alleles present in \male{} and
    two thirds in \female
  - if the reproductive fitness of a \male affected with an X-linked
    recessive disorder is low or nil, then in a population
    - *one-third of all affected X chromosomes will be removed from
      the gene pool every* *generation*
  - if the incidence of a serious X-linked disease is not changing and
    selection is operating against (and only against) hemizygous \male,
    the mutation rate \mu, must equal the coefficient of selection, s
    times q, the allele frequency, adjusted by a factor of 3 because
    selection is operating only on the third of the mutant alleles in
    the population that are present in males at any time. 
    - \mu = sq/3
  - *if the incidence of the disease is constant, then one-third of
    cases must be due to mutations arising /de novo/ in a family*

*** COMMENT Haldane Hypothesis 
 - applies to X-linked recessive traits
   - a study of fertility rates in hemophillia

 - in a large population of 2N (N \male and N \female)
 - (1 - f)xN genes removed per generation
   - x = proportion of affected males in the population
   - f = effective fertility

 - each of N \female{} has 2X/cell
 - each of N \male{} has 1X/cell
 - the mean mutation rate per X-chromosome per generation is: 

 \[
 u = 1/3(1 - f)x  
 \]

[fn:haldane] Haldane JB. The rate of spontaneous mutation of a human gene. 1935. J Genet 2004;83:235-44.

* Identifying the Genetic Basis for Human Disease
- linkage analysis is family-based
  - takes advantage of family pedigrees to follow the inheritance of a
    disease among family members and to test for consistent, repeated
    coinheritance of the disease with a particular genomic region or
    even with a specific variant or variants, whenever the disease is
    passed on in a family
- association analysis is population-based
  - does not depend explicitly on pedigrees but instead takes
    advantage of the entire history of a population to look for
    increased or decreased frequency of a particular allele or set of
    alleles in a sample of affected individuals taken from the
    population, compared with a control set of unaffected people from
    that same population
  - particularly useful for complex diseases that do not show a
    mendelian inheritance pattern
- direct genome sequencing of affected individuals and their parents
  and/or other individuals in the family or population
  - particularly useful for rare mendelian disorders in which linkage
    analysis is not possible because there are simply not enough such
    families to do linkage analysis or because the disorder is a
    genetic lethal that always results from new mutations and is
    never inherited

- *linkage* is the term used to describe a departure from the
  independent assortment of two loci
- analysis of linkage depends on determining the frequency of
  recombination as a measure of how close two loci are to each other
  on a chromosome.
- a common notation for recombination frequency is \theta
  - where \theta varies from 0 (no recombination at all) to 0.5 (independent assortment)
  - if two loci are so close together that \theta = 0 between them
    they are said to be completely linked
  - if they are so far apart that θ = 0.5 they are assorting
    independently and are unlinked
  - in between these two extremes are various degrees of linkage

- map distance between two loci is a theoretical concept that is based
  on the extent of observed recombination, \theta, between the
  loci
  - measured in units called centimorgans (cM)
  - defined as the genetic length over which, on average, one
    crossover occurs in 1% of meioses
  - \therefore a recombination fraction of 1% (\theta = 0.01)
    translates approximately into a map distance of 1 cM

- linkage disequilibrium (LD) is the due to close map distance between loci
  - frequency of a haplotype, freq(A-S) for example, may not be equal
    to the product of the frequencies of the individual alleles that
    make up that haplotype
  - in this situation, freq(A-S) \neq freq(A) x freq(S)
    - the alleles are in LD
* Molecular Basis of Genetic Disease
- effect of mutation on protein function
  - loss of function :: \downarrow protein function (i.e. CF)
    - almost always recessive
  - gain of function :: \uparrow expression or \uparrow activity (i.e BCR-ABL)
    - almost always dominant
  - novel property :: i.e. sickle cell
  - heterochronic expression :: wrong time i.e. hereditary persistence
    of Hb F
  - ectopic expression :: wrong place i.e oncogene

* Treatment of Genetic Disease

#+CAPTION[]:Levels of Treatment in Genetic Disease
#+NAME: fig:treament
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch13_treatment.png]]

** Manipulation of Metabolism
- substrate reduction :: diet in PKU
- replacement :: T4 in CH
- diversion :: sodium benzoate in UCD
- enzyme inhibition :: nitisinone in Tyrosinemia I
- receptor antagonism :: TGF-\beta in Marfan
- depletion :: aphoresis in homozygous familial hypercholesterolemia

** Function of Affected Gene or Protein
- skipping nonsense codons :: CF
- folding :: CF
- membrane trafficking :: CF
- protein augmentation :: hemophilia
- ERT :: Gaucher Type I, ADA 

** Modulation of Gene Expression
- DNA methylation :: \uparrow HbF in \beta-thalassemias
- siRNA :: Huntington
- exon skipping :: DMD
- gene editing :: CRISPR/Cas9

** Transplantation
- HSCT :: Hurler, SCID
- Liver :: \alpha1AT, UCD

** Gene Therapy Risks
- adverse response to vector
- insertional mutagenesis \to malignancy
- insertional inactivation of essential gene

* Developmental Genetics
** Malformations 
- arise from intrinsic defects in genes that specify a series of
  developmental steps or programs, and because such programs are often
  used more than once in different parts of the embryo or fetus at
  different stages of development, a malformation in one part of the
  body is often but not always associated with malformations elsewhere
  as well
- polydactyly (extra fingers or toes) and syndactyly (fused fingers)
  are malformation
** Deformations
- caused by extrinsic factors impinging physically on the fetus during
  development
- common during the second trimester of development when the fetus is
  constrained within the amniotic sac and uterus
  - for example contractions of the joints of the extremities, known
    as arthrogryposes, in combination with deformation of the
    developing skull, occasionally accompany constraint of the fetus
    due to twin or triplet gestations or prolonged leakage of amniotic
    fluid
- most deformations apparent at birth either resolve spontaneously or
  can be treated by external fixation devices to reverse the effects
  of the instigating cause
** Disruptions
- result from destruction of irreplaceable normal fetal
  tissue
- more difficult to treat than deformations because they involve
  actual loss of normal tissue
- may be the result of vascular insufficiency, trauma, or
  teratogens
- one example is amnion disruption, the partial amputation of a fetal
  limb associated with strands of amniotic tissue
  - amnion disruption is often recognized clinically by the presence of
    partial and irregular digit amputations in conjunction with
    constriction rings

* Cancer
- cancer is not a single disease but rather comes in many forms and
  degrees of malignancy
- there are three main classes of cancer:
  - sarcomas :: tumour has arisen in mesenchymal tissue, such as bone,
    muscle, connective tissue or nervous system
  - carcinomas :: originate in epithelial tissue, such as the cells
    lining the intestine, bronchi, or mammary ducts
  - hematopoietic and lymphoid :: leukemia and lymphoma which spread
       throughout the bone marrow, lymphatic system, and peripheral
       blood
** Genetic Basis of Cancer
- driver and passenger mutations
- particular translocations are sometimes highly specific for certain
  types of cancer and involve specific genes
  - BCR-ABL translocation in chronic myelogenous leukemia
- cancers can also show complex rearrangements in which chromosomes
  break into numerous pieces and rejoin, forming novel and complex
  combinations (a process known as “chromosome shattering”)
- large genomic alterations involving many kilobases of DNA can form
  the basis for loss of function or increased function of one or more
  driver genes

- genes in which mutations cause cancer are referred to as *driver*
  *genes*, and the cancer causing mutations in these genes are *driver*
  *mutations*
- driver genes fall into two distinct categories
  1) activated oncogenes
  2) tumour suppressor genes (TSGs)

- an activated oncogene is a mutant allele of a proto-oncogene, a
  class of normal cellular protein-coding genes that promotes growth
  and survival of cells
- oncogenes facilitate malignant transformation by stimulating
  proliferation or inhibiting apoptosis
- oncogenes encode proteins such as the following:
  - proteins in signaling pathways for cell proliferation
  - transcription factors that control the expression of growth-promoting genes
  - inhibitors of programmed cell death machinery
- a TSG is a gene in which loss of function through mutation or
  epigenomic silencing directly removes normal regulatory controls on
  cell growth or leads indirectly to such losses through an increased
  mutation rate or aberrant gene expression
- TSGs encode proteins involved in many aspects of cellular function including:
  - maintenance of correct chromosome number and structure
  - DNA repair proteins
  - proteins involved in regulating the cell cycle, cellular
    proliferation or contact inhibition

- tumour initiation can be caused by different types of genetic
  alterations:
  - activating or gain-of-function mutations
  - ectopic and heterochronic mutations of protooncogenes
  - chromosome translocations that cause misexpression of genes or chimeric genes
  - LOF of both alleles, or a dominant negative mutation of one allele, of TSGs

- tumour progression occurs as a result of accumulating additional
  genetic damage,through mutations or epigenetic silencing, of driver
  genes that encode the machinery that repairs damaged DNA and
  maintains cytogenetic normality

** Cancer in Families
- germline mutation - inherited
- second-hit
  - somatic mutation
  - loss of heterozygozity around locus

** Sporadic Cancer
- activation of oncogenes by point mutation
- activation of oncogenes by chromosome translocation
  - best-known example is the translocation between chromosomes 9 and
    22 (Philadelphia chromosome) that is seen in CML
  - moves the protooncogene ABL1, a tyrosine kinase, from its normal
    position on chromosome 9q to a gene of unknown function, BCR, on
    chromosome 22q
  - results in the synthesis of a novel, chimeric protein, BCR-ABL1 w
    enhanced tyrosine kinase activity

#+CAPTION[]: Characteristic Chromosome Translocations in Selected Human Malignant Neoplasms
#+NAME: fig:trans 
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch15_neo.png]]

** Cytogenetic Changes in Cancer
*** Aneuploidy and Aneusomy
- aneuploidy - abnormal chromosome number
- aneusomy - cells with different numbers of chromosomes
- when CML, with the 9;22 Philadelphia chromosome, evolves from the
  typically indolent chronic phase to a severe, life-threatening blast
  crisis, there may be several additional cytogenetic abnormalities,
  including numerical or structural changes, such as a second copy of
  the 9;22 translocation chromosome or an isochromosome for 17q

- another cytogenetic aberration seen in many cancers is gene
  amplification, a phenomenon in which many additional copies of a
  segment of the genome are present in the cell
  - *double minutes* (very small accessory chromosomes)
  - *homogeneously staining regions* that do not band normally and
    contain multiple, amplified copies of a particular DNA segment
  - known to include extra copies of proto-oncogenes such as the genes
    encoding Myc, Ras, and epithelial growth factor receptor, which
    stimulate cell growth, block apoptosis, or both

** Targeted Cancer Therapy
- imatinib is a highly effective inhibitor of a number of tyrosine
  kinases including the ABL1 kinase in CML
  - prolonged remissions of this disease have been seen, in some cases
    with apparently indefinite postponement of the transformation into a
    virulent acute leukemia (blast crisis) that so often meant the end
    of a CML patient’s life
- additional kinase inhibitors have been developed to target other
  activated oncogene driver genes in a variety of tumour types
- initial results with targeted therapies, although very promising in
  some cases, have not led to permanent cures in most patients because
  tumours develop resistance to the targeted therapy
- the outgrowth of resistant tumours because cancer cells are highly
  mutable, and their genomes undergo recurrent mutation

#+CAPTION[]:Targeted Cancer Treatment
#+NAME: fig:target
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch15_target.png]]

* Calculation of Genetic Risk 
- prior probability :: depends on known data or stated scenario
  - population risk, affected or carrier parents
- conditional probability :: is based on observed outcomes
  - i.e. probability of 4 unaffected children
- joint probability :: probability of outcome given prior
  - i.e. probability of 4 unaffected children with carrier parent
\[
  joint_x = conditional_x \cdot prior_x
\]
- posterior probability :: fraction of total joint probability for a scenario
  - i.e  probability daughter is a carrier with four unaffected sibs
\[
 posterior_x = \frac{joint_x}{\sum_{i=1}^n joint_i}
\]  
** Approach to Problems
1. what is the question?
   - the answer is posterior probability for that scenario
     - i.e risk that daughter is a carrier
2. what are the scenarios for the unknown components in the problem
   - these will become the denominator in the posterior probability
     - mom carrier, daughter carrier
     - mom carrier, daughter non-carrier
     - mom non-carrier, daughter non-carrier
3. what are the prior probabilities for each scenario?
   - these are either given or are set for that scenario
     - when an status (affected, carrier, etc) is unknown each option
       forms a scenario and becomes a prior for that scenario
4. calculate the conditional probability for each scenario
   - simple ones are the probability of inheriting
     - 1/4 for AR
     - 1/2 from mom in XL

   - based on evidence such as number of unaffected children
5. calculate the joint probability for each scenario 
6. calculate the posterior probability for each scenario

** X linked
- transmission
  - daughters always inherit dad's X
    - P(transmission) = 1
  - moms transmit one of 2 Xs
    - P(transmission) = 1/2
  - sons have phenotype
  - daughters are carriers (X-linked recessive)
  - prior probability that female is a carrier of X linked lethal disorder 
    - H is the population frequency of female carriers
  - there are three ways a female can be a carrier:
    1) inherits from a carrier mother
       - 1/2  H
    2) receives a new mutant allele on X from mom
       - \mu
    3) receives a new mutant allele on X from dad
       - \nu
       - often simplified as \nu = \mu
\[H = (1/2 \cdot H) + \mu + \mu \]
\[H = H/2 +2\mu \]
\[2H = H +4\mu \]
\[H = 4\mu \]

 - the incidence of carrier females in next generation (C_{n+1}) will
   be 1/2 the previous generation (C_{n}) plus the mutation rate in
   females (\mu) plus the mutation rate in males (\nu)

\[C_{n+1} = 1/2 \cdot C_n + \mu + \nu \]

 - same as above

| I-2         | C    | NC  |
|-------------+------+-----|
| prior       | 4\mu | 1   |
| cond [fn:1] | 1/2  | \mu |
| joint       | 2\mu | \mu |
| post        | 2/3  | 1/3 |


- 2/3 inherited from mom
- 1/3 /de novo/

[fn:1] prop of affected son

** AR
- carrier risk for unaffected sibs of patient w AR disease
  - P(sib is carrier) = 2/3
  - P(parent is carrier) = 1

#+CAPTION[]: Punnett Square for AR Disease 
#+NAME: tab:arps
|   | A  | a    |
| A | AA | Aa   |
| a | Aa | *aa* |

* Prenatal Diagnosis and Screening
- placental mosaicism can result in CVS karyotype results that do not
  reflect the fetus (Figure [[fig:pmosaic]])
  - confined placental mosaicism can lead to incorrect interpretation

#+CAPTION[]:Mosaicism Detected in Prenatal Diagnsosis
#+NAME: fig:pmosaic
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./figures/ch17_mosaic.png]]


#+CAPTION[]:First Trimester Screening Tests
#+NAME: tab:mss1
| disorder | NT       | PAPP-A     | \beta-hCG  |
|----------+----------+------------+------------|
| T21      | \uparrow | \downarrow | \uparrow   |
| T18      | \uparrow | \downarrow | \downarrow |
| T13      | \uparrow | \downarrow | \downarrow |
| NTD      | -        | -          | -          |


#+CAPTION[]:Second Trimester Screening Tests
#+NAME: tab:mss2
| disorder | uE_3       | AFP        | hCG        | Inhibin A |
|----------+------------+------------+------------+-----------|
| T21      | \downarrow | \downarrow | \uparrow   | \uparrow  |
| T18      | \downarrow | \downarrow | \downarrow | -         |
| T13      | \downarrow | \downarrow | \downarrow | -         |
| NTD      | -          | \Uparrow   | -          | -         |

* Ethics

** Principles of Biomedical Ethics
- four cardinal principles are frequently considered in any discussion
  of ethical issues in medicine:
  - respect for individual autonomy :: safeguarding an individual's
    rights to control their medical care and medical information, free
    of coercion
  - beneficence :: doing good
  - avoidance of maleficence :: “first of all, do no harm”
  - justice :: ensuring that all individuals are treated equally and
    fairly



