% Created 2020-07-13 Mon 10:48
% Intended LaTeX compiler: pdflatex
\documentclass[12pt]{scrartcl}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{longtable}
\usepackage{wrapfig}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{amsmath}
\usepackage{textcomp}
\usepackage{amssymb}
\usepackage{capt-of}
\usepackage{hyperref}
\hypersetup{colorlinks,linkcolor=black,urlcolor=blue}
\usepackage{textpos}
\usepackage{textgreek}
\usepackage[version=4]{mhchem}
\usepackage{chemfig}
\usepackage{siunitx}
\usepackage{gensymb}
\usepackage[usenames,dvipsnames]{xcolor}
\usepackage{lmodern}
\usepackage{verbatim}
\usepackage{tikz}
\usepackage{wasysym}
\usetikzlibrary{shapes.geometric,arrows,decorations.pathmorphing,backgrounds,positioning,fit,petri}
\usepackage[automark, autooneside=false, headsepline]{scrlayer-scrpage}
\clearpairofpagestyles
\ihead{\leftmark}% section on the inner (oneside: right) side
\ohead{\rightmark}% subsection on the outer (oneside: left) side
\addtokomafont{pagehead}{\upshape}% header upshape instead of italic
\ofoot*{\pagemark}% the pagenumber in the center of the foot, also on plain pages
\pagestyle{scrheadings}
\author{Matthew Henderson}
\date{\today}
\title{Thompson \& Thompson}
\hypersetup{
 pdfauthor={Matthew Henderson},
 pdftitle={Thompson \& Thompson},
 pdfkeywords={},
 pdfsubject={},
 pdfcreator={Emacs 26.3 (Org mode 9.3.7)}, 
 pdflang={English}}
\begin{document}

\maketitle
\setcounter{tocdepth}{1}
\tableofcontents


\section{Human Genome}
\label{sec:orgbfc0311}
\subsection{Mitosis}
\label{sec:orgb3a975e}
\begin{description}
\item[{Prophase}] condensation of the chromosomes
\begin{itemize}
\item formation of the mitotic spindle
\item formation of a pair of centrosomes, from which microtubules
radiate and eventually take up positions at the poles of the cell
\end{itemize}
\item[{Prometaphase}] nuclear membrane dissolves
\begin{itemize}
\item chromosomes disperse within the cell
\item attach by their kinetochores to microtubules of the mitotic
spindle
\end{itemize}
\item[{Metaphase}] the chromosomes condensed and line up at the equatorial
plane
\item[{Anaphase}] chromosomes separate at the centromere
\begin{itemize}
\item sister chromatids of each chromosome now become independent
daughter chromosomes, which move to opposite poles of the cell
\end{itemize}
\item[{Telophase}] chromosomes begin to decondense
\begin{itemize}
\item nuclear membrane begins to re-form around each of the two daughter
nuclei
\item cytoplasm cleaves = cytokinesis
\end{itemize}
\end{description}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch2_mitosis.png}
\caption{\label{fig:org0412663}Mitosis}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch2_karyotype.png}
\caption{\label{fig:org19d0f06}Karyotype}
\end{figure}

\subsection{Meiosis}
\label{sec:org43f95f6}
\subsubsection{Meiosis I}
\label{sec:orgb4b8204}
\begin{itemize}
\item meiosis I is also known as the reduction division because it is the
division in which the chromosome number is reduced by half through
the pairing of homologues in prophase and by their segregation to
different cells at anaphase of meiosis I
\item the stage at which genetic recombination (also called meiotic crossing over) occurs
\begin{itemize}
\item homologous segments of DNA are exchanged between non-sister
chromatids of each pair of homologous chromosomes, thus ensuring
that none of the gametes produced by meiosis will be identical
\end{itemize}
\end{itemize}
\subsubsection{Meiosis II}
\label{sec:orgb6bde44}
\begin{itemize}
\item meiosis II is the second meiotic division, and involves equational
segregation, or separation of sister chromatids
\item mechanically, the process is similar to mitosis, though its genetic
results are fundamentally different
\item the end result is production of four haploid cells (n chromosomes,
23 in humans) from the two haploid cells (with n chromosomes, each
consisting of two sister chromatids) produced in meiosis I
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch2_meiosis.png}
\caption{\label{fig:org076cfcf}Meiosis}
\end{figure}

\subsubsection{Homologous Recombination}
\label{sec:org1e7ca92}
\begin{itemize}
\item occurs during meiosis I
\item the genetic content of each gamete is unique
\item \(\because\) of random assortment of the parental chromosomes to
shuffle the combination of sequence variants between chromosomes
and homologous recombination to shuffle the combination of sequence
variants within each and every chromosome
\item this has significant consequences for patterns of genomic variation
among and between different populations around the globe and for
diagnosis and counselling of many common conditions with complex
patterns of inheritance
\item the amounts and patterns of meiotic recombination are determined by
sequence variants in specific genes and at specific “hot spots” and
differ between individuals, between the sexes, between families, and
between populations
\item \(\because\) recombination involves the physical intertwining of the two
homologues until the appropriate point during meiosis I, it is also
critical for ensuring proper chromosome segregation during
meiosis
\item failure to recombine properly can lead to chromosome missegregation
(nondisjunction) in meiosis I and is a frequent cause of pregnancy
loss and of chromosome abnormalities like Down syndrome
\item although homologous recombination is normally precise, areas of
repetitive DNA in the genome and genes of variable copy number in
the population are prone to occasional unequal crossing over during
meiosis, leading to variations in clinically relevant traits such as
drug response, to common disorders such as the thalassemias or
autism, or to abnormalities of sexual differentiation
\item although homologous recombination is a normal and essential part of
meiosis, it also occurs, albeit more rarely, in somatic
cells
\item anomalies in somatic recombination are one of the causes of genome
instability in cancer
\end{itemize}

\section{Gene Structure and Function}
\label{sec:orgafcf79d}
\subsection{Allelic Imbalance in Gene Expression}
\label{sec:orgc4ab2df}
\begin{itemize}
\item monoallelic gene expression
\begin{itemize}
\item somatic rearrangement - T-cell receptors
\item random monoallelic expression
\item parent-of-origin imprinting
\end{itemize}
\item X chromosome inactivation
\begin{itemize}
\item random, X inactivation center
\item ncRNA called XIST
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch3_xist.png}
\caption{\label{fig:org2821c7f}X Inactivation}
\end{figure}

\section{Human Genetic Diversity}
\label{sec:orgd00a130}
\subsection{Inherited variation and polymorphism}
\label{sec:org8f8aaa8}
\begin{itemize}
\item SNPs
\item indels
\item microsatelite
\item mobile element insertion polymorphism
\begin{itemize}
\item retrotranspostion: Alu elements, long interspersed nucleotide elements (LINE)
\end{itemize}
\item CNVs
\begin{itemize}
\item related to indels and microsatellites but variation in
the number of copies of larger segments of the genome
\item 1000 bp to many hundreds of kilobase pairs.
\end{itemize}
\item inversion polymorphism
\begin{itemize}
\item few base pairs up to several megabase pairs
\item can be present in either of two orientations in the genomes of different individuals
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch4_polymorphism.png}
\caption{\label{fig:org64dffa0}Polymorphism}
\end{figure}


\subsection{Mutation Types}
\label{sec:org14e27ce}
\begin{itemize}
\item cell type:
\begin{itemize}
\item germline
\item somatic
\end{itemize}
\item mutations:
\begin{itemize}
\item chromosome number
\item regional: affecting the structure or regional organization of chromosomes
\item gene: base pair substitutions, insertions, and deletions
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch4_mutation.png}
\caption{\label{fig:org2e8928f}Mutation Types and Frequency}
\end{figure}

\section{Cytogenetics}
\label{sec:orgd333612}
\subsection{Clinical Indications for Chromosome and Genome Analysis}
\label{sec:org5ffa6f5}
\subsubsection{Problems of Early Growth and Development}
\label{sec:org03dff2a}
\begin{itemize}
\item failure to thrive, developmental delay, dysmorphic facies, multiple
malformations, short stature, ambiguous genitalia, and
intellectual disability are frequent findings in children with
chromosome abnormalities
\end{itemize}
\subsubsection{Stillbirth and Neonatal Death}
\label{sec:orgd278ca0}
\begin{itemize}
\item incidence of chromosome abnormalities is much higher among
stillbirths (\(\sim\)10\%) than among live births (\(\sim\)0.7\%)
\item also elevated among infants who die in the neonatal period (\textasciitilde{}10\%)
\item karyotyping (or other comprehensive ways of scanning the genome) is
essential for accurate genetic counselling
\end{itemize}
\subsubsection{Fertility}
\label{sec:org56dca4c}
\begin{itemize}
\item chromosome studies are indicated for women presenting with
amenorrhea and for couples with a history of infertility or recurrent miscarriage
\item chromosome abnormality is seen in one or the other parent in 3\% to
6\% of cases in which there is infertility or two or more
miscarriages
\end{itemize}
\subsubsection{Family History}
\label{sec:org366cd37}
\begin{itemize}
\item known or suspected chromosome or genome abnormality in a first
degree relative is an indication for chromosome and genome analysis
\end{itemize}
\subsubsection{Neoplasia}
\label{sec:org1df1c95}
\begin{itemize}
\item Virtually all cancers are associated with one or more chromosome
abnormalities
\item chromosome and genome evaluation in the tumour itself, or in bone
marrow in the case of hematological malignant neoplasms, can offer
diagnostic or prognostic information
\end{itemize}
\subsubsection{Pregnancy}
\label{sec:org8eb6e0c}
\begin{itemize}
\item a higher risk for chromosome abnormality in fetuses conceived by
women of increased age, typically defined as \textgreater{} 35 years
\item fetal chromosome and genome analysis should be offered as a routine
part of prenatal care in such pregnancies
\item NIPT is a screening approach for the most common chromosome
disorders and is now available to pregnant women of all ages
\end{itemize}

\subsection{Chromosome Identification}
\label{sec:org1d8a09b}
\begin{itemize}
\item G-banding (Giemsa) is the gold standard for detection and
characterization of structural and numerical genomic abnormalities
\begin{itemize}
\item both constitutional (postnatal or prenatal) and acquired (cancer)
\item detection of deletions and duplications \(\ge\) 5 to 10 Mb anywhere in
the genome
\end{itemize}
\item three types of chromosomes:
\begin{description}
\item[{metacentric}] central centromere
\item[{submetacentric}] off center centromere
\item[{acrocentric}] centromere at one end
\begin{itemize}
\item 13, 14, 15, 21, 22
\end{itemize}
\end{description}
\end{itemize}

\subsection{Fluorescence In Situ Hybridization}
\label{sec:org7b30954}
\begin{itemize}
\item detecting the presence or absence of a particular DNA sequence or
for evaluating the number or organization of a chromosome or
chromosomal region \emph{in situ}
\item uses ordered collections of recombinant DNA clones containing DNA
from around the entire genome
\item limited by the need to target a specific genomic region based on a
clinical diagnosis or suspicion
\end{itemize}

\subsection{Microarrays}
\label{sec:org60ce60d}
\begin{itemize}
\item comparative genome hybridization (CGH)
\begin{itemize}
\item detects relative copy number gains and losses genome-wide by
hybridizing two samples:
\begin{itemize}
\item control genome
\item patient
\end{itemize}
\item excess of sequences from one or the other genome indicates an
overrepresentation or underrepresentation of those sequences in the
patient genome relative to the control
\end{itemize}
\item SNP arrays
\begin{itemize}
\item relative representation and intensity of alleles in different
regions of the genome indicate if a chromosome or chromosomal
region is present at the appropriate dosage
\item loss of heterozygozity
\end{itemize}
\item probe spacing provides a resolution as high as 250 kb
\end{itemize}
\subsection{Chromosome Abnormalities}
\label{sec:org422a533}
\begin{itemize}
\item numerical or structural
\item incidence of 1/154 live births
\item aneuploidy is most common
\begin{itemize}
\item associated with physical and/or mental abnormalities
\end{itemize}
\item structural abnormalities are also common
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch5_freq.png}
\caption{\label{fig:org636bd27}Incidence of Chromosomal Abnormalities}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=1.2\textwidth]{./figures/ch5_nom.png}
\caption{\label{fig:org85d6046}ISCN for Common Cytogenetic Aberration}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=1.2\textwidth]{./figures/ch5_nom2.png}
\caption{\label{fig:orga007621}ISCN (continued)}
\end{figure}

\subsection{Gene Dosage, Balance and Imbalance}
\label{sec:org2fe834e}
\begin{itemize}
\item for chromosome and genomic disorders, it is the quantitative aspects
of gene expression that underlie disease, in contrast to single-gene
disorders, in which pathogenesis often reflects qualitative aspects
of a gene's function
\end{itemize}
\subsubsection{monosomies are more deleterious than trisomies}
\label{sec:org649e99b}
\begin{itemize}
\item complete monosomies are generally not viable, except for monosomy
for the X chromosome
\item complete trisomies are viable for chromosomes 13, 18, 21, X, and Y
\end{itemize}

\subsubsection{phenotype in partial aneuploidy depends on a number of factors}
\label{sec:orgb2ad0de}
\begin{itemize}
\item size of the unbalanced segment
\item which regions of the genome are affected
\item which genes are involved
\item whether the imbalance is monosomic or trisomic
\end{itemize}
\subsubsection{risk in cases of inversions depends on the location of the inversion with respect to the centromere and on the size of the inverted segment}
\label{sec:org514f1e3}
\begin{itemize}
\item paracentric inversions do not involve the centromere
\begin{itemize}
\item very low risk for an abnormal phenotype in the next generation
\end{itemize}
\item pericentric inversions do involve the centromere
\begin{itemize}
\item risk for birth defects in offspring may be significant and
increases with the size of the inverted segment
\end{itemize}
\end{itemize}

\subsubsection{mosaic karyotype involving any chromosome abnormality, all bets are off!}
\label{sec:org14470df}
\begin{itemize}
\item the degree of mosaicism in relevant tissues or relevant stages of
development is generally unknown
\item there is uncertainty about the severity of the phenotype
\end{itemize}

\subsection{Abnormalities of Chromosome Number}
\label{sec:org72fc1cf}
\begin{description}
\item[{heteroploid}] chromosome number that is neither haploid (n=23) or diploid(2n=46)
\item[{euploid}] exact multiple of n (e.g. triploid)
\begin{itemize}
\item triploidy and tetraploidy
\begin{itemize}
\item most result from fertilization of an egg by two sperm (dispermy)
\item also failure of one of the meiotic divisions in either sex,
resulting in a diploid egg or sperm
\item maternal source are aborted
\item paternal source \(\to\) degenerative placenta (parital hydatidiform
mole) w small fetus
\end{itemize}
\end{itemize}
\item[{aneuploid}] non-multiple of n (e.g. trisomy 21)
\begin{itemize}
\item most common cause is meiotic nondisjunction in meiosis I or II (Figure \ref{fig:org5e43e96})
\begin{description}
\item[{trisomy}] 21, 18, 13
\item[{monosomy}] X (Turner syndrome)
\end{description}
\end{itemize}
\end{description}




\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch5_nondys.png}
\caption{\label{fig:org5e43e96}Nondisjunction}
\end{figure}

\subsection{Abnormalities of Chromosome Structure}
\label{sec:org339e2d2}
\begin{itemize}
\item present in 1/375 newborns
\item balanced or unbalanced
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch5_struct.png}
\caption{\label{fig:org8ff2812}Structural Rearrangements of Chromosomes}
\end{figure}

\subsubsection{Unbalanced Rearrangements}
\label{sec:orge43a492}
\begin{itemize}
\item delections and duplications
\item marker and ring chromosomes
\begin{itemize}
\item very small, unidentified chromosomes
\end{itemize}
\item isochromosomes
\begin{itemize}
\item one arm is missing and the other duplicated in a mirror-image
\end{itemize}
\item dicentric
\begin{itemize}
\item two chromosome segments, each with a centromere, fuse end to end
\end{itemize}
\end{itemize}
\subsubsection{Balanced Rearrangements}
\label{sec:orgcd94856}
\begin{itemize}
\item "balanced" depends on resolution
\item translocations
\begin{itemize}
\item reciprocal translocations
\item robertsonian translocations
\item insertions
\end{itemize}
\item inversions
\begin{itemize}
\item paracentric - outside the centromere
\item pericentric - includes the centromere
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch5_trans.png}
\caption{\label{fig:org133877a}Balanced Translocations}
\end{figure}

\section{Chromosomal and Genomic Basis of Disease}
\label{sec:orgda195ae}
\begin{itemize}
\item disorders due to:
\begin{itemize}
\item abnormal chromosome segregation (nondisjunction)
\item recurrent chromosomal syndromes, involving
deletions or duplications at genomic hot spots
\item idiopathic chromosomal abnormalities, typically \emph{de novo}
\item unbalanced familial chromosomal abnormalities
\item chromosomal and genomic events that reveal regions
of genomic imprinting
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch6_mech.png}
\caption{\label{fig:org977883e}Mechanisms of Chromosome Abnormalities and Genomic Imbalance}
\end{figure}

\subsection{Lessons from Genomic Disorders}
\label{sec:org2efb0dc}
\begin{itemize}
\item altered gene dosage for any extensive chromosomal or genomic region
is likely to result in a clinical abnormality, the phenotype of
which will, in principle, reflect haploinsufficiency for or
overexpression of one or more genes encoded within the region.
\begin{itemize}
\item in some cases the clinical presentation appears to be accounted
for by dosage imbalance for just a single gene; in other
syndromes, however, the phenotype appears to reflect imbalance for
multiple genes across the region
\end{itemize}
\item the distribution of these duplication/deletion disorders is not random
\begin{itemize}
\item segmental duplications in pericentromeric and subtelomeric
regions, predisposes particular regions to the unequal
recombination events that underlie these syndromes
\end{itemize}
\item patients carrying what appears to be the same chromosomal deletion
or duplication can present with a range of variable phenotypes
\end{itemize}

\subsection{Aneuploidy}
\label{sec:orgf432ef4}
\begin{itemize}
\item common mutation in our species involves errors in chromosome segregation
\item only three well-defined non-mosaic chromosome disorders compatible
with postnatal survival in which there is an abnormal dose of an
entire autosome:
\begin{enumerate}
\item trisomy 21 (Down syndrome)
\item trisomy 18
\item trisomy 13
\end{enumerate}
\item also have the lowest number of genes among autosomes
\end{itemize}

\subsubsection{Down Syndrome}
\label{sec:org5921cce}
\begin{itemize}
\item see Conditions
\end{itemize}
\begin{enumerate}
\item Robertsonian Translocation
\label{sec:orgeb17167}
\begin{itemize}
\item \(\sim\)4\% of Down syndrome patients have 46 chromosomes
\item one of which is a Robertsonian translocation between chromosome
21q and the long arm of one of the other acrocentric chromosomes
(usually chromosome 14 or 22)
\item 46,XX or XY,rob(14;21)(q10;q10),+21
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch6_rtgam.png}
\caption{\label{fig:org2555c8f}Chromosomes of Gametes that Theoretically can be Produced by a Carrier of a Robertsonian Translocation, rob(14;21)}
\end{figure}
\end{enumerate}

\subsubsection{Uniparental Disomy}
\label{sec:org78771f0}
\begin{itemize}
\item nondisjunction \(\to\) both copies of a chromosome derive from the same
parent
\begin{itemize}
\item called uniparental disomy
\item defined as the presence of a disomic cell line containing two
chromosomes, or portions thereof, that are inherited from only one
parent
\end{itemize}
\item isodisomy if  two chromosomes are derived from identical sister chromatids
\item heterodisomy if if both homologues from one parent are present
\item common explanation for uniparental disomy is trisomy “rescue” due to
chromosome nondisjunction in cells of a trisomic conceptus to
restore a disomic state
\end{itemize}

\subsubsection{Contiguous Gene Syndrome}
\label{sec:org12c74f5}
\begin{itemize}
\item segmental aneusomy is a form of genetic imbalance due to recurrent
subchromosomal or regional abnormalities
\begin{itemize}
\item typically detected by microarray
\item called contiguous gene syndrome
\end{itemize}
\end{itemize}

\subsection{Idiopathic Chromosome Abnormalities}
\label{sec:orgbf54e62}
\begin{itemize}
\item autosomal deletion syndromes
\begin{itemize}
\item Cri du Chat syndrome is either a terminal or interstitial deletion
of part of the short arm of chromosome 5
\end{itemize}
\item balanced translocations with developmental phenotypes
\end{itemize}

\subsection{Disorders Associated with Genomic Imprinting}
\label{sec:org6ca11f6}
\begin{itemize}
\item Prader-Willi
\item Angelman syndrome
\item Beckwith-Wiedemann syndrome
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch6_pw_as.png}
\caption{\label{fig:org58b6925}Mechanism causing Prader-Willi and Angelman Syndrome}
\end{figure}

\begin{itemize}
\item see Conditions
\end{itemize}
\subsection{Sex Chromosomes and Their Abnormalities}
\label{sec:orgeb16f32}
\begin{itemize}
\item SRY found on the Y chromosome is the major testis-determining gene
\item inactivation of an X chromosome depends on the presence of the X
inactivation center region (XIC)
\item in females structurally abnormal X chromosomes are almost always inactive
\begin{itemize}
\item nonrandom inactivation observed in X;autosome translocations
\item if balanced the normal X chromosome is preferentially inactivated
\item the two parts of the translocated chromosome remain active
\item reflecting selection against cells in which critical autosomal
genes have been inactivated
\end{itemize}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org7ed951c}Sex Chromosome Constitution and X Inactiviation}
\centering
\begin{tabular}{llrr}
phenotype & karyotype & active X & inactive X\\
\hline
\male & 46,XY; 47,Xyy & 1 & 0\\
 & 47,XXY (Klinefelter); 48,XXYY & 1 & 1\\
 & 48,XXXY; 49,XXXYY & 1 & 2\\
 & 49,XXXXY & 1 & 3\\
\hline
\female & 45,X (Turner) & 1 & 0\\
 & 46,XX & 1 & 1\\
 & 47,XXX & 1 & 2\\
 & 48,XXXX & 1 & 3\\
 & 49,XXXXX & 1 & 4\\
\end{tabular}
\end{table}

\section{Patterns of Single Gene Inheritance}
\label{sec:org7a21a63}
\begin{description}
\item[{penetrance}] the probabilty that a mutant allele(s) will have a
phenotypic expression
\begin{itemize}
\item penetrance is all or nothing (affected vs unaffected)
\end{itemize}
\item[{expressivity}] the severity of the phenotype among those with the
disease causing genotype
\end{description}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch7_pedigree.png}
\caption{\label{fig:orgbd26983}Pedigree Symbols}
\end{figure}

\subsection{Autosomal Dominant Inheritance}
\label{sec:org1cc9410}
\begin{itemize}
\item disorder occurs in every generation
\item affected person has affected parents
\begin{itemize}
\item except in \emph{de novo} mutation
\item non-penetrant
\end{itemize}
\item 50\% risk of trait in child of affected parent
\item significant number of cases due to \emph{de novo} mutations

\item pure dominant inheritance is rare
\begin{itemize}
\item generally homozygotes are more severe
\end{itemize}
\item achondroplasia is an example of incomplete dominant inheritance
\item sex limited phenotype in some disorders
\end{itemize}

\subsection{X-linked Inheritance}
\label{sec:orgdc782f1}
\begin{itemize}
\item X-linked dominant or recessive depends on phenotype in \female
\begin{itemize}
\item \(\sim\) 1/3 of XL disorders are penetrant in het \female
\end{itemize}
\end{itemize}

\subsection{Mosaicism}
\label{sec:org9474a6e}
\begin{description}
\item[{confined placental}] discrepancy between the chromosomal makeup of
the cells in the placenta and the cells in the fetus (Figure \ref{fig:org03a4d9e})
\item[{somatic}] somatic cells
\item[{germline}] germline cells
\item[{segmental}] mutation affecting morphogenesis, occuring during
embryonic development
\end{description}

\begin{table}[htbp]
\caption{\label{tab:orga724c6e}Genotypes and Phenotypes in X-linked Disease}
\centering
\begin{tabular}{lll}
 & genotype & phenotype\\
\hline
\male & hemizygous X\textsubscript{H} & unaffected\\
 & hemizygous X\textsubscript{h} & affected\\
\hline
\female & homozygous X\textsubscript{H}/X\textsubscript{H} & unaffected\\
 & heterozygous X\textsubscript{H}/X\textsubscript{h} & carrier \textpm{} affected\\
 & homozygous X\textsubscript{h}/X\textsubscript{h} & affected\\
 & compound het X\textsubscript{h1}/X\textsubscript{h2} & affected\\
\end{tabular}
\end{table}
\subsection{Mitochondrial DNA}
\label{sec:org33afa9c}
\subsubsection{Maternal Inheritance}
\label{sec:org2bc68b3}
\begin{itemize}
\item mitochondria and therefore the mtDNA, usually come from the egg
\begin{itemize}
\item the egg cell contains relatively few mitochondria
\item these mitochondria divide to populate the cells
\end{itemize}
\item sperm mitochondria enter the egg, but do not contribute genetic
information to the embryo.
\begin{itemize}
\item paternal mitochondria are marked with ubiquitin for destruction
inside the embryo.
\end{itemize}
\item mitochondria are randomly distributed to the daughter cells during
the division of the cytoplasm.
\end{itemize}

\subsubsection{Heteroplasmy}
\label{sec:orgcfa3c12}
\begin{itemize}
\item heteroplasmy is the presence of more than one type of organellar
genome within a cell or individual
\item it is an important factor in considering the severity of
mitochondrial diseases
\begin{itemize}
\item can also be beneficial
\end{itemize}
\item microheteroplasmy is present in most individuals
\begin{itemize}
\item hundreds of independent mutations, with each mutation found in
about 1–2\% of all mitochondrial genomes
\end{itemize}
\end{itemize}

\subsection{Genotype - Phenotype}
\label{sec:org08817c0}
\begin{description}
\item[{allelic heterogeneity}] different mutations in a gene cause the same phenotype
\item[{locus heterogeneity}] mutations in different genes cause the same phenotype
\item[{phenotypic heterogeneity}] different mutations in the same gene
result in different phenotypes
\end{description}

\section{Complex Traits}
\label{sec:org02fefcb}
\begin{itemize}
\item \textbf{qualitative trait} trait is present or absent
\item \textbf{quantitative trait} measurable physiological or biochemical quantity
\item relative risk ratio \(\lambda\)\textsubscript{r} is used to measure famillial
aggregation of disease in qualitative traits
\item \(\lambda\)\textsubscript{r} = prevalence of disease in relatives/prevalence in population
\item H\textsuperscript{2} is heritability
\begin{itemize}
\item fraction of phenotypic variance of a quantitative trait that is
due to allelic variation
\end{itemize}
\end{itemize}

\section{Genetic Variation in Populations}
\label{sec:org9aefbd1}
\subsection{Hardy-Weinberg}
\label{sec:orgb231422}
\begin{itemize}
\item the Hardy-Weinberg law states that the frequency of the three
genotypes AA, Aa, and aa is given by the terms of the binomial
expansion of
\begin{itemize}
\item (p + q)\textsuperscript{2} = p\textsuperscript{2} + 2pq + q\textsuperscript{2}
\begin{itemize}
\item p = the frequency of the dominant allele (represented here by A)
\item q = the frequency of the recessive allele (represented here by a)
\end{itemize}
\end{itemize}

\item for a population in genetic equilibrium:
\begin{itemize}
\item p + q = 1 (sum of the frequencies of both alleles is 100\%)
\item (p + q)\textsuperscript{2} = 1
\item p\textsuperscript{2} + 2pq + q\textsuperscript{2} = 1
\end{itemize}
\item the three terms of this binomial expansion indicate the frequencies
of the three genotypes (Table \ref{tab:orgc456d18})
\begin{itemize}
\item p\textsuperscript{2} = frequency of AA
\item 2pq = frequency of Aa
\item q\textsuperscript{2} = frequency of aa
\item q = frequency of affected males in XL disease
\end{itemize}
\end{itemize}



\begin{table}[htbp]
\caption{\label{tab:orgc456d18}Population Frequency in Modes of Inheritance}
\centering
\begin{tabular}{llll}
inheritance & frequency & genotype & \\
\hline
AR & q\textsuperscript{2} & aa & homozygote\\
AD & 2pq & Aa & heterozygotes\\
XL & q\footnotemark & X\textsubscript{M}Y & hemizygote \male\\
\end{tabular}
\end{table}\footnotetext[1]{\label{orgb05d8f1}q = frequency of affected \male{} in XL disease}


\begin{itemize}
\item the Hardy-Weinberg law rests on these assumptions:
\begin{itemize}
\item the population under study is large and matings are random with
respect to the locus in question
\item allele frequencies remain constant over time because of the
following:
\begin{itemize}
\item there is no appreciable rate of new mutation
\item individuals with all genotypes are equally capable of mating and
passing on their genes
\begin{itemize}
\item \(\therefore\) no selection against	any particular genotype
\end{itemize}
\item there has been no significant immigration of individuals from a
population with allele frequencies very different from the
endogenous population
\end{itemize}
\end{itemize}
\item a population that reasonably appears to meet these assumptions is
considered to be in Hardy-Weinberg equilibrium
\begin{itemize}
\item population genotype frequencies from generation to generation will
remain constant if the allele frequencies p and q remain constant
\end{itemize}
\end{itemize}

\subsection{Factors That Disturb Hardy-Weinberg Equilibrium}
\label{sec:org9f03733}
\subsubsection{exceptions to large populations with random mating}
\label{sec:orgc1479af}
\begin{description}
\item[{stratification}] isolation by distance
\item[{assortative mating}] social factors
\item[{consaguinity}] 
\end{description}
\subsubsection{exceptions to constant allele frequencies}
\label{sec:org3b5e2d9}
\begin{itemize}
\item mutation
\item selection and fitness
\begin{itemize}
\item selection in dominant disease
\item selection in recessive disease - less effect
\end{itemize}
\item mutation and selection balance in dominant disease:
\begin{itemize}
\item \(\mu\) = sq
\begin{itemize}
\item \(\mu\) = mutation rate per generation
\item q = mutant allele freq
\item s = coefficient of selection
\end{itemize}
\end{itemize}
\item mutation and selection balance in X-linked recessive mutations
\begin{itemize}
\item 1/3 rule
\end{itemize}
\end{itemize}

\subsection{1/3 Rule}
\label{sec:org2a2e30b}
\begin{itemize}
\item \male{} have one X chromosome and \female{} two, the pool of X-linked
alleles in the entire population's gene pool is partitioned at any
given time, with one third of mutant alleles present in \male{} and
two thirds in \female
\item if the reproductive fitness of a \male affected with an X-linked
recessive disorder is low or nil, then in a population
\begin{itemize}
\item \textbf{one-third of all affected X chromosomes will be removed from
the gene pool every} \textbf{generation}
\end{itemize}
\item if the incidence of a serious X-linked disease is not changing and
selection is operating against (and only against) hemizygous \male,
the mutation rate \(\mu\), must equal the coefficient of selection, s
times q, the allele frequency, adjusted by a factor of 3 because
selection is operating only on the third of the mutant alleles in
the population that are present in males at any time. 
\begin{itemize}
\item \(\mu\) = sq/3
\end{itemize}
\item \textbf{if the incidence of the disease is constant, then one-third of
cases must be due to mutations arising \emph{de novo} in a family}
\end{itemize}

\section{Identifying the Genetic Basis for Human Disease}
\label{sec:org2557175}
\begin{itemize}
\item linkage analysis is family-based
\begin{itemize}
\item takes advantage of family pedigrees to follow the inheritance of a
disease among family members and to test for consistent, repeated
coinheritance of the disease with a particular genomic region or
even with a specific variant or variants, whenever the disease is
passed on in a family
\end{itemize}
\item association analysis is population-based
\begin{itemize}
\item does not depend explicitly on pedigrees but instead takes
advantage of the entire history of a population to look for
increased or decreased frequency of a particular allele or set of
alleles in a sample of affected individuals taken from the
population, compared with a control set of unaffected people from
that same population
\item particularly useful for complex diseases that do not show a
mendelian inheritance pattern
\end{itemize}
\item direct genome sequencing of affected individuals and their parents
and/or other individuals in the family or population
\begin{itemize}
\item particularly useful for rare mendelian disorders in which linkage
analysis is not possible because there are simply not enough such
families to do linkage analysis or because the disorder is a
genetic lethal that always results from new mutations and is
never inherited
\end{itemize}

\item \textbf{linkage} is the term used to describe a departure from the
independent assortment of two loci
\item analysis of linkage depends on determining the frequency of
recombination as a measure of how close two loci are to each other
on a chromosome.
\item a common notation for recombination frequency is \(\theta\)
\begin{itemize}
\item where \(\theta\) varies from 0 (no recombination at all) to 0.5 (independent assortment)
\item if two loci are so close together that \(\theta\) = 0 between them
they are said to be completely linked
\item if they are so far apart that θ = 0.5 they are assorting
independently and are unlinked
\item in between these two extremes are various degrees of linkage
\end{itemize}

\item map distance between two loci is a theoretical concept that is based
on the extent of observed recombination, \(\theta\), between the
loci
\begin{itemize}
\item measured in units called centimorgans (cM)
\item defined as the genetic length over which, on average, one
crossover occurs in 1\% of meioses
\item \(\therefore\) a recombination fraction of 1\% (\(\theta\) = 0.01)
translates approximately into a map distance of 1 cM
\end{itemize}

\item linkage disequilibrium (LD) is the due to close map distance between loci
\begin{itemize}
\item frequency of a haplotype, freq(A-S) for example, may not be equal
to the product of the frequencies of the individual alleles that
make up that haplotype
\item in this situation, freq(A-S) \(\neq\) freq(A) x freq(S)
\begin{itemize}
\item the alleles are in LD
\end{itemize}
\end{itemize}
\end{itemize}
\section{Molecular Basis of Genetic Disease}
\label{sec:org1552d83}
\begin{itemize}
\item effect of mutation on protein function
\begin{description}
\item[{loss of function}] \(\downarrow\) protein function (i.e. CF)
\begin{itemize}
\item almost always recessive
\end{itemize}
\item[{gain of function}] \(\uparrow\) expression or \(\uparrow\) activity (i.e BCR-ABL)
\begin{itemize}
\item almost always dominant
\end{itemize}
\item[{novel property}] i.e. sickle cell
\item[{heterochronic expression}] wrong time i.e. hereditary persistence
of Hb F
\item[{ectopic expression}] wrong place i.e oncogene
\end{description}
\end{itemize}

\section{Treatment of Genetic Disease}
\label{sec:org1bc69e4}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch13_treatment.png}
\caption{\label{fig:orgcaa949d}Levels of Treatment in Genetic Disease}
\end{figure}

\subsection{Manipulation of Metabolism}
\label{sec:orgef8bde7}
\begin{description}
\item[{substrate reduction}] diet in PKU
\item[{replacement}] T4 in CH
\item[{diversion}] sodium benzoate in UCD
\item[{enzyme inhibition}] nitisinone in Tyrosinemia I
\item[{receptor antagonism}] TGF-\(\beta\) in Marfan
\item[{depletion}] aphoresis in homozygous familial hypercholesterolemia
\end{description}

\subsection{Function of Affected Gene or Protein}
\label{sec:org008c54d}
\begin{description}
\item[{skipping nonsense codons}] CF
\item[{folding}] CF
\item[{membrane trafficking}] CF
\item[{protein augmentation}] hemophilia
\item[{ERT}] Gaucher Type I, ADA
\end{description}

\subsection{Modulation of Gene Expression}
\label{sec:org5d49bf1}
\begin{description}
\item[{DNA methylation}] \(\uparrow\) HbF in \(\beta\)-thalassemias
\item[{siRNA}] Huntington
\item[{exon skipping}] DMD
\item[{gene editing}] CRISPR/Cas9
\end{description}

\subsection{Transplantation}
\label{sec:org9c39a00}
\begin{description}
\item[{HSCT}] Hurler, SCID
\item[{Liver}] \(\alpha\)1AT, UCD
\end{description}

\subsection{Gene Therapy Risks}
\label{sec:org71ee123}
\begin{itemize}
\item adverse response to vector
\item insertional mutagenesis \(\to\) malignancy
\item insertional inactivation of essential gene
\end{itemize}

\section{Developmental Genetics}
\label{sec:orge6f2644}
\subsection{Malformations}
\label{sec:org26fef03}
\begin{itemize}
\item arise from intrinsic defects in genes that specify a series of
developmental steps or programs, and because such programs are often
used more than once in different parts of the embryo or fetus at
different stages of development, a malformation in one part of the
body is often but not always associated with malformations elsewhere
as well
\item polydactyly (extra fingers or toes) and syndactyly (fused fingers)
are malformation
\end{itemize}
\subsection{Deformations}
\label{sec:orgfde65b6}
\begin{itemize}
\item caused by extrinsic factors impinging physically on the fetus during
development
\item common during the second trimester of development when the fetus is
constrained within the amniotic sac and uterus
\begin{itemize}
\item for example contractions of the joints of the extremities, known
as arthrogryposes, in combination with deformation of the
developing skull, occasionally accompany constraint of the fetus
due to twin or triplet gestations or prolonged leakage of amniotic
fluid
\end{itemize}
\item most deformations apparent at birth either resolve spontaneously or
can be treated by external fixation devices to reverse the effects
of the instigating cause
\end{itemize}
\subsection{Disruptions}
\label{sec:org2c82860}
\begin{itemize}
\item result from destruction of irreplaceable normal fetal
tissue
\item more difficult to treat than deformations because they involve
actual loss of normal tissue
\item may be the result of vascular insufficiency, trauma, or
teratogens
\item one example is amnion disruption, the partial amputation of a fetal
limb associated with strands of amniotic tissue
\begin{itemize}
\item amnion disruption is often recognized clinically by the presence of
partial and irregular digit amputations in conjunction with
constriction rings
\end{itemize}
\end{itemize}

\section{Cancer}
\label{sec:org4136c08}
\begin{itemize}
\item cancer is not a single disease but rather comes in many forms and
degrees of malignancy
\item there are three main classes of cancer:
\begin{description}
\item[{sarcomas}] tumour has arisen in mesenchymal tissue, such as bone,
muscle, connective tissue or nervous system
\item[{carcinomas}] originate in epithelial tissue, such as the cells
lining the intestine, bronchi, or mammary ducts
\item[{hematopoietic and lymphoid}] leukemia and lymphoma which spread
throughout the bone marrow, lymphatic system, and peripheral
blood
\end{description}
\end{itemize}
\subsection{Genetic Basis of Cancer}
\label{sec:org575a88e}
\begin{itemize}
\item driver and passenger mutations
\item particular translocations are sometimes highly specific for certain
types of cancer and involve specific genes
\begin{itemize}
\item BCR-ABL translocation in chronic myelogenous leukemia
\end{itemize}
\item cancers can also show complex rearrangements in which chromosomes
break into numerous pieces and rejoin, forming novel and complex
combinations (a process known as “chromosome shattering”)
\item large genomic alterations involving many kilobases of DNA can form
the basis for loss of function or increased function of one or more
driver genes

\item genes in which mutations cause cancer are referred to as \textbf{driver}
\textbf{genes}, and the cancer causing mutations in these genes are \textbf{driver}
\textbf{mutations}
\item driver genes fall into two distinct categories
\begin{enumerate}
\item activated oncogenes
\item tumour suppressor genes (TSGs)
\end{enumerate}

\item an activated oncogene is a mutant allele of a proto-oncogene, a
class of normal cellular protein-coding genes that promotes growth
and survival of cells
\item oncogenes facilitate malignant transformation by stimulating
proliferation or inhibiting apoptosis
\item oncogenes encode proteins such as the following:
\begin{itemize}
\item proteins in signaling pathways for cell proliferation
\item transcription factors that control the expression of growth-promoting genes
\item inhibitors of programmed cell death machinery
\end{itemize}
\item a TSG is a gene in which loss of function through mutation or
epigenomic silencing directly removes normal regulatory controls on
cell growth or leads indirectly to such losses through an increased
mutation rate or aberrant gene expression
\item TSGs encode proteins involved in many aspects of cellular function including:
\begin{itemize}
\item maintenance of correct chromosome number and structure
\item DNA repair proteins
\item proteins involved in regulating the cell cycle, cellular
proliferation or contact inhibition
\end{itemize}

\item tumour initiation can be caused by different types of genetic
alterations:
\begin{itemize}
\item activating or gain-of-function mutations
\item ectopic and heterochronic mutations of protooncogenes
\item chromosome translocations that cause misexpression of genes or chimeric genes
\item LOF of both alleles, or a dominant negative mutation of one allele, of TSGs
\end{itemize}

\item tumour progression occurs as a result of accumulating additional
genetic damage,through mutations or epigenetic silencing, of driver
genes that encode the machinery that repairs damaged DNA and
maintains cytogenetic normality
\end{itemize}

\subsection{Cancer in Families}
\label{sec:org1e20931}
\begin{itemize}
\item germline mutation - inherited
\item second-hit
\begin{itemize}
\item somatic mutation
\item loss of heterozygozity around locus
\end{itemize}
\end{itemize}

\subsection{Sporadic Cancer}
\label{sec:org8bf3f3e}
\begin{itemize}
\item activation of oncogenes by point mutation
\item activation of oncogenes by chromosome translocation
\begin{itemize}
\item best-known example is the translocation between chromosomes 9 and
22 (Philadelphia chromosome) that is seen in CML
\item moves the protooncogene ABL1, a tyrosine kinase, from its normal
position on chromosome 9q to a gene of unknown function, BCR, on
chromosome 22q
\item results in the synthesis of a novel, chimeric protein, BCR-ABL1 w
enhanced tyrosine kinase activity
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch15_neo.png}
\caption{\label{fig:org0776074}Characteristic Chromosome Translocations in Selected Human Malignant Neoplasms}
\end{figure}

\subsection{Cytogenetic Changes in Cancer}
\label{sec:orge873bc1}
\subsubsection{Aneuploidy and Aneusomy}
\label{sec:orge854078}
\begin{itemize}
\item aneuploidy - abnormal chromosome number
\item aneusomy - cells with different numbers of chromosomes
\item when CML, with the 9;22 Philadelphia chromosome, evolves from the
typically indolent chronic phase to a severe, life-threatening blast
crisis, there may be several additional cytogenetic abnormalities,
including numerical or structural changes, such as a second copy of
the 9;22 translocation chromosome or an isochromosome for 17q

\item another cytogenetic aberration seen in many cancers is gene
amplification, a phenomenon in which many additional copies of a
segment of the genome are present in the cell
\begin{itemize}
\item \textbf{double minutes} (very small accessory chromosomes)
\item \textbf{homogeneously staining regions} that do not band normally and
contain multiple, amplified copies of a particular DNA segment
\item known to include extra copies of proto-oncogenes such as the genes
encoding Myc, Ras, and epithelial growth factor receptor, which
stimulate cell growth, block apoptosis, or both
\end{itemize}
\end{itemize}

\subsection{Targeted Cancer Therapy}
\label{sec:org2bcf934}
\begin{itemize}
\item imatinib is a highly effective inhibitor of a number of tyrosine
kinases including the ABL1 kinase in CML
\begin{itemize}
\item prolonged remissions of this disease have been seen, in some cases
with apparently indefinite postponement of the transformation into a
virulent acute leukemia (blast crisis) that so often meant the end
of a CML patient's life
\end{itemize}
\item additional kinase inhibitors have been developed to target other
activated oncogene driver genes in a variety of tumour types
\item initial results with targeted therapies, although very promising in
some cases, have not led to permanent cures in most patients because
tumours develop resistance to the targeted therapy
\item the outgrowth of resistant tumours because cancer cells are highly
mutable, and their genomes undergo recurrent mutation
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch15_target.png}
\caption{\label{fig:org2af63a4}Targeted Cancer Treatment}
\end{figure}

\section{Calculation of Genetic Risk}
\label{sec:orged11e80}
\begin{description}
\item[{prior probability}] depends on known data or stated scenario
\begin{itemize}
\item population risk, affected or carrier parents
\end{itemize}
\item[{conditional probability}] is based on observed outcomes
\begin{itemize}
\item i.e. probability of 4 unaffected children
\end{itemize}
\item[{joint probability}] probability of outcome given prior
\begin{itemize}
\item i.e. probability of 4 unaffected children with carrier parent
\end{itemize}
\end{description}
\[
  joint_x = conditional_x \cdot prior_x
\]
\begin{description}
\item[{posterior probability}] fraction of total joint probability for a scenario
\begin{itemize}
\item i.e  probability daughter is a carrier with four unaffected sibs
\end{itemize}
\end{description}
\[
 posterior_x = \frac{joint_x}{\sum_{i=1}^n joint_i}
\]  
\subsection{Approach to Problems}
\label{sec:org9614b7e}
\begin{enumerate}
\item what is the question?
\begin{itemize}
\item the answer is posterior probability for that scenario
\begin{itemize}
\item i.e risk that daughter is a carrier
\end{itemize}
\end{itemize}
\item what are the scenarios for the unknown components in the problem
\begin{itemize}
\item these will become the denominator in the posterior probability
\begin{itemize}
\item mom carrier, daughter carrier
\item mom carrier, daughter non-carrier
\item mom non-carrier, daughter non-carrier
\end{itemize}
\end{itemize}
\item what are the prior probabilities for each scenario?
\begin{itemize}
\item these are either given or are set for that scenario
\begin{itemize}
\item when an status (affected, carrier, etc) is unknown each option
forms a scenario and becomes a prior for that scenario
\end{itemize}
\end{itemize}
\item calculate the conditional probability for each scenario
\begin{itemize}
\item simple ones are the probability of inheriting
\begin{itemize}
\item 1/4 for AR
\item 1/2 from mom in XL
\end{itemize}

\item based on evidence such as number of unaffected children
\end{itemize}
\item calculate the joint probability for each scenario
\item calculate the posterior probability for each scenario
\end{enumerate}

\subsection{X linked}
\label{sec:org50636d3}
\begin{itemize}
\item transmission
\begin{itemize}
\item daughters always inherit dad's X
\begin{itemize}
\item P(transmission) = 1
\end{itemize}
\item moms transmit one of 2 Xs
\begin{itemize}
\item P(transmission) = 1/2
\end{itemize}
\item sons have phenotype
\item daughters are carriers (X-linked recessive)
\item prior probability that female is a carrier of X linked lethal disorder 
\begin{itemize}
\item H is the population frequency of female carriers
\end{itemize}
\item there are three ways a female can be a carrier:
\begin{enumerate}
\item inherits from a carrier mother
\begin{itemize}
\item 1/2  H
\end{itemize}
\item receives a new mutant allele on X from mom
\begin{itemize}
\item \(\mu\)
\end{itemize}
\item receives a new mutant allele on X from dad
\begin{itemize}
\item \(\nu\)
\item often simplified as \(\nu\) = \(\mu\)
\end{itemize}
\end{enumerate}
\end{itemize}
\end{itemize}
\[H = (1/2 \cdot H) + \mu + \mu \]
\[H = H/2 +2\mu \]
\[2H = H +4\mu \]
\[H = 4\mu \]

\begin{itemize}
\item the incidence of carrier females in next generation (C\textsubscript{n+1}) will
be 1/2 the previous generation (C\textsubscript{n}) plus the mutation rate in
females (\(\mu\)) plus the mutation rate in males (\(\nu\))
\end{itemize}

\[C_{n+1} = 1/2 \cdot C_n + \mu + \nu \]

\begin{itemize}
\item same as above
\end{itemize}

\begin{center}
\begin{tabular}{lll}
I-2 & C & NC\\
\hline
prior & 4\(\mu\) & 1\\
cond \footnotemark & 1/2 & \(\mu\)\\
joint & 2\(\mu\) & \(\mu\)\\
post & 2/3 & 1/3\\
\end{tabular}
\end{center}\footnotetext[2]{\label{org0f23037}prop of affected son}


\begin{itemize}
\item 2/3 inherited from mom
\item 1/3 \emph{de novo}
\end{itemize}

\subsection{AR}
\label{sec:org38b0eae}
\begin{itemize}
\item carrier risk for unaffected sibs of patient w AR disease
\begin{itemize}
\item P(sib is carrier) = 2/3
\item P(parent is carrier) = 1
\end{itemize}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org13b9654}Punnett Square for AR Disease}
\centering
\begin{tabular}{lll}
 & A & a\\
A & AA & Aa\\
a & Aa & \textbf{aa}\\
\end{tabular}
\end{table}

\section{Prenatal Diagnosis and Screening}
\label{sec:org68dfe4d}
\begin{itemize}
\item placental mosaicism can result in CVS karyotype results that do not
reflect the fetus (Figure \ref{fig:org03a4d9e})
\begin{itemize}
\item confined placental mosaicism can lead to incorrect interpretation
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./figures/ch17_mosaic.png}
\caption{\label{fig:org03a4d9e}Mosaicism Detected in Prenatal Diagnsosis}
\end{figure}


\begin{table}[htbp]
\caption{\label{tab:org92e868c}First Trimester Screening Tests}
\centering
\begin{tabular}{llll}
disorder & NT & PAPP-A & \(\beta\)-hCG\\
\hline
T21 & \(\uparrow\) & \(\downarrow\) & \(\uparrow\)\\
T18 & \(\uparrow\) & \(\downarrow\) & \(\downarrow\)\\
T13 & \(\uparrow\) & \(\downarrow\) & \(\downarrow\)\\
NTD & - & - & -\\
\end{tabular}
\end{table}


\begin{table}[htbp]
\caption{\label{tab:org29c5826}Second Trimester Screening Tests}
\centering
\begin{tabular}{lllll}
disorder & uE\textsubscript{3} & AFP & hCG & Inhibin A\\
\hline
T21 & \(\downarrow\) & \(\downarrow\) & \(\uparrow\) & \(\uparrow\)\\
T18 & \(\downarrow\) & \(\downarrow\) & \(\downarrow\) & -\\
T13 & \(\downarrow\) & \(\downarrow\) & \(\downarrow\) & -\\
NTD & - & \(\Uparrow\) & - & -\\
\end{tabular}
\end{table}

\section{Ethics}
\label{sec:org19af605}

\subsection{Principles of Biomedical Ethics}
\label{sec:orgca2df9f}
\begin{itemize}
\item four cardinal principles are frequently considered in any discussion
of ethical issues in medicine:
\begin{description}
\item[{respect for individual autonomy}] safeguarding an individual's
rights to control their medical care and medical information, free
of coercion
\item[{beneficence}] doing good
\item[{avoidance of maleficence}] “first of all, do no harm”
\item[{justice}] ensuring that all individuals are treated equally and
fairly
\end{description}
\end{itemize}


\section{Nomenclature}
\label{sec:orgf9461ca}
\subsection{General Recommendations}
\label{sec:org758ed2d}
\begin{itemize}
\item all variants should be described at the most basic level, the DNA
level. Descriptions at the RNA and/or protein level may be given in
addition
\begin{itemize}
\item descriptions should make clear whether the change was
experimentally determined or theoretically deduced by giving
predicted consequences in parentheses
\item descriptions at RNA/protein level should describe the changes
observed on that level (RNA/protein) and not try to incorporate
any knowledge regarding the change at DNA-level
\end{itemize}
\item all variants should be described in relation to an accepted
reference sequence
\begin{itemize}
\item the reference sequence file used should be public and clearly
described
\begin{itemize}
\item when variants are not reported in relation to a genomic
reference sequence from a recent genome build, the preferred
reference sequence is a Locus Reference Genomic sequence (LRG)
\item when no LRG is available, one should be requested
\item the reference sequence used must contain the residue(s) described to be changed.
\end{itemize}
\item a letter prefix is mandatory to indicate the type of reference sequence used
\begin{itemize}
\item "c." for a coding DNA reference sequence
\item "g." for a linear genomic reference sequence
\item "m." for a mitochondrial DNA reference sequence
\item "n." for a non-coding DNA reference sequence
\item "o." for a circular genomic reference sequence
\item "p." for a protein reference sequence
\item "r." for an RNA reference sequence (transcript)
\end{itemize}
\item numbering of the residues (nucleotide or amino acid) in relation to the reference sequence used should follow the approved scheme
\item 3'rule: for all descriptions the most 3' position possible of the reference sequence is arbitrarily assigned to have been changed
\begin{itemize}
\item the 3'rule also applies for changes in single residue stretches and tandem repeats (nucleotide or amino acid)
\item the 3'rule applies to ALL descriptions (genome, gene, transcript and protein) of a given variant
\item exception: deletion/duplication around exon/exon junctions using c., r. or n. reference sequences
\end{itemize}
\item descriptions at DNA, RNA and protein level are clearly different:
\begin{itemize}
\item DNA-level 123456A>T: number(s) referring to the nucleotide(s)
affected, nucleotides in capitals using IUPAC-IUBMB assigned
nucleotide symbols
\item RNA-level 76a>u: number(s) referring to the nucleotide(s) affected, nucleotides in lower case using IUPAC-IUBMB assigned nucleotide symbols
\item protein level Lys76Asn: the amino acid(s) affected in three- or one-letter code followed by a number IUPAC-IUBMB assigned amino acid symbols
\begin{itemize}
\item three-letter amino acid code is preferred
\item the "*" can be used to indicate the translation stop codon in both one- and three-letter amino acid code descriptions
\end{itemize}
\end{itemize}
\item prioritisation: when a description is possible according to several types, the preferred description is:
\begin{enumerate}
\item substitution
\item deletion
\item inversion
\item duplication
\item conversion
\item 6) insertion
\end{enumerate}
\item when a variant can be described as a duplication or an insertion,
prioritisation determines it should be described as a duplication
\item descriptions removing part of a reference sequence replacing it
with part of the same sequence are not allowed
(e.g. NM\textunderscore004006.2:c.[762\textunderscore768del;767\textunderscore774dup])
\end{itemize}
\item only approved HGNC gene symbols should be used to describe genes
\end{itemize}

\subsection{Characters Used}
\label{sec:orgc9f46dd}

\begin{itemize}
\item in HGVS nomenclature some characters have a specific meaning

\begin{itemize}
\item "+" (plus) is used in nucleotide numbering; c.123+45A>G
\item "-" (minus) is used in nucleotide numbering; c.124-56C>T
\item "\textbf{" (asterisk) is used in nucleotide numbering and to indicate a
translation termination (stop) codon; c.*32G>A and P.Trp41}
\item "\textunderscore" (underscore) is used to indicate a range; g.12345\textunderscore12678del
\item "[ ]" (square brackets) are used for alleles, which includes
multiple inserted sequences at one position and insertions from a
second reference sequence
\begin{itemize}
\item ";" (semi colon) is used to separate variants and alleles;
g.[123456A>G;345678G>C] or g.[123456A>G];[345678G>C]
\item "," (comma) is used to separate different transcripts/proteins
derived from one allele; r.[123a>u, 122\textunderscore154del]
\item NC\textunderscore000002.11:g.48031621\textunderscore48031622ins[TAT;48026961\textunderscore48027223;GGC]
\item NC\textunderscore000002.11:g.47643464\textunderscore47643465ins[NC\textunderscore000022.10:35788169\textunderscore35788352]
\end{itemize}
\item ":" (colon) is used to separate the reference sequence file
identifier (accession.version\textunderscorenumber) from the
actual description of a variant;
NC\textunderscore000011.9:g.12345611G>A
\item "::" (double colon) is used to describe RNA fusion transcripts
(RNA Deletion-insertion) and to designate break point junctions
creating a ring chromosome (DNA Complex (HGVS/ISCN))
\item "( )" (parentheses) are used to indicate uncertainties and
predicted consequences;
NC\textunderscore000023.9:g.(123456\textunderscore234567)\textunderscore(345678\textunderscore456789)del, p.(Ser123Arg)
\begin{itemize}
\item NOTE: the range of the uncertainty should be described as precisely as possible
\end{itemize}
\item "?" (question mark) is used to indicate unknown positions (nucleotide or amino acid); g.(?\textunderscore234567)\textunderscore(345678\textunderscore?)del
\item "\^{}" (caret) is used as "or"; c.(370A>C\textsuperscript{\^{}}372C>R) as back translation of p.Ser124Arg (i.e. changing the AGC codon to CGC, AGG or AGA)
\item ">" (greater than) is used to describe substitution variants (DNA
and RNA level); g.12345A>T, r.123a>u
\item "DNA and RNA level); g.123456G>A, r.123c>u (see DNA, RNA)
\begin{itemize}
\item a substitution at the protein level is described as p.Ser321Arg
\end{itemize}
\item "del" indicates a deletion; c.76delA
\item "dup" indicates a duplication; c.76dupA
\item "ins" indicates an insertion; c.76\textunderscore77insG
\begin{itemize}
\item duplicating insertions are described as duplications, not as insertions
\end{itemize}
\item "inv" indicates an inversion; c.76\textunderscore83inv
\begin{itemize}
\item not used at protein level, usually described as "delins"
\end{itemize}
\item "con" indicates a conversion; NC\textunderscore000022.10:g.42522624\textunderscore42522669con42536337\textunderscore42536382
\item "fs" indicates a frame shift; p.Arg456GlyfsTer17 (or p.Arg456Glyfs*17)
\item "ext" indicates an extension; p.Met1ext-5
\item HGVS/ISCN 
\begin{itemize}
\item "cen" indicates the centromere of a chromosome
\item "chr" indicates a chromosome; chr11:g.12345611G>A (NC\textunderscore000011.9)
\item "pter indicates the first nucleotide of a chromosome
\item "qter" indicates the last nucleotide of a chromosome
\item "sup" indicates an supernumary chromosome (marker chromosome)
\end{itemize}
\item changes of state (modifications)
\item "gom" indicates a gain of methylation; g.12345678\textunderscore12345901|gom
\begin{itemize}
\item "lom" indicates a loss of methylation; g.12345678\textunderscore12345901|lom
\item "met" indicates a methylation; g.12345678\textunderscore12345901|met=
\end{itemize}
\end{itemize}
\end{itemize}

\section{Variant Classification}
\label{sec:orga2681eb}
\subsection{Pathogenic Variant}
\label{sec:org5617b28}
\begin{itemize}
\item the variant is considered the cause of the patient's disease
\end{itemize}
\subsubsection{Main evaluation criteria}
\label{sec:org41ae77b}
\begin{itemize}
\item the variant is well established as disease causing in the
databases and literature, and a wide consensus on the variant's
pathogenicity exists
\item in these cases, significant family segregation has been verified
and several publications support pathogenicity
\end{itemize}
\subsubsection{Recommendations for clinical usage}
\label{sec:orgbcb1e9c}
\begin{itemize}
\item this genetic information can be used independently in clinical
judgment and in evaluating risks for family members
\end{itemize}

\subsection{Likely Pathogenic Variant}
\label{sec:org57163e0}
\begin{itemize}
\item the identified variant is considered the probable cause of the patient's disease
\item this information should be used cautiously for clinical
decision-making, as there is still a degree of uncertainty
\end{itemize}
\subsubsection{Main evaluation criteria}
\label{sec:org5f1527d}
\begin{itemize}
\item a clear genotype-phenotype correlation exists
\item in these cases, it is essential to have thorough background
information from the referring clinician about the patient's
phenotype, which helps to determine the probable pathogenicity
\item the variant typically results in premature truncation in a gene
where loss of function has been established as a mechanism of
pathogenicity for the patient's suspected disease
\item the variant is an missense, which is predicted deleterious by the
majority of in silico tools applied
\item the variant is novel or very rare in control populations
\end{itemize}

\subsubsection{Recommendations for clinical usage}
\label{sec:orgf78515e}
\begin{itemize}
\item the variant alone should not be used for family risk
stratification
\item a likely pathogenic variant could be used to rationalize family
member risk stratification and a follow-up strategy on a
case-by-case basis
\item this could include additional genetic counseling after two to five
years to evaluate the status of the variant
\item family member testing may offer new evidence to support further
classification of the variant as pathogenic
\end{itemize}


\subsection{Variant of Uncertain Significance (VUS)}
\label{sec:org6291247}
\begin{itemize}
\item The variant has characteristics of being an independent
disease-causing mutation, but insufficient or conflicting evidence
exists
\end{itemize}

\subsubsection{Recommendations for clinical usage}
\label{sec:orgf445645}
\begin{itemize}
\item management of the patient and their family should be based on clinical judgment
\item this genetic information should not be used for family risk
stratification, and we do not recommend family member testing in a
diagnostic setting
\end{itemize}


\subsection{Likely Benign Variant}
\label{sec:orge93c272}
\begin{itemize}
\item the variant is not likely to be the cause of the tested disease
\end{itemize}

\subsubsection{Main evaluation criteria}
\label{sec:org28a5c6f}
\begin{itemize}
\item taking disease prevalence and penetrance into account, the minor
allele frequency (MAF) in control populations is considerable (MAF <
0.001)
\end{itemize}

\subsubsection{Recommendations for clinical usage}
\label{sec:org6d5662e}
\begin{itemize}
\item tests with only likely benign variants are considered a negative
test result
\end{itemize}

\subsection{Benign Variant}
\label{sec:orgd26b2ae}
\begin{itemize}
\item the variant is not considered to be the cause of the tested disease
\end{itemize}
\subsubsection{Main evaluation criteria}
\label{sec:org2069fc1}
\begin{itemize}
\item it is evident that the variant does not segregate with the disease
in families with two or more affected individuals
\end{itemize}
\subsubsection{Recommendations for clinical}
\label{sec:orge14d312}
\begin{itemize}
\item tests with only likely benign variants are considered a negative
test result
\end{itemize}
\end{document}