% Created 2019-11-18 Mon 09:28
% Intended LaTeX compiler: pdflatex
\documentclass{scrartcl}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{graphicx}
\usepackage{grffile}
\usepackage{longtable}
\usepackage{wrapfig}
\usepackage{rotating}
\usepackage[normalem]{ulem}
\usepackage{amsmath}
\usepackage{textcomp}
\usepackage{amssymb}
\usepackage{capt-of}
\usepackage{hyperref}
\hypersetup{colorlinks,linkcolor=black,urlcolor=blue}
\usepackage{textpos}
\usepackage{textgreek}
\usepackage[version=4]{mhchem}
\usepackage{chemfig}
\usepackage{siunitx}
\usepackage{gensymb}
\usepackage[usenames,dvipsnames]{xcolor}
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage{verbatim}
\usepackage{tikz}
\usepackage{wasysym}
\usetikzlibrary{shapes.geometric,arrows,decorations.pathmorphing,backgrounds,positioning,fit,petri}
\usepackage{fancyhdr}
\pagestyle{fancy}
\author{Matthew Henderson, PhD, FCACB}
\date{\today}
\title{Mitochondrial Disease}
\hypersetup{
 pdfauthor={Matthew Henderson, PhD, FCACB},
 pdftitle={Mitochondrial Disease},
 pdfkeywords={},
 pdfsubject={},
 pdfcreator={Emacs 26.1 (Org mode 9.1.9)}, 
 pdflang={English}}
\begin{document}

\maketitle
\tableofcontents


\section{Mitochondria}
\label{sec:org379bc4c}
\subsection{Background}
\label{sec:org5664ccb}
\begin{enumerate}
\item Mitochondria
\label{sec:org2b4e51f}
\begin{itemize}
\item A double-membrane-bound organelle found in most eukaryotic organisms
\begin{itemize}
\item liver cells can have more than 2000
\item red blood cells have no mitochondria
\end{itemize}

\item The organelle is composed of an outer membrane, intermembrane
space, and inner membrane with cristae and matrix.

\item The mitochondrion has its own independent genome that shows
substantial similarity to bacterial genomes

\item Mitochondrial proteins transcribed from mtDNA vary depending on the
tissue and the species.

\item In humans, 615 distinct types of protein have been identified from
cardiac mitochondria.
\end{itemize}

\item Maternal Inheritance
\label{sec:orgb71ba96}
\begin{itemize}
\item Mitochondria and therefore the mtDNA, usually come from the egg
\begin{itemize}
\item the egg cell contains relatively few mitochondria
\item these mitochondria divide to populate the cells
\end{itemize}
\item Sperm mitochondria enter the egg, but do not contribute genetic
information to the embryo.
\begin{itemize}
\item paternal mitochondria are marked with ubiquitin for destruction
inside the embryo.
\end{itemize}
\item mitochondria are randomly distributed to the daughter cells during
the division of the cytoplasm.
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./mitochondria/figures/Mitochondrial_Bottleneck.png}
\caption[mom]{\label{fig:org7cb5329}
Maternal Inheritance}
\end{figure}

\item Heteroplasmy
\label{sec:org37d66b9}

\begin{itemize}
\item Heteroplasmy is the presence of more than one type of organellar
genome within a cell or individual.

\item It is an important factor in considering the severity of
mitochondrial diseases.
\begin{itemize}
\item can also be beneficial
\end{itemize}

\item Microheteroplasmy is present in most individuals.
\begin{itemize}
\item hundreds of independent mutations, with each mutation found in
about 1â€“2\% of all mitochondrial genomes.
\end{itemize}
\end{itemize}

\item Muller's Ratchet and the Mitochondrial Bottleneck
\label{sec:org68b96b4}
\begin{itemize}
\item Entities undergoing uniparental inheritance and with little to no
recombination can be subject to "Muller's ratchet"
\begin{itemize}
\item the inexorable accumulation of deleterious mutations until
functionality is lost.
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./mitochondria/figures/heteroplasmy.png}
\caption[heter]{\label{fig:orga45443e}
Heteroplasmy}
\end{figure}

\begin{itemize}
\item mitochondria avoid this buildup through a developmental process
known as the mtDNA bottleneck. 
\begin{itemize}
\item selection acts to remove those cells with more deleterious mtDNA
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./mitochondria/figures/bottle_neck.jpg}
\caption[bottle]{\label{fig:org8f7888d}
Mitochondrial bottle neck}
\end{figure}

\item Fusion
\label{sec:org7dc7970}
\begin{itemize}
\item response to cellular stress
\item enables genetic complementation
\begin{itemize}
\item fusion of the mitochondria allows two mitochondrial genomes with
different defects within the same organelle to encode what the
other lacks.
\item mtDNA damage
\end{itemize}
\end{itemize}

\item Replication and Fission
\label{sec:org6430f11}
\begin{itemize}
\item Mitochondria divide by binary fission, similar to bacterial cell division
\item mammalian mitochondria replicate their DNA and divide mainly in response
to the energy needs of the cell, not in phase with the cell cycle.
\begin{itemize}
\item When the energy needs of a cell are high, mitochondria grow and
divide.
\item When the energy use is low, mitochondria are destroyed
or become inactive.
\end{itemize}
\end{itemize}
\item Human Mitochondrial DNA
\label{sec:org2d6c260}
\begin{itemize}
\item a circular DNA molecule \textasciitilde{} 16 kb
\item encodes 37 genes
\begin{itemize}
\item 13 for subunits of respiratory complexes I, III, IV and V
\item 22 for mitochondrial tRNA
\begin{itemize}
\item 20 standard amino acids, plus extra gene for leu and ser
\end{itemize}
\item 2 for rRNA.
\end{itemize}
\item One mitochondrion can contain two to ten copies of its DNA.
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=.8\textwidth]{./mitochondria/figures/mitochondrial_genome.png}
\caption[mtdna]{\label{fig:orgabead58}
Human mitochondrial genome}
\end{figure}

\item Alternative genetic code
\label{sec:org0c03766}
\begin{itemize}
\item The mitochondria of many eukaryotes, including most plants, use the
standard code.
\end{itemize}

\begin{table}[htbp]
\caption[mito code]{\label{tab:org57cba15}
Exceptions to the standard genetic code in mamalian mitochondria}
\centering
\begin{tabular}{lll}
Codon & Standard & Mitochondria\\
\hline
AGA, AGG & Arginine & Stop codon\\
AUA & Isoleucine & Methionine\\
UGA & Stop codon & Tryptophan\\
\end{tabular}
\end{table}

\begin{itemize}
\item AUA, AUC, and AUU codons are all allowable start codons.
\item Some of these differences are pseudo-changes in the genetic code due
to the phenomenon of RNA editing, common in mitochondria.
\end{itemize}

\item Mitochondrial Disease
\label{sec:orge65b913}
\begin{itemize}
\item About 15\% of mitochondrial disease is caused by mutations in the
mitochondrial DNA that affect mitochondrial function.
\item Other mitochondrial diseases are caused by
\begin{itemize}
\item mutations in nuclear DNA
\item acquired mitochondrial conditions (drugs, toxins)
\end{itemize}
\end{itemize}
\end{enumerate}

\section{Pyruvate}
\label{sec:org2b9fa27}
\subsection{Introduction}
\label{sec:orgfb27f8a}
\begin{enumerate}
\item Disorders of Pyruvate Metabolism
\label{sec:orgc6808f3}

\begin{itemize}
\item Pyruvate Carboxylase Deficiency
\item Phospoenolpyruvate Carboxykinase Deficiency
\item Pyruvate Dehydrogenase Complex Deficiency
\begin{itemize}
\item Pyruvate Transporter Defect
\end{itemize}
\item Dihydrolipoamide Dehydrogenase Deficiency
\end{itemize}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.7\textwidth]{./pyruvate/figures/pyruvate_disorders.png}
\caption[TCA]{\label{fig:org8bf4729}
Disorders of Pyruvate Metabolism}
\end{figure}

\begin{enumerate}
\item Pyruvate Reactions
\label{sec:orgaee0876}

\begin{description}
\item[{pyruvate dehydrogenase complex}] decarboxylation \(\to\) acetyl-CoA
\end{description}

\ce{pyruvate + CoA + NAD+ ->[PDHC] acetyl-CoA + CO2 + NADH + H+}

\begin{description}
\item[{pyruvate carboxylase}] carboxylation \(\to\) oxaloacetate
\end{description}

\ce{pyruvate + ATP + CO2 ->[PC] oxaloacetate + ADP +Pi}

\begin{description}
\item[{alanine transaminase}] transamination \(\to\) alanine
\end{description}

\ce{pyruvate + glutamate ->[ALT] alanine + \alpha-ketoglutarate}

\begin{description}
\item[{lactate dehydrogenase}] reduction \(\to\) lactate
\end{description}

\ce{pyruvate + NADH ->[LDH] lactate + NAD+}
\end{enumerate}
\end{enumerate}

\subsection{Pyruvate Carboxylase Deficiency}
\label{sec:org40e21b8}
\begin{enumerate}
\item Pyruvate Carboxylase
\label{sec:orgdd443ba}

\begin{itemize}
\item PC is a biotinylated mitochondrial matrix enzyme.
\begin{itemize}
\item carboxylation of pyruvate to oxaloacetate
\end{itemize}

\ce{pyruvate + ATP + CO2 ->[PC] oxaloacetate + ADP +Pi}

\item important role in:
\begin{itemize}
\item gluconeogenesis
\begin{itemize}
\item urea cycle indirectly
\end{itemize}
\item anaplerosis
\begin{itemize}
\item \(\downarrow\) 2-ketoglutarate \(\to\) \(\downarrow\) glutamate
\end{itemize}
\item lipogenesis
\begin{itemize}
\item oxaloacetate + acetyl-CoA \(\to\) citrate
\end{itemize}
\end{itemize}
\end{itemize}

\item Biotin and Bicarbonate
\label{sec:orga5ad289}
\begin{itemize}
\item PC requires biotin and bicarbonate
\item Metabolic derangements associated with PC are observed in:
\begin{itemize}
\item Biotin deficiency and biotinidase deficiency

\item CA-VA deficiency
\begin{itemize}
\item results in dysfunction of all four enzymes to which CA-VA
provides bicarbonate as substrate in mitochondria
\end{itemize}

\item ?Carbamoyl phosphate synthetase 1 deficiency?
\item The three biotin-dependent carboxylases:
\begin{itemize}
\item Propionyl-CoA carboxylase (PCC)
\item 3-methylcrotonyl-CoA carboxylase (3MCC)
\item Pyruvate carboxylase (PC)
\end{itemize}
\end{itemize}
\end{itemize}

\item Clinical Presentation
\label{sec:org5a3d016}
\begin{itemize}
\item French phenotype (type B), most severe
\begin{itemize}
\item acute illness 3-48h after birth
\item hypothermia, hypotonia, lethargy, vomiting
\item severe neurological dysfunction
\item death prior to 5 months
\end{itemize}
\item North American phenotype (type A)
\begin{itemize}
\item severe illness between 2 and 5 months of age
\item progressive hypotonia
\item acute vomiting, dehydration, tachypnoea, metabolic acidosis
\item severe intellectual disability
\item progressive with death in infancy
\end{itemize}
\item Benign phenotype (type c)
\begin{itemize}
\item rare
\item acute episodes of lactic acidosis and ketoacidosis
\item near normal cognitive and motor development
\end{itemize}
\end{itemize}
\item Genetics
\label{sec:org63cb452}
\begin{itemize}
\item Autosomal recessive with incidence of 1 in 250000
\item PC is a homo-tetramer
\item PC protein and mRNA absent in 50\% of French phenotype
\item American and Benign phenotypes have cross-reacting material
\item Mosaicism has been observed with prolonged survival
\end{itemize}

\item Diagnostic Tests
\label{sec:org32f3f87}
\begin{itemize}
\item PC deficiency should be considered in any child presenting with
lactic acidosis and neurological abnormalities
\begin{itemize}
\item with hypoglycemia, hyperammonemia, or ketosis
\end{itemize}

\item \(\uparrow\) L/P with \(\downarrow\) BHB/acetoacetate in severely affected patients
\begin{itemize}
\item pathognomonic in neonates
\end{itemize}

\item post-prandial ketosis, hypercitrullinemia, hyperammonemia, low glutamine

\item CSF lactate, alanine and L/P are elevated, glutamine decreased

\item PC activity in cultured skin fibroblasts
\begin{itemize}
\item can not distinguish severity
\end{itemize}
\end{itemize}

\item Treatment
\label{sec:orga21fe49}

\begin{itemize}
\item Currently, no treatment.
\end{itemize}
\end{enumerate}

\subsection{Phospoenolpyruvate Carboxykinase Deficiency}
\label{sec:org9ba8935}
\begin{itemize}
\item PEPCK has cytosolic and mitochondria isoforms
\item Cytosolic PEPCK deficiency is secondary to hyperinsulinism
\begin{itemize}
\item insulin represses expression of the cytosolic form
\end{itemize}
\item Mitochondrial PEPCK deficiency has not been clearly demonstrated
\end{itemize}

\subsection{Pyruvate Dehydrogenase Complex Deficiency}
\label{sec:org4b79601}
\begin{enumerate}
\item Pyruvate Dehydrogenase Complex
\label{sec:org927cd7e}
\begin{itemize}
\item PDHC decarboxylates pyruvate \(\to\) acetyl-CoA
\begin{itemize}
\item thiamine dependant
\end{itemize}
\item PDHC, KDHC and BCKD have similar structure and mechanism
\item Composed of:
\begin{itemize}
\item E1 \(\alpha\)-ketoacid dehydrogenase
\item E2 dihydrolipoamide acyltransferase
\item E3 dihydrolipoamide dehydrogenases
\end{itemize}
\item E1 is specific to each complex
\begin{itemize}
\item Composed of E1\(\alpha\) and E1\(\beta\)
\end{itemize}
\item E1 is the rate limiting step in PDHC
\begin{itemize}
\item regulated by phosphorylation
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.4\textwidth]{./pyruvate/figures/pdhe1_phos.png}
\caption[pdhe1]{\label{fig:org8cae44e}
Activation/deactivation of PDHE1}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./pyruvate/figures/pdhc.png}
\caption[pdhc]{\label{fig:org3cb03c2}
Pyruvate Dehydrogenase Complex}
\end{figure}

\item Pyruvate Dehydrogenase Complex Deficiency
\label{sec:org52f65a1}
\begin{itemize}
\item PDHC deficiency provokes conversion of pyruvate to lactate and alanine rather than acetly-CoA
\item Metabolism of glucose \(\to\) lactate, produces 1/10 ATP compared to
complete oxidation via TCA and ETC
\item Impairs production of NADH but not oxidation
\item NADH/\ce{NAD+} is normal, \(\therefore\) normal L/P
\begin{itemize}
\item ETC deficiencies \(\to\) \(\uparrow\) L/P
\end{itemize}
\end{itemize}

\item Clinical Presentation: PDHE1\(\alpha\)
\label{sec:org2f3182e}
\begin{itemize}
\item Majority of cases involve the X encoded to \(\alpha\)-subunit of the dehydrogenase (E1)
\begin{itemize}
\item PDHE1\(\alpha\) deficiency
\item developmental delay, hypotonia, seizures and ataxia
\end{itemize}

\item Common presentations in hemizygous males:
\begin{enumerate}
\item neonatal lactic acidosis
\begin{itemize}
\item most severe
\end{itemize}
\item Leigh's encephalopathy
\begin{itemize}
\item most common
\item present in first 5 years
\end{itemize}
\item intermittent ataxia
\begin{itemize}
\item rare
\item ataxia after carbohydrate rich meals \(\to\) Leigh's
\end{itemize}
\end{enumerate}

\item Females with PDHE1\(\alpha\), uniform presentation, variable severity
\begin{itemize}
\item dismorphic features
\item moderate to severe intellectual disability
\item seizures common
\item severe neonatal lactic acidosis can be present
\end{itemize}
\end{itemize}

\item Clinical Presentation: PDHE1\(\beta\)
\label{sec:org0d8ff0f}
\begin{itemize}
\item Only a few cases
\item similar to PDHE1\(\alpha\)
\end{itemize}

\item Genetics
\label{sec:org26e1175}
\begin{itemize}
\item All components of PDHC are encoded by nuclear genes
\item Autosomal except E1\(\alpha\) on Xp22.11
\begin{itemize}
\item \(\therefore\) most PDHC deficiency is X-linked
\end{itemize}
\item No null E1\(\alpha\) identified except in a mosaic state
\begin{itemize}
\item suggests E1\(\alpha\) is essential
\end{itemize}
\end{itemize}

\item Diagnostic Tests
\label{sec:org40a5467}
\begin{itemize}
\item Lactate and pyruvate in blood and CSF
\item CSF lactate is generally \(\uparrow\) compared to blood
\item Urine organic acids
\begin{itemize}
\item lactic and pyruvic acid
\end{itemize}
\item plasma amino acids
\begin{itemize}
\item alanine
\end{itemize}
\item L/P ratio is usually normal
\item Skin fibroblasts for PDHC
\begin{itemize}
\item also lymphocytes, separated from EDTA <2days
\end{itemize}
\item PDHE1\(\alpha\) genotype in females is useful
\end{itemize}

\item Treatment
\label{sec:org23627b0}
\begin{itemize}
\item Early adoption of ketogenic diet may have a benefit
\item Thiamine responsive forms
\item DCA is a pyruvate analog, inhibits E1 kinase, keeps E1 dephosphorylated (active) (Figure \ref{fig:org8cae44e})
\end{itemize}

\item Pyruvate Transport Defect
\label{sec:orgc8c2b33}
\begin{itemize}
\item MPC1 mutations have been described in 5 patients
\item mediates the proton symport of pyruvate across the IMM.
\item \(\therefore\) metabolic derangement similar to PDHC deficiency
\item No treatment
\end{itemize}
\end{enumerate}

\subsection{Dihydrolipoamide Dehydrogenase Deficiency}
\label{sec:org86c2322}
\begin{enumerate}
\item Dihydrolipoamide Dehydrogenase
\label{sec:org6d2888f}
\begin{itemize}
\item DLD (E3) is a flavoprotein common to all three mitochondrial
\(\alpha\)-ketoacid dehydrogenase complexes
\begin{itemize}
\item PDHC, KDHC, and BCKD
\end{itemize}
\item Combined PDHC, TCA , BCAA defect
\begin{itemize}
\item \(\uparrow\) lactate , pyruvate,
\item alanine, glutamate, glutamine, BCAA
\item urinary lactic, pyruvic, 2-ketoglutaric, BC 2-hydroxy \& 2-ketoacids
\end{itemize}
\end{itemize}

\item Genetics and Diagnotic Testing
\label{sec:org2316e66}
\begin{itemize}
\item DLD mutations AR
\item 13 unrelated patients revealed 14 unique mutations
\item Blood lactate, pyruvate
\item plasma amino acids
\item urinary organic acids
\item Pattern of abnormalities not seen in all patients at all times.
\end{itemize}
\end{enumerate}
\section{Tricarboxylic Acid Cycle}
\label{sec:org589cb62}
\subsection{Introduction}
\label{sec:org26eda25}
\begin{itemize}
\item Pathways for oxidation of fatty acids, glucose, amino acids and ketones produce acetyl-CoA
\end{itemize}
%%\setchemfig{lewis style=red}
\centering
\chemname{\chemfig{\lewis{0.,H}-\lewis{0.2.4.6.,{\color{red}C}}(-[6]\lewis{2.,H})(-[2]\lewis{6.,H})-\lewis{4.,{\color{red}C}}(=[2]O)-[,,,,decorate, decoration=snake]SCoA}}{acetyl-CoA}
\begin{itemize}
\item Part of aerobic respiration
\begin{itemize}
\item ETC regenerates \ce{NAD+} from NADH
\end{itemize}
\item Cofactors:
\begin{itemize}
\item niacin (\ce{NAD+})
\item riboflavin (FAD and FMN)
\item panthothenic acid (CoA)
\item thiamine
\item \ce{Mg^2+}, \ce{Ca^2+}, \ce{Fe+} and phosphate
\end{itemize}

\item TCA net reaction
\end{itemize}

\ce{AcetylCoA + 3NAD+ + FAD + GDP + Pi + 2H2O -> 2CO2 + CoA + 3NADH + FADH2 + GTP + 2H+}

\begin{itemize}
\item release of energy via oxidation of acetly-CoA
\item one molecule of glucose breaks down into two molecules of pyruvate
\item Pyruvate is converted into acetyl-CoA, which is the main input TCA
\end{itemize}

\begin{enumerate}
\item Disorders of the TCA cycle
\label{sec:orgd27e540}

\begin{itemize}
\item \(\alpha\)-Ketoglutarate Dehydrogenase Complex Deficiency
\item Succinate Dehydrogenase Deficiency
\item Fumarase Deficiency
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.7\textwidth]{./tca/figures/TCA_disorders.png}
\caption{\label{fig:org61b15bc}
TCA disorders}
\end{figure}


\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./tca/figures/gr2.png}
\caption{\label{fig:org115887b}
Model for a functional splitting of the Krebs cycle reactions into complementary mini-cycles.}
\end{figure}

-uses aspartate-amino acid transferase
\begin{itemize}
\item (A) would allow to convert pyruvate up to \(\alpha\)-KG, even when the second mini-cycle (B) does not function.
\item accounts for the urinary excretion of \(\alpha\)-KG in patients
presenting with defect of \(\alpha\)-KG, SDH or fumarase activity.
\item could produce reduced equivalents to sustain the normal oxygen
uptake measured in circulating lymphocytes or cultured skin
fibroblast from these patients.
\end{itemize}
\end{enumerate}

\subsection{\(\alpha\)-ketoglutarate Dehydrogenase Complex Deficiency}
\label{sec:org7a0aac2}
\begin{enumerate}
\item \(\alpha\)-ketoglutarate Dehydrogenase Complex
\label{sec:org979215c}
\begin{itemize}
\item KDHC is a \(\alpha\)-ketoacid dehydrogenase analogous to PDHC and BCKD.
\end{itemize}

\ce{\alpha-ketoglutarate + NAD+ + CoA ->[KDHC] Succinyl CoA + CO2 + NADH}

\begin{table}[htbp]
\caption{\label{tab:org91ac1ad}
\(\alpha\)-ketoglutarate Dehydrogenase Complex}
\centering
\begin{tabular}{llll}
Unit & Name & Gene & Cofactor\\
\hline
E1 & \(\alpha\)-ketoglutarate dehydrogenase & OGDH & thiamine pyrophosphate(TPP)\\
E2 & dihydrolipoyl succinyltransferase & DLST & lipoic acid, Coenzyme A\\
E3 & dihydrolipoyl dehydrogenase & DLD & FAD, NAD\\
\end{tabular}
\end{table}

\begin{itemize}
\item E1 subunit is the thiamine dependant substrate specific dehydrogenase
\begin{itemize}
\item Not regulated by phosphorylation.
\end{itemize}
\item E2 subunit dihydrolipyoyl succinyl-transferase is also specific to KDHC
\end{itemize}

\item Clinical Presentation
\label{sec:orgdb8e228}
\begin{itemize}
\item Similar to PDHC
\item Developmental delay, hypotonia, opisthotonos and ataxia
\begin{itemize}
\item seizures less common
\end{itemize}
\item Present as neonate and early childhood
\end{itemize}

\item Genetics
\label{sec:orgb1b6b5f}
\begin{itemize}
\item AR inheritance, encoded by nuclear DNA
\item E1 gene mapped to 7p13
\item E2 gene mapped to 14q24.3
\item Molecular basis of KDHC deficiencies is not resolved.
\item \(\alpha\)-ketoglutarate dehydrogenase deficiency is sometimes a feature of DLD deficiency
\end{itemize}

\item Diagnostic Tests
\label{sec:org842860d}
\begin{itemize}
\item Urine organic acids
\begin{itemize}
\item \(\uparrow\) \(\alpha\)-KGA, \textpm{} other TCA intermediates
\item \(\alpha\)-KGA is a common finding, not specific for KDHC deficiency
\end{itemize}
\item Blood lactate
\begin{itemize}
\item Normal or increased L/P
\end{itemize}
\item KDHC activity
\begin{itemize}
\item \ce{^14CO2} release from \ce{[1-^14C]} \(\alpha\)-ketoglutarate (or \ce{[1-^14C]} leucine)
\item cultured skin fibroblasts
\item muscle
\end{itemize}
\end{itemize}

\item Treatment
\label{sec:orgc826d53}
\begin{itemize}
\item None to date
\end{itemize}
\end{enumerate}

\subsection{Succinate Dehydrogenase Deficiency}
\label{sec:org273cde7}
\begin{enumerate}
\item Succinate Dehydrogenase | Complex II
\label{sec:org4243c9c}
\begin{itemize}
\item Four subunits compose Complex II of the mitochondrial respiratory chain
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org906cb9e}
Succinate Dehydrogenase}
\centering
\begin{tabular}{ll}
Subunit name & Protein description\\
\hline
SdhA & Succinate dehydrogenase flavoprotein subunit\\
SdhB & Succinate dehydrogenase iron-sulfur subunit\\
SdhC & Succinate dehydrogenase cytochrome b560 subunit\\
SdhD & Succinate dehydrogenase cytochrome b small subunit\\
\end{tabular}
\end{table}

\begin{itemize}
\item The SdhA subunit contains an FAD binding site where succinate
is deprotonated and converted to fumarate.
\end{itemize}

succinate + ubiquinone \(\to\) fumarate + ubiquinol

\begin{itemize}
\item Electrons removed from succinate transfer to SdhA
\item transfer across SdhB through iron sulphur clusters to the SdhC/SdhD subunits
\begin{itemize}
\item SdhC/SdhD are anchored in the mitochondrial membrane.
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./tca/figures/SuccDeh.png}
\caption{\label{fig:orga1023d5}
Succinate Dehydrogenase Deficiency}
\end{figure}

\item Clinical Presentation
\label{sec:org2d9e542}
\begin{itemize}
\item Very rare disorder with highly variable phenotype
\item Complex II is part of the TCA cycle and ETC
\begin{itemize}
\item phenotype resembles defects in respiratory chain
\end{itemize}
\item Clinical picture can include:
\begin{itemize}
\item Kearns-Sayre syndrome
\item isolated hypertrophic cardiomyopathy
\item combined cardiac and skeletal myopathy
\item generalized muscle weakness, \(\uparrow\) fatiguability
\item early onset Leigh encephalopathy
\end{itemize}
\item Also:
\begin{itemize}
\item cerebral ataxia
\item optic atropy
\item tumour formation in adults
\end{itemize}
\end{itemize}

\item Genetics
\label{sec:org1389f33}

\begin{itemize}
\item All components of Complex II are encoded by nuclear DNA.
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org2b22144}
Succinate Dehydrogenase Genetics}
\centering
\begin{tabular}{ll}
Gene & Location\\
\hline
SDHA & 5p15.33\\
SDHB & 1p36.13\\
SDHC & 1q23.3\\
SDHD & 11q23.1\\
\end{tabular}
\end{table}


\begin{itemize}
\item AR with highly variable phenotype
\item Case of affected sisters with one identified SDHA mutation suggested
dominant transmission
\item Mutations in SDHB, SDHC and SDHD cause susceptibility to familial
phaeochromocytoma and familial paraganglioma.
\end{itemize}

\item Diagnostic Tests
\label{sec:orgb9e5d8e}
\begin{itemize}
\item Unlike other TCA cycle disorders Complex II deficiency does not always
result in characteristic organic aciduria
\begin{itemize}
\item succinic aciduria.
\end{itemize}
\item Organic acids can show variable amounts of lactate, pyruvate, succinate, fumarate and malate
\item Measurement of complex II activity in muscle is the most reliable
means of diagnosis
\begin{itemize}
\item there is no clear correlation between residual complex II activity
and severity or clinical outcome.
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./tca/figures/gr4.jpg}
\caption{\label{fig:org4a8c046}
Coupled spectrophotometric assay}
\end{figure}

\item Treatment
\label{sec:orgfc29a36}
\begin{itemize}
\item In some cases, treatment with riboflavin may have clinical benefit
\end{itemize}
\end{enumerate}

\subsection{Fumarase Deficiency}
\label{sec:org5e6f227}
\begin{enumerate}
\item Fumarase
\label{sec:orgf7371d7}
\begin{itemize}
\item Fumarase catalyses reversible hydration/dehydration of fumarate to malate
\item Two forms: mitochondrial and cytosolic.
\begin{itemize}
\item The mitochondrial isoenzyme is involved in the TCA Cycle
\item The cytosolic isoenzyme is involved in the metabolism of amino acids and fumarate.
\end{itemize}
\item Subcellular localization is established by the presence/absence of an N-terminal mitochondrial signal
sequence
\item Deficiency causes impaired energy production
\end{itemize}

\item Clinical Presentation
\label{sec:org9bf78e8}
\begin{itemize}
\item Characterized by polyhydramnios and fetal brain abnormalities.
\item In the newborn period, findings include:
\begin{itemize}
\item severe neurologic abnormalities,
\item poor feeding,
\item failure to thrive
\item hypotonia.
\end{itemize}
\item Fumarase deficiency is suspected in infants with multiple severe
neurologic abnormalities in the absence of an acute metabolic
crisis.
\item Inactivity of both cytosolic and mitochondrial forms of
fumarase are potential causes.
\end{itemize}
\item Genetics
\label{sec:org1ebd4d0}
\begin{itemize}
\item AR inheritance, encoded by nuclear DNA
\item Single gene and mRNA encode mito and cyto isoforms
\end{itemize}
\item Diagnostic Tests
\label{sec:orgb7380a3}
\begin{itemize}
\item Isolated, increased concentration of fumaric acid on urine organic
acid analysis is highly suggestive of fumarase deficiency.
\begin{itemize}
\item Succinate, \(\alpha\)-KGA can also be elevated
\end{itemize}
\item Molecular genetic testing for fumarase deficiency
\end{itemize}
\end{enumerate}
\subsection{Isocitrate Dehydrogenase}
\label{sec:org2e65cae}
\begin{itemize}
\item IDH exists in three isoforms:
\begin{itemize}
\item IDH3 catalyzes the third step of the citric acid cycle while converting \ce{NAD+} to NADH in the mitochondria.
\end{itemize}
\end{itemize}

\ce{isocitrate + NAD+ ->[IHD3] \alpha-ketoglutarate + CO2 + NADH + H+}

\begin{itemize}
\item IDH1 and IDH2 catalyze the same reaction outside TCA cycle and use \ce{NADP+} as a cofactor.
\begin{itemize}
\item They localize to the cytosol as well as the mitochondrion and peroxisome.
\end{itemize}
\end{itemize}

\ce{isocitrate + NADP+ ->[IHD1/2] \alpha-ketoglutarate + CO2 + NADPH + H+}

\begin{enumerate}
\item Clinical relevance
\label{sec:orgf3a18da}
\begin{itemize}
\item IDH3 deficiency is associated with retinitis pigmentosa
\item IDH1/2 mutations linked to malignant gliomas and acute myeloid leukemia
\item Mutations in IDH2 identified in half of patients
\end{itemize}
\end{enumerate}
\section{Electron Transport Chain}
\label{sec:orgbc673f3}
\subsection{Introduction}
\label{sec:org18bcea5}
\begin{enumerate}
\item ETC and OxPhos system
\label{sec:org821dd9f}
\begin{itemize}
\item Responsible for ATP production
\item ETC complexes I-IV
\item OxPhos system complexes I-V
\end{itemize}
\begin{enumerate}
\item Chemiosmotic Coupling Hypothesis
\label{sec:org116b05c}
\begin{itemize}
\item the ETC and OxPhos are coupled by a proton gradient across the IMM.
\item The efflux of protons from the mitochondrial matrix creates an electrochemical gradient.
\begin{itemize}
\item used by the F\(_{\text{O}}\)F\(_{\text{1}}\) ATP synthase complex to make ATP via oxidative phosphorylation.
\end{itemize}
\end{itemize}
\end{enumerate}
\item Electron Transport Chain
\label{sec:org1abb022}
\begin{itemize}
\item Energy from transfer of electrons down the ETC is used to pump
protons from the mitochondrial matrix into the intermembrane space.
\begin{itemize}
\item creates an electrochemical proton gradient (\(\Delta\)pH) across the IMM.
\begin{itemize}
\item largely responsible for the mitochondrial membrane potential (\(\Delta \Psi\)M).
\end{itemize}
\item ATP synthase uses flow of \ce{H+} through the enzyme back into the
matrix to generate ATP from ADP and Pi.
\end{itemize}
\item There are three energy-transducing enzymes in the electron transport
chain:
\begin{itemize}
\item NADH:ubiquinone oxidoreductase (complex I)
\item Coenzyme Q â€“ cytochrome c reductase (complex III)
\item cytochrome c oxidase (complex IV).
\item also ETF-QO and mitochondrial GPD
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./etc/figures/etc.pdf}
\caption{\label{fig:orgd209180}
Electron Transport Chain}
\end{figure}

\begin{itemize}
\item Complex I (NADH coenzyme Q reductase) accepts electrons from the
Krebs cycle electron carrier NADH
\item passes them to CoQ (ubiquinone; labeled Q),
\item CoQ also receives electrons from complex II (succinate dehydrogenase)
\item CoQ passes electrons to complex III (cytochrome bc1 complex; labeled
III), which passes them to cytochrome c (cyt c).
\item Cyt c passes electrons to Complex IV (cytochrome c oxidase; labeled
IV), which uses the electrons and hydrogen ions to reduce molecular
oxygen to water.
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.7\textwidth]{./etc/figures/hsa00190.png}
\caption[ETC]{\label{fig:orgfa5a986}
Oxidative Phosphorylation}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.6\textwidth]{./etc/figures/etc_supercomplex.jpg}
\caption[ETC mega complex]{\label{fig:org7d4bbba}
ETC Mega Complex}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./etc/figures/potential.png}
\caption[redox]{\label{fig:org0a9cb19}
Electron flow to O\(_{\text{2}}\) and release free energy}
\end{figure}
\end{enumerate}

\subsection{Complex I}
\label{sec:orged92e21}
\begin{enumerate}
\item Complex I | NADH-ubiquinone oxidoreductase
\label{sec:orgb269f7c}
\begin{itemize}
\item catalyzes the transfer of electrons from NADH to coenzyme Q10
(CoQ10) and translocates protons across the inner mitochondrial
membrane
\end{itemize}

{\small\ce{NADH + H+ + CoQ + 4H^{+}_{in} ->[CI] NAD+ + CoQH2 + 4H^{+}_{out}}}

\begin{itemize}
\item Mechanism: 
\begin{enumerate}
\item Seven iron sulfur centers carry electrons from the site of NADH
dehydration to ubiquinone.

\item ubiquinone (CoQ) is reduced to ubiquinol (\ce{CoQH2}).

\item The energy from the redox reaction results in conformational
change allowing hydrogen ions to pass through four transmembrane
helix channels.
\end{enumerate}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.9\textwidth]{./etc/figures/c1.pdf}
\caption[c1]{\label{fig:orgb60ec17}
Complex I | NADH-ubiquinone oxidoreductase}
\end{figure}

\item Complex I Inhibitors
\label{sec:orgd39f91b}
\begin{itemize}
\item The best-known inhibitor of complex I is rotenone
\begin{itemize}
\item commonly used as an organic pesticide
\end{itemize}
\item Rotenone binds to the ubiquinone binding site of complex I
\begin{itemize}
\item piericidin A a potent inhibitor and structural homologue to ubiquinone.
\end{itemize}
\item Hydrophobic inhibitors like rotenone or piericidin likely disrupt electron transfer between FeS cluster N2 and ubiquinone.
\item Bullatacin is the most potent known inhibitor of NADH dehydrogenase (ubiquinone)
\item Complex I is also blocked by adenosine diphosphate ribose â€“ a reversible competitive inhibitor of NADH oxidation
\end{itemize}
\end{enumerate}

\subsection{Complex II}
\label{sec:org97ed545}
\begin{enumerate}
\item Complex II | Succinate Dehydrogenase
\label{sec:org5bde501}
\begin{itemize}
\item Four subunits compose Complex II of the mitochondrial respiratory chain
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org32a3433}
Complex II Succinate Dehydrogenase}
\centering
\begin{tabular}{ll}
Subunit name & Protein description\\
\hline
SdhA & Succinate dehydrogenase flavoprotein subunit\\
SdhB & Succinate dehydrogenase iron-sulfur subunit\\
SdhC & Succinate dehydrogenase cytochrome b560 subunit\\
SdhD & Succinate dehydrogenase cytochrome b small subunit\\
\end{tabular}
\end{table}

\begin{itemize}
\item The SdhA subunit contains an FAD binding site where succinate
is deprotonated and converted to fumarate.
\end{itemize}

{\small\ce{succinate + ubiquinone ->[CII] fumarate + ubiquinol}}

\begin{itemize}
\item Electrons removed from succinate transfer to SdhA
\item transfer across SdhB through iron sulphur clusters to the SdhC/SdhD subunits
\begin{itemize}
\item SdhC/SdhD are anchored in the mitochondrial membrane.
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.4\textwidth]{./etc/figures/CII.png}
\caption[cII]{\label{fig:org4ce26c8}
Complex II | Succinate Dehydrogenase}
\end{figure}

\item Complex II Inhibitors
\label{sec:org407b72f}
\begin{itemize}
\item There are two distinct classes of inhibitors of complex II:
\begin{itemize}
\item those that bind in the succinate pocket and those that bind in the ubiquinone pocket.
\end{itemize}
\item Ubiquinone type inhibitors include carboxin and thenoyltrifluoroacetone.
\item Succinate-analogue inhibitors include the synthetic compound malonate as well as the TCA cycle intermediates, malate and oxaloacetate.
\begin{itemize}
\item oxaloacetate is one of the most potent inhibitors of Complex II.
\end{itemize}
\end{itemize}
\end{enumerate}
\subsection{Glycerol-3-phosphate shuttle}
\label{sec:orgf7fdb2c}
\begin{itemize}
\item Oxidation of cytoplasmic NADH by the cytosolic form of the enzyme
creates glycerol-3-phosphate from dihydroxyacetone phosphate.
\item Glycerol-3-phosphate diffuses into IMM and is oxidised by mitochondrial glycerol-3-phosphate dehydrogenase
\begin{itemize}
\item uses quinone as an oxidant and FAD as a co-factor.
\end{itemize}
\item maintains the cytoplasmic NAD+/NADH ratio.
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.4\textwidth]{./etc/figures/GPDH_shuttle.png}
\caption[g3ps]{\label{fig:org33c64cf}
Glycerol-3-phosphate shuttle}
\end{figure}

\subsection{Electron Transferring Flavoprotein/ Dehydrogenase}
\label{sec:org6c3319b}

\begin{itemize}
\item ETFs are heterodimeric proteins composed of an alpha and beta
subunit (ETFA and ETFB), and contain an FAD cofactor and AMP

\item ETQ-QO links the oxidation of fatty acids and some amino acids to
oxidative phosphorylation in the mitochondria.
\item catalyzes the transfer of electrons from electron transferring
flavoprotein (ETF) to ubiquinone, reducing it to ubiquinol.
\end{itemize}

{\small\ce{Acyl-CoA + FAD ->[ACAD] FADH2 + ETF ->[ETF-QO] UQ ->[CIII] CytC}}

\begin{itemize}
\item ETF-QO deficiency results in \textbf{Glutaric Acidemia Type II} (AKA MADD
for multiple acyl-CoA dehydrogenase deficiency)
\begin{itemize}
\item Discussed more in Lysine Catabolism
\end{itemize}
\end{itemize}
\subsection{Complex III}
\label{sec:org62411f0}
\begin{enumerate}
\item Complex III | Coenzyme Q â€“ cytochrome c reductase
\label{sec:orgcc3611e}
\begin{itemize}
\item Complex III is a multi-subunit transmembrane protein encoded by both
mitochondrial (cytochrome B) and the nuclear genomes (all other
subunits)

\item The bc1 complex contains 11 subunits:
\begin{itemize}
\item 3 respiratory subunits (cytochrome B, cytochrome C1, Rieske protein)
\item 2 core proteins
\item 6 low-molecular weight proteins
\end{itemize}
\end{itemize}

{\small\ce{QH2 + 2Fe^{3+}-cyt c + 2H+_{in} ->[CIII]  Q + 2Fe^{2+}-cyt c + 4H+_{out}}}

\begin{enumerate}
\item Mechanism
\label{sec:org043cf82}
\begin{itemize}
\item Round 1:
\begin{itemize}
\item Cytochrome b binds a ubiquinol and a ubiquinone.
\item The 2Fe/2S center and BL heme each pull an electron off the bound ubiquinol, releasing two hydrogens into the intermembrane space.
\item One electron is transferred to cytochrome c1 from the 2Fe/2S centre, whilst another is transferred from the BL heme to the BH Heme.
\item Cytochrome c1 transfers its electron to cytochrome c (not to be confused with cytochrome c1), and the BH Heme transfers its electron to a nearby ubiquinone, resulting in the formation of a ubisemiquinone.
\item Cytochrome c diffuses. The first ubiquinol (now oxidised to ubiquinone) is released, whilst the semiquinone remains bound.
\end{itemize}

\item Round 2:
\begin{itemize}
\item A second ubiquinol is bound by cytochrome b.
\item The 2Fe/2S center and BL heme each pull an electron off the bound ubiquinol, releasing two hydrogens into the intermembrane space.
\item One electron is transferred to cytochrome c1 from the 2Fe/2S centre, whilst another is transferred from the BL heme to the BH Heme.
\item Cytocrome c1 then transfers its electron to cytochrome c, whilst the nearby semiquinone produced from round 1 picks up a second electron from the BH heme, along with two protons from the matrix.
\item The second ubiquinol (now oxidised to ubiquinone), along with the newly formed ubiquinol are released.
\end{itemize}
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.7\textwidth]{./etc/figures/CIII.png}
\caption[cIII]{\label{fig:org7e905b0}
Complex III | Coenzyme Q â€“ cytochrome c reductase: two step reaction}
\end{figure}
\end{enumerate}

\item Complex III Inhibitors
\label{sec:orgab99442}
\begin{itemize}
\item There are three distinct groups of Complex III inhibitors:
\begin{itemize}
\item Antimycin A binds to the Q\(_{\text{i}}\) site and inhibits the transfer of electrons in Complex III from heme b\(_{\text{H}}\) to oxidized Q (Q\(_{\text{i}}\) site inhibitor).
\item Myxothiazol and stigmatellin bind to distinct but overlapping pockets within the Q\(_{\text{o}}\) site.
\begin{itemize}
\item Myxothiazol binds nearer to cytochrome bL (hence termed a "proximal" inhibitor).
\item Stigmatellin binds farther from heme bL and nearer the Rieske Iron sulfur protein.
\item Both inhibit the transfer of electrons from reduced QH\(_{\text{2}}\) to the Rieske Iron sulfur protein.
\end{itemize}
\end{itemize}
\end{itemize}
\end{enumerate}
\subsection{Complex IV}
\label{sec:orgd302023}
\begin{enumerate}
\item Complex IV | Cytochrome c oxidase
\label{sec:orga072aca}

\begin{itemize}
\item last enzyme in the respiratory electron transport chain.
\item large IMM integral membrane protein composed of several metal prosthetic sites and 14 protein subunits.
\item eleven subunits are nuclear in origin, and three are synthesized in the mitochondria. 
\begin{itemize}
\item contains two hemes,
\item cytochrome a and cytochrome a3,
\item two copper centers, CuA and CuB
\end{itemize}
\item the cytochrome a3 and CuB form a binuclear center that is the site of oxygen reduction.
\item receives an electron from four cytochrome c molecules and transfers them to one O\(_{\text{2}}\) molecule
\end{itemize}

{\small\ce{4Fe^{2+}-cyt c + 8H^{+}_{in} + O2 ->[CIV] 4Fe^{3+}-cyt c + 2H2O + 4H^{+}_{out}}}

\begin{itemize}
\item In the process binds four protons from the inner aqueous phase to
make two water molecules, and translocates another four protons
across the membrane, increasing the transmembrane difference of
proton electrochemical potential which the ATP synthase then uses to
synthesize ATP.
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./etc/figures/CIV.png}
\caption[cIV]{\label{fig:orgbb804a8}
Complex IV | Cytochrome c oxidase}
\end{figure}

\item Complex IV | Inhibitors
\label{sec:org4bcd4ce}
\begin{itemize}
\item Cyanide, azide, and carbon monoxide all bind to cytochrome c
oxidase
\item Nitric oxide and hydrogen sulfide, can also inhibit COX by
binding to regulatory sites on the enzyme
\end{itemize}
\end{enumerate}
\subsection{Complex V}
\label{sec:orgbd4098f}
\begin{enumerate}
\item Complex V | ATP synthase
\label{sec:org6aadc37}

\begin{itemize}
\item ATP synthase is a molecular machine that creates the energy storage
molecule adenosine triphosphate (ATP).

\item The overall reaction catalyzed by ATP synthase is:
\end{itemize}

{\small\ce{ADP + P_i + H+_{out} <=> ATP + H2O + H+_{in}}}


\begin{itemize}
\item Formation of ATP from ADP and P\(_{\text{i}}\) is energetically unfavourable
\begin{itemize}
\item would normally proceed in the reverse direction.
\end{itemize}

\item To drive this reaction forward, ATP synthase couples ATP synthesis
to the electrochemical gradient (\(\Delta \Psi\)M) created by complexes
I,III and IV

\item ATP synthase consists of two main subunits, FO and F1, which has a
rotational motor mechanism allowing for ATP production.
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./etc/figures/atp_synthase.jpg}
\caption{\label{fig:org6b15dce}
Complex V | ATP synthase}
\end{figure}


{\small\ce{ADP + Pi + H+_{out} <=> ATP + H2O + H+_{in}}}

\item Complex V Inhibitors
\label{sec:org35fda20}
\begin{itemize}
\item Oligomycin A inhibits ATP synthase by blocking its proton channel
(Fo subunit), which is necessary for oxidative phosphorylation of
ADP to ATP (energy production).
\item The inhibition of ATP synthesis by oligomycin A will significantly
reduce electron flow through the electron transport chain; however,
electron flow is not stopped completely due to a process known as
proton leak or mitochondrial uncoupling.
\begin{itemize}
\item This process is due to facilitated diffusion of protons into the
mitochondrial matrix through an uncoupling protein such as
thermogenin, or UCP1.
\end{itemize}

\item Administering oligomycin to an individual can result in very high
levels of lactate accumulating in the blood and urine.
\end{itemize}
\end{enumerate}
\section{OxPhos Disorders}
\label{sec:orge726806}
\subsection{Metabolic Derangement}
\label{sec:orgb9ed5ba}
\begin{enumerate}
\item OxPhos Deficiency and Anaerobic Glycolysis
\label{sec:org59fa82c}
\begin{itemize}
\item Insufficient ATP severely affects highly energy dependant tissues
\begin{itemize}
\item A complete loss of OxPhos is not observed in human disease.
\end{itemize}
\item In the absence of OxPhos cells survive using ATP from anaerobic glycolysis
\begin{itemize}
\item 20x less efficient, generates lactate
\item pyruvate \(\to\) alanine if glutamate is available
\end{itemize}
\item Lactate, pyruvate and alanine are the typical products of anaerobic glycolysis
\end{itemize}

\item Factors Affecting OxPhos System
\label{sec:orgcfc67ad}
\begin{itemize}
\item \textasciitilde{} 90 subunits
\begin{itemize}
\item 13 subunits of Complexes I, III, IV and V encoded by mtDNA
\end{itemize}
\item mitochondrial replication, transcription and translation
\begin{itemize}
\item require \textgreater{} 200 proteins, rRNAs and tRNAs
\end{itemize}
\item Cofactors: coenzyme Q\(_{\text{10}}\), iron-sulfur clusters, heme, copper
\begin{itemize}
\item require synthesis and/or transport to OxPhos system
\end{itemize}
\item Cardiolipin required for cristae formation
\item Mitochondrial function
\begin{itemize}
\item protein import, turnover
\item fission, fusion
\end{itemize}
\item Toxic metabolites
\item \textgreater{} 1500 proteins in the human mitochondrial proteome
\begin{itemize}
\item other additional factors - lipids, cofactors
\item up to 10\% of human proteome may have role in maintaining mitochondrial function
\end{itemize}
\end{itemize}

\item Types of genetic defects
\label{sec:org136fb5c}
\begin{itemize}
\item OxPhos Subunit
\item Assembly Factor
\item mtDNA replication
\item mtDNA transcription
\item cofactor
\item mitochondrial homeostasis
\begin{itemize}
\item fission and fusion
\end{itemize}
\item the primary biochemical phenotype is impaired OxPhos
\end{itemize}
\end{enumerate}

\subsection{OxPhos Clinical Manifestations}
\label{sec:org7fefa08}
\begin{itemize}
\item Clinical diagnosis is extremely challenging
\begin{itemize}
\item can affect any organ system
\item antenatal (IUGR, birth defects) \(\to\) elderly (myopathy)
\end{itemize}

\item Clinical Suspicion based on:
\begin{enumerate}
\item Constellation of symptoms/signs consistent with a mitochondrial syndrome
\item Complex multi-system presentation involving two/more organ systems,
best described by an underlying disorder of energy generation.
\item Lactic acidosis, characteristic neuro-imaging, 3-methylglutaconic
aciduria, ragged red fiber mypopathy.
\item Pathogenic mutation in a known mitochondrial disease gene.
\end{enumerate}
\end{itemize}


\begin{itemize}
\item Mitochondrial disease commonly presents with:
\begin{enumerate}
\item Myopathy
\item Encephalopathy
\item Leberâ€™s hereditary optic neuropathy
\item Pearson Syndrome
\item Diabetes
\end{enumerate}
\end{itemize}

\begin{table}[htbp]
\caption{\label{tab:org798960c}
OxPhos Clinical Manifestations}
\centering
\begin{tabular}{ll}
System & Manifestation\\
\hline
CNS & \textbf{myoclonus}\\
 & \textbf{seizures}\\
 & \textbf{ataxia}\\
Skeletal muscle & \textbf{myopathy, hypotonia}\\
 & \textbf{CPEO}\\
 & myoglobinuria\\
Marrow & sideroblastic anemia/pancytopenia\\
Kidney & fanconi\\
Endocrine & \textbf{diabetes}\\
 & hypoparathyroidism,\\
 & growth/multiple hormone deficiency\\
Heart & cardiomyopathy\\
 & conduction defects\\
GI & pancreatic failure\\
 & villous atrophy\\
Ear & \textbf{sensorineural deafness}\\
 & aminoglycoside deafness\\
Systemic & \textbf{lactic acidosis}\\
\end{tabular}
\end{table}

\begin{enumerate}
\item Myopathies
\label{sec:org64683f0}
\begin{itemize}
\item Chronic progressive external ophthalmoplegia (CPEO)
\begin{itemize}
\item w/wo retinitis pigmentosa
\item most common clinical manifestation
\item muscle biopsy is diagnostic
\end{itemize}
\item Kearns Sayre syndrome is a subtype of CPEO
\begin{itemize}
\item onset \textless{} 20
\item pigmentary retinopathy
\item cardiac conduction defect
\item ataxia, \(\uparrow\) CSF protein
\end{itemize}
\item Isolated limb myopathy
\end{itemize}

\item Encephalopathies
\label{sec:org4feb220}
\begin{itemize}
\item encephalopathic features:
\begin{itemize}
\item dementia/ID, ataxia, seizures, myoclonus, deafness, dystonia.
\end{itemize}
\item MELAS: myopathy, encephalopathy, lactic acidosis, stroke-like episodes
\begin{itemize}
\item most common mito encephalopathy
\end{itemize}
\item MERRF: myoclonic epilepsy w ragged red fibres
\begin{itemize}
\item ptosis, ataxia, deafness
\end{itemize}
\item Leigh Syndrome
\begin{itemize}
\item most frequent presentation of MD in childhood
\item subacute necrotising encephalomyelopathy
\item several biochemical defects including: PDH, OxPhos
\item MRI - lesions affecting basal ganglia and/or brain stem
\item \(\uparrow\) lactate blood and CSF
\item hypo/er-ventilation, spasticity, dystonia, ataxia, tremor, optic atrophy
\item cardiomyopathy, renal tubulopathy, GI disfunction
\item \textgreater{} 75 genes(mt and nuclear)
\item Saguenay-Lac-St-Jean type incidence 1/2000, gene prevelance 1/23
\end{itemize}
\end{itemize}
\item Leberâ€™s Hereditary Optic Neuropathy
\label{sec:org6e9542e}
\begin{itemize}
\item most common cause of blindness in otherwise healthy young men.
\item maternally inherited and manifests in late adolescence or early
adulthood as bilateral sequential visual failure.
\item 90\% of patients are affected by age 40
\end{itemize}

\item Pearson's Syndrome
\label{sec:orge559267}
\begin{itemize}
\item transfusion dependent sideroblastic anemia/pancytopenia
\item exocrine pancreas failure
\item progressive liver disease
\item renal tubular disease
\end{itemize}

\item Neonatal and Infantile Presentation
\label{sec:org3db90fe}
\begin{itemize}
\item Congenital Lactic Acidosis
\item Leigh Syndrome
\item MEGDEL: 3-methylglutaconic aciduria, deafness, encephalopathy and Leigh-like disease
\item Pearson's marrow-pancreas syndrome
\item MDDS: mitochondrial DNA depletion syndrome
\item Alper-Huttenlocher syndrome
\item Reversible infantile respiratory chain deficiency
\item Infantile onset Q\(_{\text{10}}\) biosynthetic defects
\end{itemize}

\item Childhood and Adolescent Presentation
\label{sec:org1c782bd}
\begin{itemize}
\item Kearn-Sayre syndrome
\item MELAS: myopathy, encephalopathy, lactic acidosis, stroke-like episodes
\item MERRF: myoclonic epilepsy w ragged red fibres
\item NARP: neuropathy, ataxia, retinitis pigmentosa
\item LHON: Leber's Hereditary Optic Neuropathy
\item MEMSA: myoclonic epilepsy, myopathy, sensory ataxia
\item MNGIE: mitochondrial neurogastrointestinal encephalopathy
\end{itemize}

\item Adult Presentation
\label{sec:orgb4910eb}
\begin{itemize}
\item MIDD: maternally inherited diabetes and deafness
\item PEO: Progressive External Opthalmoplegia
\item SANDO: Sensory Ataxic Neuropathy, dysarthria and opthalmoparesis
\end{itemize}
\end{enumerate}

\subsection{Investigations}
\label{sec:orgb4b31ba}

\begin{enumerate}
\item Biochemistry
\label{sec:orgcecd9c1}
\begin{itemize}
\item blood lactate, CSF lactate
\item L/P \(\uparrow\) at rest, \(\Uparrow\) after excercise
\item renal tubular dysfunction: urine anion gap, pH, serum K
\item Plasma amino acids:
\begin{itemize}
\item alanine \(\propto\) pyruvate
\item ala/lys normally \textless{} 3:1
\item \(\uparrow\) gly in lipoic acid biosynthesis defects
\item \(\downarrow\) cit and arg in Leigh, NARP, MELAS and Pearson
\end{itemize}
\item Urine organic acids
\begin{itemize}
\item lactate, pyruvate, TCA intermediates
\item 3-methylglutaconic acid in Barth, Sengers, MEGDEL, ATP synthase deficiency
\item ethylmalonic
\item mma in succinyl-CoA-ligase deficiency
\end{itemize}
\item Acylcarnitines
\begin{itemize}
\item flavin cofactor metabolism
\end{itemize}
\item Purine and pyrimidines (plasma or urine)
\begin{itemize}
\item MNGIE \(\uparrow\) thymidine and deoxyuridine
\end{itemize}
\item FGF-21, GDF15 and creatinine \(\propto\) mito disfunction in myopathy
\end{itemize}

\item Imaging
\label{sec:org072036e}
\begin{itemize}
\item Cranial CT shows cerebral and cerebellar atrophy in many encephalopathic patients
\begin{itemize}
\item basal ganglia calcification may be seen in MELAS.
\end{itemize}
\item MRI in MELAS-associated stroke reveals increased T2 weighted signals in the grey and white matter
\item Symmetrical changes in the basal ganglia and brainstem observed in Leigh syndrome.
\end{itemize}

\item Histology
\label{sec:orgda327b6}
\begin{itemize}
\item Muscle biopsy is diagnostic
\begin{itemize}
\item mitochondrial myopathy due to mtDNA mutations and LHON may have normal biopsies.
\end{itemize}
\item Ragged red fibres on Gomori trichrome staining, due to mitochondrial proliferation
\item fibres stain strongly for succinate dehydrogenase
\item fibres often negative for COX (complex IV) in CPEO, KSS, or MERRF but positive in MELAS.
\item Leigh syndrome patients may have no ragged red fibres and  COX-negative fibres only
\end{itemize}

\begin{figure}[htbp]
\centering
\includegraphics[width=0.5\textwidth]{./oxphos_disorders/figures/Ragged_red_fibers_in_MELAS.jpg}
\caption[rrf]{\label{fig:orged1045a}
Ragged red fibers - Gomori stain}
\end{figure}

\item Molecular
\label{sec:org9625404}
\begin{itemize}
\item no strict relation between phenotype and genotype.
\item mtDNA tRNA mutations are most common of the single base change abnormalities.
\begin{itemize}
\item A3243G in the tRNA\(^{\text{Leu(UUR)}}\) gene is most frequently found in MELAS
\item G8344A in tRNA\(^{\text{Lys}}\) in MERRF.
\item Many other tRNA mutations have been associated with other clinical phenotypes.
\end{itemize}
\item The primary mutations associated with LHON (G11778A, G3460A,T14484C) are in complex I genes ND4, ND1, and ND6.
\begin{itemize}
\item G11778A is most common, found in over 50\% of LHON families in the UK.
\end{itemize}
\end{itemize}
\end{enumerate}
\section{{\bfseries\sffamily TODO} Ketogenesis and Ketolysis}
\label{sec:org31fc3f9}
\section{{\bfseries\sffamily TODO} Creatine Deficiency}
\label{sec:orga5e1405}
\end{document}