#+TITLE: The Electron Transport Chain and Oxidative Phosphorylation
#+AUTHOR: Matthew Henderson, PhD, FCACB
#+DATE: \today

* Introduction
** ETC and OxPhos system
- Responsible for ATP production
- ETC complexes I-IV
- OxPhos system complexes I-V
*** Chemiosmotic Coupling Hypothesis
 - proposed by Nobel Prize in Chemistry winner Peter D. Mitchell
 - the ETC and OxPhos are coupled by a proton gradient across the IMM.
 - The efflux of protons from the mitochondrial matrix creates an electrochemical gradient.
   - used by the F_{O}F_{1} ATP synthase complex to make ATP via oxidative phosphorylation.

** Electron Transport Chain
- Energy from transfer of electrons down the ETC is used to pump
  protons from the mitochondrial matrix into the intermembrane space.
  - creates an electrochemical proton gradient (\Delta{}pH) across the IMM.
    - largely responsible for the mitochondrial membrane potential (\Delta\Psi{}M).
  - ATP synthase uses flow of \ce{H+} through the enzyme back into the
    matrix to generate ATP from ADP and Pi.
- There are three energy-transducing enzymes in the electron transport
  chain:
  - NADH:ubiquinone oxidoreductase (complex I)
  - Coenzyme Q – cytochrome c reductase (complex III)
  - cytochrome c oxidase (complex IV).
  - also ETF-QO and mitochondrial GPD

#+BEGIN_SRC dot :file ./etc/figures/etc.pdf :cmdline -Kdot -Tpdf
  digraph {
           // graph from left to right
           rankdir=LR;
           splines=true;
           node [shape=oval];
           fadh [label="FADH2", fillcolor=grey, style=filled];
           etf[label="ETF", fillcolor=steelblue3, style=filled];
           etfd[label="ETF-QO", fillcolor=steelblue3, style=filled];
	       g3p[label="glycerol-3-phosphate", fillcolor=grey, style=filled];
           gpd[label="GPD", fillcolor=steelblue3, style=filled];
           nadh [label="NADH", fillcolor=grey, style=filled];
           c1 [label="Complex I", fillcolor=steelblue3, style=filled];
           q [label="CoQ", fillcolor=red3, style=filled];    
           c3 [label="Complex III", fillcolor=steelblue3, style=filled];     
           cytc [label="Cytochrome C", fillcolor=red3, style=filled];     
           c4 [label="Complex IV", fillcolor=steelblue3, style=filled];     
           h2o [label="H2O", fillcolor=grey, style=filled];     
           c2 [label="Complex II", fillcolor=steelblue3, style=filled];     
           s [label="succinate", fillcolor=grey, style=filled];     
           fadh -> etf; 
           etf -> etfd;
           etfd -> q;
           g3p -> gpd;
           gpd -> q;
           nadh -> c1;
           c1 -> q;
           q -> c3;
           c3 -> cytc; 	 
           cytc -> c4;
           c4 -> h2o;
           c2 -> q;
           s -> c2;
           }    
#+END_SRC

#+RESULTS:
[[file:./etc/figures/etc.pdf]]

- Complex I (NADH coenzyme Q reductase) accepts electrons from the Krebs cycle electron carrier NADH
- passes them to CoQ (ubiquinone; labeled Q),
- CoQ also receives electrons from complex II (succinate dehydrogenase).
- CoQ passes electrons to complex III (cytochrome bc1 complex; labeled III), which passes them to cytochrome c (cyt c).
- Cyt c passes electrons to Complex IV (cytochrome c oxidase; labeled IV), which uses the electrons and hydrogen ions to reduce molecular oxygen to water. 

** Oxidative Phosphorylation

#+CAPTION[ETC]: Oxidative Phosphorylation, KEGG
#+NAME: fig:oxpho
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./etc/figures/hsa00190.png]]

** Mega- Complex

#+CAPTION[ETC mega complex]:ETC Mega Complex
#+NAME: fig:mega
#+ATTR_LaTeX: :width 0.8\textwidth
[[file:./etc/figures/etc_supercomplex.jpg]]

** Redox Potential and Free Energy in ETC

#+CAPTION[redox]:Electron flow to O_2 and release free energy
#+NAME: fig:redox
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./etc/figures/potential.png]]

* Complex I
** Complex I | NADH-ubiquinone oxidoreductase
- catalyzes the transfer of electrons from NADH to coenzyme Q10
  (CoQ10) and translocates protons across the inner mitochondrial
  membrane

#+BEGIN_EXPORT LaTeX
\centering
\small
\ce{NADH + H+ + CoQ + 4H^{+}_{in} ->[CI] NAD+ + CoQH2 + 4H^{+}_{out}}
#+END_EXPORT


- Mechanism: 
  1. Seven iron sulfur centers carry electrons from the site of NADH
     dehydration to ubiquinone.

  2. ubiquinone (CoQ) is reduced to ubiquinol (\ce{CoQH2}).

  3. The energy from the redox reaction results in conformational
     change allowing hydrogen ions to pass through four transmembrane
     helix channels.

#+BEGIN_SRC dot :file ./etc/figures/c1.pdf :cmdline -Kdot -Tpdf
  digraph {
           // graph from left to right
           rankdir=LR;
           splines=true;
           node [shape=circle];
           nadh [label="NADH", fillcolor=grey, style=filled]
           fmn [label="FMN", fillcolor=forestgreen, style=filled]
           n3 [label="N3", fillcolor=red3, style=filled]    
           n1b [label="N1b", fillcolor=red3, style=filled]     
           n4 [label="N4", fillcolor=red3, style=filled]     
           n5 [label="N5", fillcolor=red3, style=filled]     
           n6a [label="N6a", fillcolor=red3, style=filled]     
           n6b [label="N6b", fillcolor=red3, style=filled]     
           n2 [label="N2", fillcolor=red3, style=filled]     
           q [label="Q", fillcolor=steelblue3, style=filled]     
           nadh-> fmn
           fmn -> n3
           n3 -> n1b
	       n1b -> n4
           n4 -> n5 
           n5 -> n6a
           n6a -> n6b
           n6b-> n2
           n2 -> q
           }    
#+END_SRC

#+CAPTION[c1]:Complex I
#+NAME: fig:c1
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./etc/figures/CI.png]]

# https://upload.wikimedia.org/wikipedia/commons/4/42/NADH_Dehydrogenase_Mechanism_%28Fixed%29.png

# https://upload.wikimedia.org/wikipedia/commons/4/42/Complex_I.svg

** Complex I Inhibitors

- The best-known inhibitor of complex I is rotenone
  - commonly used as an organic pesticide
- Rotenone binds to the ubiquinone binding site of complex I
  -  piericidin A a potent inhibitor and structural homologue to ubiquinone.
- Hydrophobic inhibitors like rotenone or piericidin likely disrupt electron transfer between FeS cluster N2 and ubiquinone.
- Bullatacin is the most potent known inhibitor of NADH dehydrogenase (ubiquinone)
- Complex I is also blocked by adenosine diphosphate ribose – a reversible competitive inhibitor of NADH oxidation

* Complex II
** Complex II | Succinate Dehydrogenase 
- Four subunits compose Complex II of the mitochondrial respiratory chain

| Subunit name | Protein description                                |
|--------------+----------------------------------------------------|
| SdhA         | Succinate dehydrogenase flavoprotein subunit       |
| SdhB         | Succinate dehydrogenase iron-sulfur subunit        |
| SdhC         | Succinate dehydrogenase cytochrome b560 subunit    |
| SdhD         | Succinate dehydrogenase cytochrome b small subunit |

- The SdhA subunit contains an FAD binding site where succinate
  is deprotonated and converted to fumarate.

#+BEGIN_EXPORT LaTeX
\centering
\ce{succinate + ubiquinone ->[CII] fumarate + ubiquinol}
#+END_EXPORT

- Electrons removed from succinate transfer to SdhA
- transfer across SdhB through iron sulphur clusters to the SdhC/SdhD subunits
  - SdhC/SdhD are anchored in the mitochondrial membrane.


#+CAPTION[cII]:Complex II
#+NAME: fig:cII
#+ATTR_LaTeX: :width 0.8\textwidth
[[file:./etc/figures/CII.png]]

** Complex II Inhibitors

- There are two distinct classes of inhibitors of complex II:
  - those that bind in the succinate pocket and those that bind in the ubiquinone pocket.
- Ubiquinone type inhibitors include carboxin and thenoyltrifluoroacetone.
- Succinate-analogue inhibitors include the synthetic compound malonate as well as the TCA cycle intermediates, malate and oxaloacetate.
  - oxaloacetate is one of the most potent inhibitors of Complex II. 
* Glycerol-3-phosphate shuttle
** Glycerol-3-phosphate shuttle
- Oxidation of cytoplasmic NADH by the cytosolic form of the enzyme
  creates glycerol-3-phosphate from dihydroxyacetone phosphate.
- Glycerol-3-phosphate diffuses into IMM and is oxidised by mitochondrial glycerol-3-phosphate dehydrogenase
  - uses quinone as an oxidant and FAD as a co-factor.
- maintains the cytoplasmic NAD+/NADH ratio.

#+CAPTION[g3ps]:Glycerol-3-phosphate shuttle
#+NAME: fig:g3ps
#+ATTR_LaTeX: :width 0.8\textwidth
[[file:./etc/figures/GPDH_shuttle.png]]

* Electron Transferring Flavoprotein/ Dehydrogenase
** Electron Transferring Flavoprotein/ Dehydrogenase
- ETFs are heterodimeric proteins composed of an alpha and beta subunit (ETFA and ETFB), and contain an FAD cofactor and AMP

- ETQ-QO links the oxidation of fatty acids and some amino acids to
  oxidative phosphorylation in the mitochondria.
- catalyzes the transfer of electrons from electron transferring
  flavoprotein (ETF) to ubiquinone, reducing it to ubiquinol.

#+BEGIN_EXPORT LaTeX
\small
\centering
\ce{Acyl-CoA + FAD ->[ACAD] FADH2 + ETF ->[ETF-QO] UQ ->[CIII] CytC}
#+END_EXPORT

* Complex III
** Complex III | Coenzyme Q – cytochrome c reductase

- Complex III is a multi-subunit transmembrane protein encoded by both
  mitochondrial (cytochrome b) and the nuclear genomes (all other
  subunits)

- The bc1 complex contains 11 subunits:
  - 3 respiratory subunits (cytochrome B, cytochrome C1, Rieske protein)
  - 2 core proteins
  - 6 low-molecular weight proteins

#+BEGIN_EXPORT LaTeX
\centering
\small
\ce{QH2 + 2Fe^{3+}-cyt c + 2H+_{in} ->[CIII]  Q + 2Fe^{2+}-cyt c + 4H+_{out}}
#+END_EXPORT

*** Mechanism 
- Round 1:
    - Cytochrome b binds a ubiquinol and a ubiquinone.
    - The 2Fe/2S center and BL heme each pull an electron off the bound ubiquinol, releasing two hydrogens into the intermembrane space.
    - One electron is transferred to cytochrome c1 from the 2Fe/2S centre, whilst another is transferred from the BL heme to the BH Heme.
    - Cytochrome c1 transfers its electron to cytochrome c (not to be confused with cytochrome c1), and the BH Heme transfers its electron to a nearby ubiquinone, resulting in the formation of a ubisemiquinone.
    - Cytochrome c diffuses. The first ubiquinol (now oxidised to ubiquinone) is released, whilst the semiquinone remains bound.

- Round 2:
    - A second ubiquinol is bound by cytochrome b.
    - The 2Fe/2S center and BL heme each pull an electron off the bound ubiquinol, releasing two hydrogens into the intermembrane space.
    - One electron is transferred to cytochrome c1 from the 2Fe/2S centre, whilst another is transferred from the BL heme to the BH Heme.
    - Cytocrome c1 then transfers its electron to cytochrome c, whilst the nearby semiquinone produced from round 1 picks up a second electron from the BH heme, along with two protons from the matrix.
    - The second ubiquinol (now oxidised to ubiquinone), along with the newly formed ubiquinol are released.[8]

#+CAPTION[cIII]: Complex III two step reaction 
#+NAME: fig:cIII
#+ATTR_LaTeX: :width 0.9\textwidth
[[file:./etc/figures/CIII.png]]

** Complex III Inhibitors

- There are three distinct groups of Complex III inhibitors:
  - Antimycin A binds to the Q_i site and inhibits the transfer of electrons in Complex III from heme b_H to oxidized Q (Q_i site inhibitor).
  - Myxothiazol and stigmatellin bind to distinct but overlapping pockets within the Q_o site.
    - Myxothiazol binds nearer to cytochrome bL (hence termed a "proximal" inhibitor).
    - Stigmatellin binds farther from heme bL and nearer the Rieske Iron sulfur protein.
    - Both inhibit the transfer of electrons from reduced QH_2 to the Rieske Iron sulfur protein.

* Complex IV
** Complex IV | Cytochrome c oxidase

- last enzyme in the respiratory electron transport chain.
- large IMM integral membrane protein composed of several metal prosthetic sites and 14 protein subunits.
- eleven subunits are nuclear in origin, and three are synthesized in the mitochondria. 
  - contains two hemes,
  - cytochrome a and cytochrome a3,
  - two copper centers, CuA and CuB
- the cytochrome a3 and CuB form a binuclear center that is the site of oxygen reduction.
- receives an electron from four cytochrome c molecules and transfers them to one O_2 molecule
 
#+BEGIN_EXPORT LaTeX
\centering
\small
\ce{4Fe^{2+}-cyt c + 8H^{+}_{in} + O2 ->[CIV] 4Fe^{3+}-cyt c + 2H2O + 4H^{+}_{out}}
#+END_EXPORT

- In the process binds four protons from the inner aqueous phase to
  make two water molecules, and translocates another four protons
  across the membrane, increasing the transmembrane difference of
  proton electrochemical potential which the ATP synthase then uses to
  synthesize ATP.

#+CAPTION[cIV]:Complex IV
#+NAME: fig:cIV
#+ATTR_LaTeX: :width 0.7\textwidth
[[file:./etc/figures/CIV.png]]

** Complex IV | Inhibitors

- Cyanide, azide, and carbon monoxide all bind to cytochrome c
  oxidase 

- Nitric oxide and hydrogen sulfide, can also inhibit COX by
  binding to regulatory sites on the enzyme

* Complex V
** Complex V | ATP synthase 

- ATP synthase is a molecular machine that creates the energy storage
  molecule adenosine triphosphate (ATP).

- The overall reaction catalyzed by ATP synthase is:

#+BEGIN_EXPORT LaTeX
\centering
  \ce{ADP + P_i + H+_{out} <=> ATP + H2O + H+_{in}}
#+END_EXPORT


- Formation of ATP from ADP and P_{i} is energetically unfavourable
  - would normally proceed in the reverse direction. 

- To drive this reaction forward, ATP synthase couples ATP synthesis
  to the electrochemical gradient (\Delta\Psi{}M) created by complexes
  I,III and IV

- ATP synthase consists of two main subunits, FO and F1, which has a
  rotational motor mechanism allowing for ATP production.


Simplified picture of ATP syntase The Fo part through which hydrogen
ions (H+) stream is located in the membrane. The F1 part which
synthesises ATP is outside the membrane. When the hydrogen ions flow
through the membrane via the disc of c subunits in the Fo part, the
disc is forced to twist around. The gamma subunit in the F1 part is
attached to the disc and therefore rotates with it. The three alpha
and three beta subunits in the F1 part cannot rotate, however. They
are locked in a fixed position by the b subunit. This in turn is
anchored in the membrane. Thus the gamma subunit rotates inside the
cylinder formed by the six alpha and beta subunits. Since the gamma
subunit is asymmetrical it compels the beta subunits to undergo
structural changes. This leads to the beta subunits binding ATP and
ADP with differing strengths (see Figure 2).


Figure 2. Boyer’s “Binding Change Mechanism” The picture shows the
cylinder with alternating alpha and beta subunits at four different
stages of ATP synthesis. The asymmetrical gamma subunit that causes
changes in the structure of the beta subunits can be seen in the
centre. The structures are termed open betaO (light grey sector),
loose betaL (grey sector) and tight betaT (black sector). At stage A
we see an already-fully-formed ATP molecule bound to betaT. In the
step to stage B betaL binds ADP and inorganic phosphate (Pi ). At the
next stage, C, we see how the gamma subunit has twisted due to the
flow of hydrogen ions (see Figure 1). This brings about changes in the
structure of the three beta subunits. The tight beta subunit now
becomes open and the bound ATP molecule is released. The loose beta
subunit becomes tight and the open becomes loose. In the last stage
the chemical reaction takes place in which phosphate ions react with
the ADP molecule to form a new ATP molecule. We are back at the first
stage.


#+NAME: fig:atps
#+ATTR_LaTeX: :width 0.5\textwidth
[[file:./etc/figures/atp_synthase.jpg]]


#+BEGIN_EXPORT LaTeX
\centering
\ce{ADP + Pi + H+_{out} <=> ATP + H2O + H+_{in}}
#+END_EXPORT


** Complex V Inhibitors


- Oligomycin A inhibits ATP synthase by blocking its proton channel
  (Fo subunit), which is necessary for oxidative phosphorylation of
  ADP to ATP (energy production).
- The inhibition of ATP synthesis by oligomycin A will significantly
  reduce electron flow through the electron transport chain; however,
  electron flow is not stopped completely due to a process known as
  proton leak or mitochondrial uncoupling.
  - This process is due to facilitated diffusion of protons into the
    mitochondrial matrix through an uncoupling protein such as
    thermogenin, or UCP1.

- Administering oligomycin to an individual can result in very high
  levels of lactate accumulating in the blood and urine.

* Metabolic Derangement

** Anaerobic Glycolysis
- Complex V harnesses the proton gradient created by Complexes I, III, and IV
  - produces the majority of cellular ATP
- Insufficient ATP severely affects highly energy dependant tissues
  - A complete loss of OxPhos is not observed in human disease.
- In the absence of OxPhos cells survive using ATP from anaerobic glycolysis
  - 20x less efficient, generates lactate
  - pyruvate \to alanine if glutamate is available 
- Lactate, pyruvate and alanine are the typical products of anaerobic glycolysis


** Factors Affecting OxPhos System

- ~ 90 subunits
  - 13 subunits of Complexes I, III, IV and V encoded by mtDNA
- mitochondrial replication, transcription and translation
  - require \gt 200 proteins, rRNAs and tRNAs
- Cofactors: coenzyme Q_{10}, iron-sulfur clusters, heme, copper
  - require synthesis and/or transport to OxPhos system
- Cardiolipin required for cristae formation
- Mitochondrial function
  - protein import, turnover
  - fission, fusion
- Toxic metabolites

- \gt 1500 proteins in the human mitochondrial proteome
  - other additional factors - lipids, cofactors
  - up to 10% of human proteome may be involved in mitochondria 

** Types of genetic defects and affected systems

*** Type of Defect
- OxPhos Subunit
- Assembly Factor
- mtDNA replication
- mtDNA transcription
- mitochondrial transcription
- cofactor
- mitochondrial homeostasis
- inhibitor

*** Affected Systems
\scriptsize
- Neurological disease
  - \scriptsize Leigh disease
  - Epilepsy
  - Leukodystropy
  - Periperal neuropathy
- Eye disease
- Deafness
- Cardiac disease
- Pulmonary disease
- GI disease
- Pancreas endocrine/exocrine
- Endocrine
- Liver disease
- Kidney
- Ovarian failure
- Hematological
- Myopathy

